<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>7.3 Activity - Introducing 16S rRNA Data Using Spreadsheet | miniCURE: Human Gut 16S rRNA</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.43 and GitBook 2.6.7" />

  <meta property="og:title" content="7.3 Activity - Introducing 16S rRNA Data Using Spreadsheet | miniCURE: Human Gut 16S rRNA" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="7.3 Activity - Introducing 16S rRNA Data Using Spreadsheet | miniCURE: Human Gut 16S rRNA" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  



<meta name="date" content="2025-09-18" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/dasl_favicon.ico" type="image/x-icon" />
<link rel="prev" href="lecture---introducing-16s-rrna-data-using-spreadsheets.html"/>
<link rel="next" href="exploring-16s-rrna-data-with-phyloseq.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<base target="_blank">


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="assets/style.css" type="text/css" />
<link rel="stylesheet" href="assets/C-MOOR_Template/c-moor-ottr.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.c-moor.org/" target="_blank"><img src="assets/C-MOOR_Template/c-moor-logo-vertical.png" style="width: 70%; padding-left: 10px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#learning-goals"><i class="fa fa-check"></i>Learning Goals</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="c-moor-overview.html"><a href="c-moor-overview.html"><i class="fa fa-check"></i><b>1</b> C-MOOR Overview</a>
<ul>
<li class="chapter" data-level="1.0.1" data-path="c-moor-overview.html"><a href="c-moor-overview.html#about-c-moor"><i class="fa fa-check"></i><b>1.0.1</b> About C-MOOR</a></li>
<li class="chapter" data-level="1.0.2" data-path="c-moor-overview.html"><a href="c-moor-overview.html#why-data-science"><i class="fa fa-check"></i><b>1.0.2</b> Why data science?</a></li>
<li class="chapter" data-level="1.0.3" data-path="c-moor-overview.html"><a href="c-moor-overview.html#learning-goals-1"><i class="fa fa-check"></i><b>1.0.3</b> Learning Goals</a></li>
</ul></li>
<li class="part"><span><b>INSTRUCTOR GUIDE</b></span></li>
<li class="chapter" data-level="2" data-path="for-new-instructors.html"><a href="for-new-instructors.html"><i class="fa fa-check"></i><b>2</b> For new instructors</a>
<ul>
<li class="chapter" data-level="2.0.1" data-path="for-new-instructors.html"><a href="for-new-instructors.html#course-based-undergraduate-research-experiences"><i class="fa fa-check"></i><b>2.0.1</b> Course-based undergraduate research experiences</a></li>
<li class="chapter" data-level="2.0.2" data-path="for-new-instructors.html"><a href="for-new-instructors.html#how-c-moor-supports-you"><i class="fa fa-check"></i><b>2.0.2</b> How C-MOOR supports you</a></li>
<li class="chapter" data-level="2.0.3" data-path="for-new-instructors.html"><a href="for-new-instructors.html#new-instructor-to-do-list"><i class="fa fa-check"></i><b>2.0.3</b> New instructor to-do list</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html"><i class="fa fa-check"></i><b>3</b> Human gut 16S rRNA</a>
<ul>
<li class="chapter" data-level="3.0.1" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#content-overview"><i class="fa fa-check"></i><b>3.0.1</b> Content overview</a></li>
<li class="chapter" data-level="3.0.2" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#sample-schedules"><i class="fa fa-check"></i><b>3.0.2</b> Sample Schedules</a></li>
<li class="chapter" data-level="3.0.3" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#what-to-know"><i class="fa fa-check"></i><b>3.0.3</b> What to know</a></li>
<li class="chapter" data-level="3.0.4" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#where-to-get-help"><i class="fa fa-check"></i><b>3.0.4</b> Where to get help</a></li>
<li class="chapter" data-level="3.0.5" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#grades"><i class="fa fa-check"></i><b>3.0.5</b> Grades</a></li>
<li class="chapter" data-level="3.0.6" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#footnotes"><i class="fa fa-check"></i><b>3.0.6</b> Footnotes</a></li>
</ul></li>
<li class="part"><span><b>Setting up on a cloud platform</b></span></li>
<li class="chapter" data-level="4" data-path="anvil.html"><a href="anvil.html"><i class="fa fa-check"></i><b>4</b> AnVIL</a>
<ul>
<li class="chapter" data-level="4.1" data-path="about-anvil.html"><a href="about-anvil.html"><i class="fa fa-check"></i><b>4.1</b> About AnVIL</a></li>
<li class="chapter" data-level="4.2" data-path="sign-up-for-an-anvil-account.html"><a href="sign-up-for-an-anvil-account.html"><i class="fa fa-check"></i><b>4.2</b> Sign up for an AnVIL account</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="sign-up-for-an-anvil-account.html"><a href="sign-up-for-an-anvil-account.html#create-an-anvil-account"><i class="fa fa-check"></i><b>4.2.1</b> Create an AnVIL account</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="set-up-billing-in-anvil.html"><a href="set-up-billing-in-anvil.html"><i class="fa fa-check"></i><b>4.3</b> Set up billing in AnVIL</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="set-up-billing-in-anvil.html"><a href="set-up-billing-in-anvil.html#what-is-a-billing-project"><i class="fa fa-check"></i><b>4.3.1</b> What is a billing project?</a></li>
<li class="chapter" data-level="4.3.2" data-path="set-up-billing-in-anvil.html"><a href="set-up-billing-in-anvil.html#adding-students-to-a-billing-account"><i class="fa fa-check"></i><b>4.3.2</b> Adding students to a billing account</a></li>
<li class="chapter" data-level="4.3.3" data-path="set-up-billing-in-anvil.html"><a href="set-up-billing-in-anvil.html#using-groups-to-manage-classes"><i class="fa fa-check"></i><b>4.3.3</b> Using groups to manage classes</a></li>
<li class="chapter" data-level="4.3.4" data-path="set-up-billing-in-anvil.html"><a href="set-up-billing-in-anvil.html#preventing-runaway-costs"><i class="fa fa-check"></i><b>4.3.4</b> Preventing runaway costs</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html"><i class="fa fa-check"></i><b>4.4</b> Running a module on AnVIL</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#what-is-a-workspace"><i class="fa fa-check"></i><b>4.4.1</b> What is a workspace?</a></li>
<li class="chapter" data-level="4.4.2" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#clone-a-workspace-on-anvil"><i class="fa fa-check"></i><b>4.4.2</b> Clone a workspace on AnVIL</a></li>
<li class="chapter" data-level="4.4.3" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#running-modules-on-anvil"><i class="fa fa-check"></i><b>4.4.3</b> Running modules on AnVIL</a></li>
<li class="chapter" data-level="4.4.4" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#closing-out-a-session-on-anvil"><i class="fa fa-check"></i><b>4.4.4</b> Closing out a session on AnVIL</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Scientific Literature</b></span></li>
<li class="chapter" data-level="5" data-path="scientific-literature.html"><a href="scientific-literature.html"><i class="fa fa-check"></i><b>5</b> Scientific Literature</a>
<ul>
<li class="chapter" data-level="5.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html"><i class="fa fa-check"></i><b>5.1</b> Pre-lab - Scientific literature</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#purpose-3"><i class="fa fa-check"></i><b>5.1.1</b> Purpose</a></li>
<li class="chapter" data-level="5.1.2" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#grading-criteria"><i class="fa fa-check"></i><b>5.1.2</b> Grading criteria</a></li>
<li class="chapter" data-level="5.1.3" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#activity"><i class="fa fa-check"></i><b>5.1.3</b> Activity</a></li>
<li class="chapter" data-level="5.1.4" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#footnotes-1"><i class="fa fa-check"></i><b>5.1.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="lecture---reading-a-paper.html"><a href="lecture---reading-a-paper.html"><i class="fa fa-check"></i><b>5.2</b> Lecture - Reading a paper</a></li>
<li class="chapter" data-level="5.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html"><i class="fa fa-check"></i><b>5.3</b> Activity - Scientific Literature</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#purpose-4"><i class="fa fa-check"></i><b>5.3.1</b> Purpose</a></li>
<li class="chapter" data-level="5.3.2" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#learning-objectives-3"><i class="fa fa-check"></i><b>5.3.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.3.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#introduction"><i class="fa fa-check"></i><b>5.3.3</b> Introduction</a></li>
<li class="chapter" data-level="5.3.4" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#instructions-1"><i class="fa fa-check"></i><b>5.3.4</b> Instructions</a></li>
<li class="chapter" data-level="5.3.5" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-1---overview-of-the-paper"><i class="fa fa-check"></i><b>5.3.5</b> Activity 1 - Overview of the paper</a></li>
<li class="chapter" data-level="5.3.6" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-2---methods-and-figure-analysis-in-class-and-homework"><i class="fa fa-check"></i><b>5.3.6</b> Activity 2 - Methods and Figure Analysis (in class and homework)</a></li>
<li class="chapter" data-level="5.3.7" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#grading-criteria-1"><i class="fa fa-check"></i><b>5.3.7</b> Grading criteria</a></li>
<li class="chapter" data-level="5.3.8" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#footnotes-2"><i class="fa fa-check"></i><b>5.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Project: 16S rRNA Research</b></span></li>
<li class="chapter" data-level="6" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html"><i class="fa fa-check"></i><b>6</b> Reference - Meet the MISO study</a>
<ul>
<li class="chapter" data-level="6.0.1" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#miso-study-diet-and-sampling-schedule"><i class="fa fa-check"></i><b>6.0.1</b> MISO study diet and sampling schedule</a></li>
<li class="chapter" data-level="6.0.2" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#miso-study-variables-and-factors"><i class="fa fa-check"></i><b>6.0.2</b> MISO study variables and factors</a></li>
<li class="chapter" data-level="6.0.3" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#amplicon-sequence-variants-asvs-data"><i class="fa fa-check"></i><b>6.0.3</b> Amplicon sequence variants (ASVs) data</a></li>
<li class="chapter" data-level="6.0.4" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#footnotes-3"><i class="fa fa-check"></i><b>6.0.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="s-rrna-data-in-spreadsheets.html"><a href="s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>7</b> 16S rRNA Data in spreadsheets</a>
<ul>
<li class="chapter" data-level="7.1" data-path="exploring-16s-rrna-data-in-spreadsheets.html"><a href="exploring-16s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>7.1</b> Exploring 16S rRNA data in spreadsheets</a></li>
<li class="chapter" data-level="7.2" data-path="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><a href="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><i class="fa fa-check"></i><b>7.2</b> Lecture - Introducing 16S rRNA Data Using Spreadsheets</a></li>
<li class="chapter" data-level="7.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html"><i class="fa fa-check"></i><b>7.3</b> Activity - Introducing 16S rRNA Data Using Spreadsheet</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#learning-objectives-4"><i class="fa fa-check"></i><b>7.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="7.3.2" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#introduction-1"><i class="fa fa-check"></i><b>7.3.2</b> Introduction</a></li>
<li class="chapter" data-level="7.3.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-1-quick-data-overview-based-on-1-sample"><i class="fa fa-check"></i><b>7.3.3</b> Activity 1 – Quick data overview based on 1 sample</a></li>
<li class="chapter" data-level="7.3.4" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-explore-variation-between-individuals"><i class="fa fa-check"></i><b>7.3.4</b> Activity 2 – Explore variation between individuals</a></li>
<li class="chapter" data-level="7.3.5" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-3.-plot-individual-variation-in-abundance-of-the-most-common-asv"><i class="fa fa-check"></i><b>7.3.5</b> Activity 2-3. Plot individual variation in abundance of the most common ASV</a></li>
<li class="chapter" data-level="7.3.6" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-3-explore-variation-within-a-single-individual-associated-with-diet"><i class="fa fa-check"></i><b>7.3.6</b> Activity 3 – Explore variation within a single individual associated with diet</a></li>
<li class="chapter" data-level="7.3.7" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-4-try-it-out"><i class="fa fa-check"></i><b>7.3.7</b> Activity 4 – Try it Out!</a></li>
<li class="chapter" data-level="7.3.8" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#grading-criteria-2"><i class="fa fa-check"></i><b>7.3.8</b> Grading criteria</a></li>
<li class="chapter" data-level="7.3.9" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#footnotes-4"><i class="fa fa-check"></i><b>7.3.9</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="exploring-16s-rrna-data-with-phyloseq.html"><a href="exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>8</b> Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="8.1" data-path="exploring-16s-rrna-data-with-phyloseq-1.html"><a href="exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>8.1</b> Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="8.2" data-path="lecture---exploring-16s-rrna-data-with-phyloseq.html"><a href="lecture---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>8.2</b> Lecture - Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="8.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>8.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="8.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#learning-objectives-5"><i class="fa fa-check"></i><b>8.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="8.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#introduction-2"><i class="fa fa-check"></i><b>8.3.2</b> Introduction</a></li>
<li class="chapter" data-level="8.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-1-explore-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>8.3.3</b> Activity 1 – Explore 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="8.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-2-try-it-out-questions"><i class="fa fa-check"></i><b>8.3.4</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="8.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#grading-criteria-3"><i class="fa fa-check"></i><b>8.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="8.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#footnotes-5"><i class="fa fa-check"></i><b>8.3.6</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><i class="fa fa-check"></i><b>8.4</b> Try it Question 1 - What has more impact on human microbial variation, diet or individuality</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1a-plot-high-level-taxonomy-phylum-level-for-each-timepoint-using-the-plot_bar-function.-specifically-your-plot-should-show-phyla-diversity-across-all-5-timepoints."><i class="fa fa-check"></i><b>8.4.1</b> Step 1A: Plot high-level taxonomy (Phylum level) for each timepoint using the plot_bar() function.** Specifically, your plot should show Phyla diversity across all 5 timepoints.</a></li>
<li class="chapter" data-level="8.4.2" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1b-plot-high-level-taxonomy-phylum-level-for-each-individual-using-only-normalized-homogenized-diet-hd-timepoints-to-minimize-contribution-of-diet-to-the-analysis-using-the-plot_bar-function.-specifically-your-plot-should-show-phyla-diversity-across-all-21-individuals-at-hd-timepoints."><i class="fa fa-check"></i><b>8.4.2</b> Step 1B: Plot high-level taxonomy (Phylum level) for each individual, using only normalized homogenized diet “HD” timepoints to minimize contribution of diet to the analysis using the plot_bar() function. Specifically, your plot should show Phyla diversity across all 21 individuals at HD timepoints.</a></li>
</ul></li>
<li class="chapter" data-level="8.5" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><i class="fa fa-check"></i><b>8.5</b> Try it Question 2 - What are some of the most abundant microbes when viewed at different taxonomic resolutions</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2a.-plot-phyla-across-all-subjects-and-identify-the-most-abundant-phyla.-specifically-your-plot-should-show-phyla-diversity-across-all-21-individuals."><i class="fa fa-check"></i><b>8.5.1</b> Step 2A. Plot Phyla across all subjects and identify the most abundant Phyla.** Specifically, your plot should show Phyla diversity across all 21 individuals.</a></li>
<li class="chapter" data-level="8.5.2" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2b.-subset-the-most-abundant-phyla-and-plot-order-skip-class"><i class="fa fa-check"></i><b>8.5.2</b> Step 2B. Subset the most abundant Phyla, and plot Order (Skip Class)</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>8.6</b> Try it Question 3 - Does gender contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="8.6.1" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3c.-test-gender-based-difference-in-species-for-the-low-abundant-phylum-you-selected.-specifically-your-plot-should-show-species-diversity-within-male-and-female-groups-for-only-your-phylum-of-choice."><i class="fa fa-check"></i><b>8.6.1</b> Step 3C. Test gender-based difference in Species for the low abundant phylum you selected.** Specifically, your plot should show species diversity within male and female groups for only your phylum of choice.</a></li>
<li class="chapter" data-level="8.6.2" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3d-plot-the-individual-abundance-of-2-candidate-microbes-for-gender-differences-based-on-your-analysis-above-andor-the-curated-menu-of-microbes-below-previously-associated-with-gender-variation-with-color-denoting-gender."><i class="fa fa-check"></i><b>8.6.2</b> Step 3D: Plot the individual abundance of 2 candidate microbes for gender differences based on your analysis above, and/or the curated menu of microbes below (previously associated with gender variation), with color denoting gender.</a></li>
</ul></li>
<li class="chapter" data-level="8.7" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>8.7</b> Try it Question 4 - Does age contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="8.7.1" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4a.-perform-an-initial-survey-to-see-if-there-are-any-differences-in-phyla-related-to-age."><i class="fa fa-check"></i><b>8.7.1</b> Step 4A. Perform an initial survey to see if there are any differences in phyla related to age.</a></li>
<li class="chapter" data-level="8.7.2" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4b.-merge-samples-by-age-then-re-normalize-and-finally-re-plot-phyla-by-age."><i class="fa fa-check"></i><b>8.7.2</b> Step 4B. Merge samples by age, then re-normalize and finally re-plot Phyla by age.</a></li>
<li class="chapter" data-level="8.7.3" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4c.-candidate-phyla-implicated-in-age-variation-include-firmicutes-and-bacteroidetes.-plot-these-specific-phyla-on-their-own-separately-to-observe-any-shift-that-occurs-with-age.-use-your-new-merged-and-normalized-merge-phyloseq-object-as-input."><i class="fa fa-check"></i><b>8.7.3</b> Step 4C. Candidate Phyla implicated in age variation include Firmicutes and Bacteroidetes. Plot these specific phyla on their own, separately, to observe any shift that occurs with age.** Use your new merged and normalized ‘merge’ phyloseq object as input.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="analyzing-16s-rrna-data-with-phyloseq.html"><a href="analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>9</b> Analyzing 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="9.1" data-path="analyzing-16s-rrna-data-with-phyloseq-1.html"><a href="analyzing-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>9.1</b> Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="9.2" data-path="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><a href="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>9.2</b> Lecture - Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="9.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>9.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#learning-objectives-6"><i class="fa fa-check"></i><b>9.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="9.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#introduction-3"><i class="fa fa-check"></i><b>9.3.2</b> Introduction</a></li>
<li class="chapter" data-level="9.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-1-analyze-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>9.3.3</b> Activity 1 – Analyze 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="9.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-2-try-it-out-questions-1"><i class="fa fa-check"></i><b>9.3.4</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="9.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#grading-criteria-4"><i class="fa fa-check"></i><b>9.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="9.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#footnotes-6"><i class="fa fa-check"></i><b>9.3.6</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><i class="fa fa-check"></i><b>9.4</b> Try it Question 1 - How sensitive is microbial diversity to variables like diet, age and gender</a>
<ul>
<li class="chapter" data-level="9.4.1" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1a.-plot-alpha-diversity-of-the-full-miso-dataset-by-subject-individuals."><i class="fa fa-check"></i><b>9.4.1</b> Step 1A. Plot alpha diversity of the full MISO dataset by subject (individuals).</a></li>
<li class="chapter" data-level="9.4.2" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1b.-plot-alpha-diversity-of-the-full-miso-dataset-by-timepointgroup."><i class="fa fa-check"></i><b>9.4.2</b> Step 1B. Plot alpha diversity of the full MISO dataset, by timepointgroup.</a></li>
<li class="chapter" data-level="9.4.3" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1c.-plot-alpha-diversity-of-the-full-miso-dataset-by-age."><i class="fa fa-check"></i><b>9.4.3</b> Step 1C. Plot alpha diversity of the full MISO dataset by age.</a></li>
<li class="chapter" data-level="9.4.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1d.-plot-alpha-diversity-of-the-full-miso-dataset-by-gender."><i class="fa fa-check"></i><b>9.4.4</b> Step 1D. Plot alpha diversity of the full MISO dataset by gender.</a></li>
<li class="chapter" data-level="9.4.5" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-2.-plot-alpha-diversity-of-the-full-miso-dataset-grouping-based-on-levels-for-5-different-metabolites-creatinine-pcs-is-hipp-pag."><i class="fa fa-check"></i><b>9.4.5</b> Step 2. Plot alpha diversity of the full MISO dataset, grouping based on levels for 5 different metabolites: Creatinine, PCS, IS, HIPP, PAG.</a></li>
<li class="chapter" data-level="9.4.6" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-3.-select-data-for-poster-and-include-the-code-used-to-generate-the-figure."><i class="fa fa-check"></i><b>9.4.6</b> Step 3. Select data for poster and include the code used to generate the figure.</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><i class="fa fa-check"></i><b>9.5</b> Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-1.-make-a-pcoa-plot-ordinating-on-the-entire-miso-dataset-and-coloring-by-individual.-investigate-resulting-plot-shape."><i class="fa fa-check"></i><b>9.5.1</b> Step 1. Make a PcOA plot, ordinating on the entire miso dataset, and coloring by individual. Investigate resulting plot shape.</a></li>
<li class="chapter" data-level="9.5.2" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-2.-using-the-pcoa-plot-from-step-1-above-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-diet-gender-and-age"><i class="fa fa-check"></i><b>9.5.2</b> Step 2. Using the PCoA plot from Step 1 above, look for any correlations between the shape of your PCoA plot and diet, gender and age</a></li>
<li class="chapter" data-level="9.5.3" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-3.-using-the-pcoa-plot-from-step-1-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-the-level-of-metabolites."><i class="fa fa-check"></i><b>9.5.3</b> Step 3. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and the level of metabolites.</a></li>
<li class="chapter" data-level="9.5.4" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-4.-one-way-of-checking-if-the-metabolite-levels-indeed-correlate-with-your-data-is-to-see-if-subsetting-smaller-chunks-or-specific-chunks-of-the-data-will-still-maintain-the-metabolite-data-relationship-or-break-it."><i class="fa fa-check"></i><b>9.5.4</b> Step 4. One way of checking if the metabolite levels indeed correlate with your data, is to see if subsetting smaller chunks or specific chunks of the data will still maintain the metabolite-data relationship or break it.</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><i class="fa fa-check"></i><b>9.6</b> Try it Question 3 - What microbes (ASVs) differ between males and females, and does age have an impact</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3a.-perform-differential-abundance-analysis-between-genders."><i class="fa fa-check"></i><b>9.6.1</b> Step 3A. Perform differential abundance analysis between genders.</a></li>
<li class="chapter" data-level="9.6.2" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3b.-determine-the-impact-of-younger-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>9.6.2</b> Step 3B. Determine the impact of younger age on differential abundance in males and females, by subsetting age &lt; 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
<li class="chapter" data-level="9.6.3" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3c.-determine-the-impact-of-older-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>9.6.3</b> Step 3C. Determine the impact of older age on differential abundance in males and females, by subsetting age &gt;= 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
<li class="chapter" data-level="9.6.4" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-4.-select-data-for-poster-and-include-the-code-used-to-generate-the-figure"><i class="fa fa-check"></i><b>9.6.4</b> Step 4. Select data for poster and include the code used to generate the figure</a></li>
</ul></li>
<li class="chapter" data-level="9.7" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><i class="fa fa-check"></i><b>9.7</b> Try it Question 4 - Is there an interaction between diet and age and the microbiome</a>
<ul>
<li class="chapter" data-level="9.7.1" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-1.-plot-alpha-diversity-based-on-age-for-all-dietary-groups-and-then-for-bd-hd-and-wo."><i class="fa fa-check"></i><b>9.7.1</b> Step 1. Plot alpha diversity based on age for all dietary groups, and then for BD, HD and WO.</a></li>
<li class="chapter" data-level="9.7.2" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-2.-perform-differential-abundance-analysis-comparing-hd-to-bd-for-the-population-in-general-and-then-for-older-individuals-specifically-because-older-individuals-produced-the-biggest-dietage-interaction-based-on-alpha-diversity."><i class="fa fa-check"></i><b>9.7.2</b> Step 2. Perform differential abundance analysis comparing HD to BD for the population in general, and then for older individuals specifically (because older individuals produced the biggest diet–age interaction based on alpha diversity).</a></li>
<li class="chapter" data-level="9.7.3" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-2b.-perform-differential-abundance-analysis-between-hd-and-bd-for-older-individual-50-yo."><i class="fa fa-check"></i><b>9.7.3</b> Step 2B. Perform differential abundance analysis between HD and BD for older individual (&gt; 50 yo.)</a></li>
<li class="chapter" data-level="9.7.4" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-3.-select-data-for-poster-and-include-the-code-used-to-generate-the-figure.-1"><i class="fa fa-check"></i><b>9.7.4</b> Step 3. Select data for poster and include the code used to generate the figure.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path=""><a href="#part-community-analysis-and-feedback"><i class="fa fa-check"></i>(PART*) Community analysis and feedback</a>
<ul>
<li class="chapter" data-level="9.8" data-path="lecture---scientific-posters.html"><a href="lecture---scientific-posters.html"><i class="fa fa-check"></i><b>9.8</b> Lecture - Scientific Posters</a></li>
<li class="chapter" data-level="9.9" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html"><i class="fa fa-check"></i><b>9.9</b> Activity - Scientific Posters</a>
<ul>
<li class="chapter" data-level="9.9.1" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html#activity-1"><i class="fa fa-check"></i><b>9.9.1</b> Activity</a></li>
<li class="chapter" data-level="9.9.2" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html#grading-criteria-5"><i class="fa fa-check"></i><b>9.9.2</b> Grading Criteria</a></li>
<li class="chapter" data-level="9.9.3" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html#footnotes-7"><i class="fa fa-check"></i><b>9.9.3</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="9.10" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html"><i class="fa fa-check"></i><b>9.10</b> Activity - Share Your Poster</a>
<ul>
<li class="chapter" data-level="9.10.1" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#introduction-4"><i class="fa fa-check"></i><b>9.10.1</b> Introduction</a></li>
<li class="chapter" data-level="9.10.2" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#activity-2-present-your-poster"><i class="fa fa-check"></i><b>9.10.2</b> Activity 2 – Present Your Poster</a></li>
<li class="chapter" data-level="9.10.3" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#grading-criteria-6"><i class="fa fa-check"></i><b>9.10.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="9.10.4" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#footnotes-8"><i class="fa fa-check"></i><b>9.10.4</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="10" data-path="professional-development.html"><a href="professional-development.html"><i class="fa fa-check"></i><b>10</b> Professional Development</a>
<ul>
<li class="chapter" data-level="10.1" data-path="lecture---next-steps.html"><a href="lecture---next-steps.html"><i class="fa fa-check"></i><b>10.1</b> Lecture - Next Steps</a></li>
<li class="chapter" data-level="10.2" data-path="activity---next-steps.html"><a href="activity---next-steps.html"><i class="fa fa-check"></i><b>10.2</b> Activity - Next Steps</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="activity---next-steps.html"><a href="activity---next-steps.html#introduction-5"><i class="fa fa-check"></i><b>10.2.1</b> Introduction</a></li>
<li class="chapter" data-level="10.2.2" data-path="activity---next-steps.html"><a href="activity---next-steps.html#activity-1-complete-feedback"><i class="fa fa-check"></i><b>10.2.2</b> Activity 1 – Complete Feedback</a></li>
<li class="chapter" data-level="10.2.3" data-path="activity---next-steps.html"><a href="activity---next-steps.html#activity-2-research-opportunities"><i class="fa fa-check"></i><b>10.2.3</b> Activity 2 – Research Opportunities</a></li>
<li class="chapter" data-level="10.2.4" data-path="activity---next-steps.html"><a href="activity---next-steps.html#activity-3-explore-communities"><i class="fa fa-check"></i><b>10.2.4</b> Activity 3 – Explore Communities</a></li>
<li class="chapter" data-level="10.2.5" data-path="activity---next-steps.html"><a href="activity---next-steps.html#optional-activity-4-share-work"><i class="fa fa-check"></i><b>10.2.5</b> (Optional) Activity 4 – Share Work</a></li>
<li class="chapter" data-level="10.2.6" data-path="activity---next-steps.html"><a href="activity---next-steps.html#grading-criteria-7"><i class="fa fa-check"></i><b>10.2.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="10.2.7" data-path="activity---next-steps.html"><a href="activity---next-steps.html#footnotes-9"><i class="fa fa-check"></i><b>10.2.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="science-talks.html"><a href="science-talks.html"><i class="fa fa-check"></i><b>10.3</b> Science Talks</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="science-talks.html"><a href="science-talks.html#science-talks---activity-1"><i class="fa fa-check"></i><b>10.3.1</b> Science Talks - Activity 1</a></li>
<li class="chapter" data-level="10.3.2" data-path="science-talks.html"><a href="science-talks.html#science-talks---activity-2"><i class="fa fa-check"></i><b>10.3.2</b> Science Talks - Activity 2</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="create-your-cv.html"><a href="create-your-cv.html"><i class="fa fa-check"></i><b>10.4</b> Create your CV</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="c-moor-scholars.html"><a href="c-moor-scholars.html"><i class="fa fa-check"></i><b>11</b> C-MOOR Scholars</a>
<ul>
<li class="chapter" data-level="11.1" data-path="c-moor-scholars-1.html"><a href="c-moor-scholars-1.html"><i class="fa fa-check"></i><b>11.1</b> C-MOOR Scholars</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="c-moor-scholars-1.html"><a href="c-moor-scholars-1.html#c-moor-scholars-links"><i class="fa fa-check"></i><b>11.1.1</b> C-MOOR Scholars links</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="pursue-further-research.html"><a href="pursue-further-research.html"><i class="fa fa-check"></i><b>11.2</b> Pursue further research</a>
<ul>
<li class="chapter" data-level="11.2.1" data-path="pursue-further-research.html"><a href="pursue-further-research.html#research-opportunities-at-your-institution"><i class="fa fa-check"></i><b>11.2.1</b> Research opportunities at your institution</a></li>
<li class="chapter" data-level="11.2.2" data-path="pursue-further-research.html"><a href="pursue-further-research.html#find-a-research-lab-to-join"><i class="fa fa-check"></i><b>11.2.2</b> Find a research lab to join</a></li>
</ul></li>
<li class="chapter" data-level="11.3" data-path="biodigs.html"><a href="biodigs.html"><i class="fa fa-check"></i><b>11.3</b> BioDIGS</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="online-community.html"><a href="online-community.html"><i class="fa fa-check"></i><b>12</b> Online Community</a></li>
<li class="part"><span><b>Resources</b></span></li>
<li class="chapter" data-level="13" data-path="s-minicure-guide.html"><a href="s-minicure-guide.html"><i class="fa fa-check"></i><b>13</b> 16S miniCURE Guide</a>
<ul>
<li class="chapter" data-level="13.0.1" data-path="s-minicure-guide.html"><a href="s-minicure-guide.html#footnotes-10"><i class="fa fa-check"></i><b>13.0.1</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">miniCURE: Human Gut 16S rRNA</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=10.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/C-MOOR_Template/dasl_thin_main_image.png">
</div>
<div id="activity---introducing-16s-rrna-data-using-spreadsheet" class="section level2 hasAnchor" number="7.3">
<h2><span class="header-section-number">7.3</span> Activity - Introducing 16S rRNA Data Using Spreadsheet<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity---introducing-16s-rrna-data-using-spreadsheet" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="learning-objectives-4" class="section level3 hasAnchor" number="7.3.1">
<h3><span class="header-section-number">7.3.1</span> Learning objectives<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#learning-objectives-4" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li>Explore Amplicon Sequence Variants (ASVs) through the taxonomy profile of select ASVs, and sample metadata for a subset of ASVs.</li>
<li>Generate initial research questions based on the above exploratory data analysis</li>
</ol>
</div>
<div id="introduction-1" class="section level3 hasAnchor" number="7.3.2">
<h3><span class="header-section-number">7.3.2</span> Introduction<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#introduction-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The most popular sequencing technique for the analysis of bacterial diversity is targeted sequencing, or sequencing of a specific gene (or region of a gene, e.g. a hypervariable region of the bacterial 16S ribosomal rRNA gene) using Polymerase Chain Reaction (PCR) to create sequences called amplicons. Sequence variation in the resulting amplicons creates Amplicon Sequence Variants (ASVs). ASVs varying from as little as one single nucleotide are defined as separate ASVs and as little as 1% difference in ASV sequence can be associated with different species. In this activity you will be exploring ASVs generated with sequencing 250 nucleotides (nt) Illumina sequencing, where Amplicon sequence variants (ASVs) were identified.</p>
</div>
<div id="activity-1-quick-data-overview-based-on-1-sample" class="section level3 hasAnchor" number="7.3.3">
<h3><span class="header-section-number">7.3.3</span> Activity 1 – Quick data overview based on 1 sample<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-1-quick-data-overview-based-on-1-sample" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 10 min</em></p>
<p>Although the full dataset contains 105 samples from 21 subjects and 5 timepoints (representing 3 dietary conditions), in this activity we will familiarize ourselves with the data by exploring data from 1 sample only.</p>
<div id="activity-1-1.-explore-data-contents-using-one-sample-only-namely-miso1-s02-1-subject-2-timepoint-1." class="section level4 hasAnchor" number="7.3.3.1">
<h4><span class="header-section-number">7.3.3.1</span> Activity 1-1. Explore data contents using one sample only, namely MISO1-S02-1 (subject 2, timepoint 1).<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-1-1.-explore-data-contents-using-one-sample-only-namely-miso1-s02-1-subject-2-timepoint-1." class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Access <a href="https://docs.google.com/spreadsheets/d/1hpSyjO0H8sXq6C1xM_oGMyv9vCY2qTvfYHVdqO9ZVhk/edit?usp=sharing">‘MISO_PS_Leah_SY_Sample1’</a> file and open it with Google Sheets.</p>
<ul>
<li>Rows = ASVs</li>
<li>Columns = Samples and taxonomy:
<ul>
<li>Col A = ASV id</li>
<li>Cols B-H = Taxonomic information</li>
<li>Col I = ASV counts (for sample MISO1-SO2-1)</li>
</ul></li>
</ul>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1-1.1 How many ASVs are there? HINT: Scroll down to see how many rows are there!</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><strong>1-1.2 How many microbes are there in an individual? In other words, what is the total number of ASVs per sample?</strong></p>
<p>IMPORTANT: An ASV is associated with an individual if it is NOT a zero. Zero means that particular ASV was not found in the individual, so you need to exclude zeroes from your calculations.</p>
<ul>
<li>Create an empty new first row for temporary calculations by clicking to insert 1 row above.</li>
<li>In the new row, calculate the number of ASVs per sample 1 (excluding the zeroes) using the following formula: =COUNTIF(range, “&lt;&gt;0”). For sample 1 (MISO1-S02-1) enter formula =COUNTIF(I2:I2185, “&lt;&gt;0”).</li>
</ul>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1-1.3 What taxonomy is associated with ASV1? Include all taxonomic ranks.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1-1.4 What taxonomy is associated with ASV2? Include all taxonomic ranks.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">1-1.5 For sample 1 (MISO1-S02-1) what is the ASV1 count?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">1-1.6 For sample 1 (MISO1-S02-1) what is the ASV2 count?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><strong>1-1.7 Plot ASV counts distribution (range of ASV abundance) for sample 1 ( (MISO1-S02-1).</strong></p>
<p><strong>A) Make a bar graph of ASV distribution.</strong> To make a bar graph, hold Command key (or Ctrl on Windows, allows you to select data that are not next to each other) and then highlight columns A, with ASV ids (column “ASV”) and I, with ASV counts (column “MISO1-S02-1”) and go to Insert Chart.</p>
<ul>
<li>Make sure your graph is a bar plot: click on the 3 dots in the top right corner of the plot → edit chart and in the Chart editor, under Chart type, selecting Column Chart.</li>
<li>Copy the bar graph into a box below: click on the 3 dots in the top right corner of the plot, and select copy chart, then, paste it below.</li>
</ul>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">B) What is the highest abundance ASV in sample 1? A simple way to get an answer is to hover over the highest bar in your ASV distribution plot from activity at its highest point.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">C) Describe abundance pattern between ASVs. E.g., is abundance even or spotty and what does that say about ASV/taxa variation?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="activity-2-explore-variation-between-individuals" class="section level3 hasAnchor" number="7.3.4">
<h3><span class="header-section-number">7.3.4</span> Activity 2 – Explore variation between individuals<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-explore-variation-between-individuals" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 10 min</em></p>
<p>One aspect of Guthrie et al. study is the extent to which diet contributes to interpersonal microbiome variation (variation in microbiome between individuals). To this end, authors ‘normalize’ the diet for all individuals with the same Homogenized Diet (HD) to evaluate how much of an effect the diet has. <strong>The study period is 28 days.</strong> The study has 3 dietary conditions: 1) <strong>BD</strong> (Baseline Diet) which lasted 14 days (2 time points collected), followed by 2) <strong>HD</strong> which lasted 7 days (2 time points collected), followed by 3) <strong>WO</strong> (washout diet), which lasted for 7 days and simply marks the return to baseline, regular diet (1 time point).</p>
<p><strong>To explore variation between individuals, here we simplify the dietary variation to a single condition, WO.</strong></p>
<ul>
<li><p>For this activity, the data file <a href="https://docs.google.com/spreadsheets/d/1vD0wbxgYuDlfKA0DF2Km-N4GHL2gRoUaEN_oCrSb2jU/edit?usp=sharing">MISO_PS_Leah_SY_WO-select</a> contains samples pre-filtered for WO condition only (and excludes samples corresponding to BD and HD).</p></li>
<li><p>Since there is only 1 WO datapoint per individual, each WO sample corresponds to a different individual, so you should see 21 samples.</p></li>
</ul>
<div id="activity-2-1---what-are-the-most-common-asvs-between-individuals" class="section level4 hasAnchor" number="7.3.4.1">
<h4><span class="header-section-number">7.3.4.1</span> Activity 2-1 - What are the most common ASVs between individuals?<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-1---what-are-the-most-common-asvs-between-individuals" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>IMPORTANT: An ASV is associated with an individual if it is NOT a zero. Zero means that particular ASV was not found in the individual, so you need to exclude zeroes from your calculations.</p>
<p><strong>A) Calculate the number of samples associated with each ASVs. Record first 5 values below.</strong></p>
<ul>
<li>Create an empty new first column for temporary calculations by inserting new column A (via clicking on column A and selecting to Insert 1 column to the left). Label new column “Occurrence”.</li>
<li>Starting with the first ASV in row 3, in an empty cell A3 use the following formula: =COUNTIF(range, “&lt;&gt;0”). For example, for ASV1 enter formula = COUNTIF(C3:W3, “&lt;&gt;0”). The answer should be 17 (so 17 of 21 WO samples have ASV1).</li>
<li>Extend the function to all ASVs</li>
</ul>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><strong>B) Calculate % of samples associated with each ASV and record the first top 5.</strong></p>
<ul>
<li>Create another new column A and label it “% Occurrence”.</li>
</ul>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><strong>C) How many ASVs are present in all 21 individuals and what are their ids? Simply find which ASVs are present in 100% of samples.</strong></p>
<ul>
<li>Copy columns A (% samples) and C (ASV id) into a new sheet and Paste Special → Values only</li>
<li>Delete empty row 1</li>
<li>Right-click on Row 1 to Create a filter, then Sort Z to A</li>
<li>Count the ASVs that have 100 in % samples</li>
</ul>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
<div id="activity-2-2.-explore-most-common-asv-taxonomy" class="section level4 hasAnchor" number="7.3.4.2">
<h4><span class="header-section-number">7.3.4.2</span> Activity 2-2. Explore most common ASV taxonomy<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-2.-explore-most-common-asv-taxonomy" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li><p>Go the original MISO_PS_Leah_SY_Sample1 google docs and record taxonomic information for most common ASV from the activity above.</p></li>
<li><p>In MISO_PS_Leah_SY_Sample1, Columns B-H correspond to taxonomic information</p></li>
</ul>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2-2.1 What is the taxonomy assigned to the most common ASV you found in Activity above?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2-2.2 According to the taxonomy for most common ASV above, what might be the most abundant phyla in the human gut?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="activity-2-3.-plot-individual-variation-in-abundance-of-the-most-common-asv" class="section level3 hasAnchor" number="7.3.5">
<h3><span class="header-section-number">7.3.5</span> Activity 2-3. Plot individual variation in abundance of the most common ASV<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-3.-plot-individual-variation-in-abundance-of-the-most-common-asv" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>A) Make a bar graph of most common ASV distribution across 21 individuals</strong>
1. In the original MISO_PS_Leah_SY_WO-select find the row corresponding to the most common ASV, from Activity 2-1 above.
2. Highlight the header row (corresponding to sample IDs) and the row corresponding to most common ASV and go to Insert Chart. Make sure you don’t include the columns with the # of samples with that ASV and % samples columns.</p>
<ul>
<li><p>Make sure your bar graph is a bar plot: click on the 3 dots in the top right corner of the plot → edit chart and in the Chart editor, under Chart type, selecting Column Chart.</p></li>
<li><p>Copy the bar graph into a box below: click on the 3 dots in the top right corner of the plot, and select copy chart, then, paste it below.</p></li>
</ul>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">B) Comment on the chart above - how similar is the ASV abundance between individuals?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
<div id="activity-3-explore-variation-within-a-single-individual-associated-with-diet" class="section level3 hasAnchor" number="7.3.6">
<h3><span class="header-section-number">7.3.6</span> Activity 3 – Explore variation within a single individual associated with diet<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-3-explore-variation-within-a-single-individual-associated-with-diet" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 10 min</em></p>
<p>Although the full dataset contains 105 samples from 21 subjects and 5 timepoints (representing 3 dietary conditions), in this activity we will zoom into ASV counts data for 1 individual (comprised of 5 samples representing the 5 timepoints collected from that individual).</p>
<ul>
<li>Timepoints 1 and 2: BD (Baseline diet) prior to normalization with homogenized diet</li>
<li>Timepoints 3 and 4: HD (Homogenized diet): normalized to be same for all</li>
<li>Timepoint 5: WO (Washout diet), which simply involves individuals going back to their baseline, regular diet</li>
</ul>
<div id="activity-3-1-access-miso_ps_leah_sy_subject1-file-and-open-it-with-google-sheets-then-explore-differences-between-dietary-changes-in-individual-1-by-exploring-differences-between-the-5-sample-timepoints." class="section level4 hasAnchor" number="7.3.6.1">
<h4><span class="header-section-number">7.3.6.1</span> Activity 3-1 Access ‘MISO_PS_Leah_SY_Subject1’ file and open it with Google Sheets, then explore differences between dietary changes in individual 1 by exploring differences between the 5 sample timepoints.<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-3-1-access-miso_ps_leah_sy_subject1-file-and-open-it-with-google-sheets-then-explore-differences-between-dietary-changes-in-individual-1-by-exploring-differences-between-the-5-sample-timepoints." class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><strong>3-1.1 Calculate variation in ASV abundance within individual 1 (MISO1-S02) and record the first 5 values below.</strong>
This involves plotting counts for ASV 1-10 for MISO1-SO2 timepoints 1 through 5.
To calculate variation within a sample, we will use the VAR.S function in google sheets to calculate variance of a sample.</p>
<ol style="list-style-type: decimal">
<li>Label an empty Column G “VAR”. Use this column to calculate variance.</li>
<li>Starting with the first ASV use the following formula: =VAR.S(range) to calculate ASV abundance variation in the 5 samples/timepoints collected for individual MISO1-SO2. For example, for ASV1 enter formula =VAR.S(B2:F2). The answer should be 14947547.8.</li>
<li>Extend the function to all ASVs and record the first 5 variance values below.</li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><strong>3-1.2 Plot ASVs with highest variance.</strong></p>
<p><strong>A) Sort Variance column and record the top 5 values below.</strong> To sort the variance column:</p>
<ol style="list-style-type: decimal">
<li>Click on the 1st (header) row</li>
<li>Right-click to select ‘Create a filter’</li>
<li>Sort ‘VAR’ column Z to A (click on the new arrow in the VAR column).</li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><strong>B) Plot ASVs with highest variance.</strong>
After sorting ASVs based on high to low variance make a plot of ASV counts corresponding to top 15 ASVs with the highest variance. Insert the plot in the grey box below. Here, variance within a sample might suggest variation (difference) between conditions. For example, high variance may come from the first 2 samples (BD diet) being different from samples 3 and 4 (HD diet).</p>
<ul>
<li>To make a bar graph highlight top 16 rows corresponding to sample names and the 15 ASVs with highest variance, for samples MISO1-S02-1 through 5. Exclude highlighting the variance column itself.</li>
<li>Go to Insert Chart. For this chart you can highlight the rows starting with title row so as to include sample names in your chart.</li>
<li>Make sure your graph is a bar plot: click on the 3 dots in the top right corner of the plot → edit chart and in the Chart editor, under Chart type, selecting Column Chart.</li>
<li>Copy the bar graph into a box below: click on the 3 dots in the top right corner of the plot, and select copy chart, then, paste it below.</li>
</ul>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
<div id="activity-3-2.-examine-the-top-15-most-variable-asvs-and-in-the-box-below-commentspeculate-on-source-of-variation-for-one-of-the-asvs.-do-you-think-the-source-of-variation-is-change-of-diet-inconsistent-replicates-or-something-else" class="section level4 hasAnchor" number="7.3.6.2">
<h4><span class="header-section-number">7.3.6.2</span> Activity 3-2. Examine the top 15 most variable ASVs and in the box below comment/speculate on source of variation for one of the ASVs. Do you think the source of variation is change of diet, inconsistent replicates or something else?<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-3-2.-examine-the-top-15-most-variable-asvs-and-in-the-box-below-commentspeculate-on-source-of-variation-for-one-of-the-asvs.-do-you-think-the-source-of-variation-is-change-of-diet-inconsistent-replicates-or-something-else" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>E.g., Variance associated with different diets would produce a plot where ASV counts for samples 1 and 2 (BD diet) might be different from samples 3 and 4 (HD diet) or sample 5 (WO diet).<br />
</li>
<li>E.g., Variance observed between samples 1 and 2 (both BD diets) might suggest difference between biological replicate 1 and 2 due to unknown factors.</li>
</ul>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="activity-4-try-it-out" class="section level3 hasAnchor" number="7.3.7">
<h3><span class="header-section-number">7.3.7</span> Activity 4 – Try it Out!<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-4-try-it-out" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 60 min</em></p>
<p>Now it’s time to explore the full dataset with all 105 samples! Working with your group, explore one of the questions below or come up with another question. When you are done, copy and paste your results into the class poster. Here are the files that you will need for this Activity:</p>
<ul>
<li><a href="https://docs.google.com/spreadsheets/d/1mDk_enQzQVQVplbLzPMYJBF5whsvZ62gOPDwkqIpJvM/edit?usp=sharing">ASV counts (2185 rows by 114 columns)</a></li>
<li><a href="https://docs.google.com/spreadsheets/d/1TFXtaantSgDnMwnUvcAllQErWcOXOUwNCwRb_vyk2f0/edit?usp=sharing">Sample metadata (106 rows by 28 columns)</a></li>
<li><a href="---">Class Poster</a></li>
</ul>
<p>Remember, to relate sample IDs from MISO_PS_Leah_SY_FULL, to the metadata in the MISO_metadata_FULL file.</p>
<p>For example, the MISO study timepoints are:</p>
<ul>
<li>Timepoints 1 and 2: <strong>BD</strong> (Baseline diet) prior to normalization with homogenized diet</li>
<li>Timepoints 3 and 4: <strong>HD</strong> (Homogenized diet): normalized to be same for all</li>
<li>Timepoint 5: <strong>WO</strong> (Washout diet), which simply involves individuals going back to their baseline, regular diet</li>
</ul>
<div id="question-1---profile-and-compare-asv-abundance-and-taxonomy-for-2-diets-bd-and-hd.-your-final-analysis-should-result-in-a-plot-andor-a-table." class="section level4 hasAnchor" number="7.3.7.1">
<h4><span class="header-section-number">7.3.7.1</span> Question 1 - Profile and compare ASV abundance and taxonomy for 2 diets, BD and HD. Your final analysis should result in a plot and/or a table.<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#question-1---profile-and-compare-asv-abundance-and-taxonomy-for-2-diets-bd-and-hd.-your-final-analysis-should-result-in-a-plot-andor-a-table." class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>Question 1A: Compare 1 BD timepoint and 1 HD timepoint (males): n = 20</li>
<li>Question 1B: Compare 1 BD timepoint and 1 HD timepoint (females) : n = 22</li>
</ul>
<p>Sample approach: Compare the 5 most abundant ASVs between BD1 and HD1. You’ll have to think about:</p>
<ol style="list-style-type: decimal">
<li>How will you identify the 5 most abundant ASVs? Will you identify the top 5 in the BD group, and top 5 in the HD group? Should you sort on 1, 2, or all samples within a group? Or just average all samples within a group?)</li>
<li>How will you compare the multiple BD1 samples against the multiple HD1 samples?</li>
</ol>
<ul>
<li>ASV counts - plot average or boxplot of abundance for top ASVs?</li>
<li>For taxonomy - is there an overlap between 5 BD and 5 HD taxa? Should you consider a pie chart, a barplot or no plot?)</li>
</ul>
</div>
<div id="question-2---profile-and-compare-asv-abundance-and-taxonomy-of-21-individuals-using-a-single-homogenized-dietary-timepoint.-your-final-analysis-should-result-in-a-plot-andor-a-table." class="section level4 hasAnchor" number="7.3.7.2">
<h4><span class="header-section-number">7.3.7.2</span> Question 2 - Profile and compare ASV abundance and taxonomy of 21 individuals using a single homogenized dietary timepoint. Your final analysis should result in a plot and/or a table.<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#question-2---profile-and-compare-asv-abundance-and-taxonomy-of-21-individuals-using-a-single-homogenized-dietary-timepoint.-your-final-analysis-should-result-in-a-plot-andor-a-table." class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>Question 2A: Examine HD1 timepoint (males and females): n = 21</li>
<li>Question 2B: Examine HD2 timepoint (males and females): n = 21</li>
</ul>
<p>Sample approach: You can compare the 5-10 most abundant ASVs between 21 individuals for HD1. You’ll have to think about
1. How will you identify the 5 most abundant ASVs? Should you show the top 5 per each sample? Sort on 1, 2, or all samples? Or just average all samples?)
1. How will you compare the 21 samples?
- ASV counts - plot average or boxplot of abundance for top ASVs?
- For taxonomy - Barplot top taxa for all individuals? Which rank - Phyla, Genus, Species?)</p>
</div>
</div>
<div id="grading-criteria-2" class="section level3 hasAnchor" number="7.3.8">
<h3><span class="header-section-number">7.3.8</span> Grading criteria<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#grading-criteria-2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Download the assignment to your local computer as a .docx, complete it, and upload the assignment to your LMS (Blackboard, Canvas, Google Classroom).</li>
<li>Download the Google Sheets to your local computers as a .xlsx, complete all the steps for the assignment, and upload the .xlsx to your LMS (Blackboard, Canvas, Google Classroom).</li>
</ul>
</div>
<div id="footnotes-4" class="section level3 hasAnchor" number="7.3.9">
<h3><span class="header-section-number">7.3.9</span> Footnotes<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#footnotes-4" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="resources-3" class="section level4 hasAnchor" number="7.3.9.1">
<h4><span class="header-section-number">7.3.9.1</span> Resources<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#resources-3" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li><a href="https://docs.google.com/document/d/12pKSIwcVXisaf6dniyoG_AGdCQh1dG1dDwZ5s1cz92c/edit?usp=sharing">Google Doc</a></li>
<li><a href="https://docs.google.com/spreadsheets/d/1hpSyjO0H8sXq6C1xM_oGMyv9vCY2qTvfYHVdqO9ZVhk/edit?usp=sharing">‘MISO_PS_Leah_SY_Sample1’</a></li>
<li><a href="https://docs.google.com/spreadsheets/d/1vD0wbxgYuDlfKA0DF2Km-N4GHL2gRoUaEN_oCrSb2jU/edit?usp=sharing">MISO_PS_Leah_SY_WO-select</a></li>
<li><a href="https://docs.google.com/spreadsheets/d/1mDk_enQzQVQVplbLzPMYJBF5whsvZ62gOPDwkqIpJvM/edit?usp=sharing">ASV counts (2185 rows by 114 columns)</a></li>
<li><a href="https://docs.google.com/spreadsheets/d/1TFXtaantSgDnMwnUvcAllQErWcOXOUwNCwRb_vyk2f0/edit?usp=sharing">Sample metadata (106 rows by 28 columns)</a></li>
<li><a href="---">Class Poster</a></li>
</ul>
</div>
<div id="contributions-and-affiliations-3" class="section level4 hasAnchor" number="7.3.9.2">
<h4><span class="header-section-number">7.3.9.2</span> Contributions and affiliations<a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#contributions-and-affiliations-3" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>Valeriya Gaysinskaya, Johns Hopkins University</li>
<li>Gauri Paul, Clovis Community College</li>
<li>Frederick Tan, Johns Hopkins University</li>
<li>Sayumi York, Notre Dame of Maryland University</li>
</ul>

<!-- Set up code of OTTR Book-->
</div>
</div>
</div>
<!-- </div> -->
<hr>
<center> 
  <div class="footer">
      All materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="lecture---introducing-16s-rrna-data-using-spreadsheets.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="exploring-16s-rrna-data-with-phyloseq.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "section"
  }
});
});
</script>

</body>

</html>
