<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6.3 Activity - Exploring 16S rRNA Data with phyloseq | Course Name</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="6.3 Activity - Exploring 16S rRNA Data with phyloseq | Course Name" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6.3 Activity - Exploring 16S rRNA Data with phyloseq | Course Name" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  



<meta name="date" content="2025-07-04" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/dasl_favicon.ico" type="image/x-icon" />
<link rel="prev" href="lecture---analyzing-16s-rrna-data-with-phyloseq.html"/>
<link rel="next" href="about-the-authors.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="assets/style.css" type="text/css" />
<link rel="stylesheet" href="assets/C-MOOR_Template/c-moor-ottr.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.c-moor.org/" target="_blank"><img src="assets/C-MOOR_Template/c-moor-logo-vertical.png" style="width: 70%; padding-left: 10px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#learning-goals"><i class="fa fa-check"></i>Learning Goals</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="motivation.html"><a href="motivation.html"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="target-audience.html"><a href="target-audience.html"><i class="fa fa-check"></i><b>1.2</b> Target Audience</a></li>
<li class="chapter" data-level="1.3" data-path="curriculum.html"><a href="curriculum.html"><i class="fa fa-check"></i><b>1.3</b> Curriculum</a></li>
</ul></li>
<li class="part"><span><b>Scientific Literature</b></span></li>
<li class="chapter" data-level="2" data-path="scientific-literature.html"><a href="scientific-literature.html"><i class="fa fa-check"></i><b>2</b> Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html"><i class="fa fa-check"></i><b>2.1</b> Pre-lab - Scientific literature</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#purpose"><i class="fa fa-check"></i><b>2.1.1</b> Purpose</a></li>
<li class="chapter" data-level="2.1.2" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#grading-criteria"><i class="fa fa-check"></i><b>2.1.2</b> Grading criteria</a></li>
<li class="chapter" data-level="2.1.3" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#activity"><i class="fa fa-check"></i><b>2.1.3</b> Activity</a></li>
<li class="chapter" data-level="2.1.4" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#footnotes"><i class="fa fa-check"></i><b>2.1.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="lecture---reading-a-paper.html"><a href="lecture---reading-a-paper.html"><i class="fa fa-check"></i><b>2.2</b> Lecture - Reading a paper</a></li>
<li class="chapter" data-level="2.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html"><i class="fa fa-check"></i><b>2.3</b> Activity - Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#purpose-1"><i class="fa fa-check"></i><b>2.3.1</b> Purpose</a></li>
<li class="chapter" data-level="2.3.2" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#learning-objectives"><i class="fa fa-check"></i><b>2.3.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.3.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#introduction-1"><i class="fa fa-check"></i><b>2.3.3</b> Introduction</a></li>
<li class="chapter" data-level="2.3.4" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#instructions-1"><i class="fa fa-check"></i><b>2.3.4</b> Instructions</a></li>
<li class="chapter" data-level="2.3.5" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-1---overview-of-the-paper"><i class="fa fa-check"></i><b>2.3.5</b> Activity 1 - Overview of the paper</a></li>
<li class="chapter" data-level="2.3.6" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-2---methods-and-figure-analysis-in-class-and-homework"><i class="fa fa-check"></i><b>2.3.6</b> Activity 2 - Methods and Figure Analysis (in class and homework)</a></li>
<li class="chapter" data-level="2.3.7" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#grading-criteria-1"><i class="fa fa-check"></i><b>2.3.7</b> Grading criteria</a></li>
<li class="chapter" data-level="2.3.8" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#footnotes-1"><i class="fa fa-check"></i><b>2.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Project: 16S rRNA Research</b></span></li>
<li class="chapter" data-level="3" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html"><i class="fa fa-check"></i><b>3</b> Meet the MISO study</a>
<ul>
<li class="chapter" data-level="3.0.1" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#miso-study-diet-and-sampling-schedule"><i class="fa fa-check"></i><b>3.0.1</b> MISO study diet and sampling schedule</a></li>
<li class="chapter" data-level="3.0.2" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#miso-study-variables-and-factors"><i class="fa fa-check"></i><b>3.0.2</b> MISO study variables and factors</a></li>
<li class="chapter" data-level="3.0.3" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#amplicon-sequence-variants-asvs-data"><i class="fa fa-check"></i><b>3.0.3</b> Amplicon sequence variants (ASVs) data</a></li>
<li class="chapter" data-level="3.0.4" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#footnotes-2"><i class="fa fa-check"></i><b>3.0.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="s-rrna-data-in-spreadsheets.html"><a href="s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>4</b> 16S rRNA Data in spreadsheets</a>
<ul>
<li class="chapter" data-level="4.1" data-path="exploring-16s-rrna-data-in-spreadsheets.html"><a href="exploring-16s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>4.1</b> Exploring 16S rRNA data in spreadsheets</a></li>
<li class="chapter" data-level="4.2" data-path="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><a href="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><i class="fa fa-check"></i><b>4.2</b> Lecture - Introducing 16S rRNA Data Using Spreadsheets</a></li>
<li class="chapter" data-level="4.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html"><i class="fa fa-check"></i><b>4.3</b> Activity - Introducing 16S rRNA Data Using Spreadsheet</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#learning-objectives-1"><i class="fa fa-check"></i><b>4.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="4.3.2" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#introduction-2"><i class="fa fa-check"></i><b>4.3.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-1-quick-data-overview-based-on-1-sample"><i class="fa fa-check"></i><b>4.3.3</b> Activity 1 – Quick data overview based on 1 sample</a></li>
<li class="chapter" data-level="4.3.4" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-explore-variation-between-individuals"><i class="fa fa-check"></i><b>4.3.4</b> Activity 2 – Explore variation between individuals</a></li>
<li class="chapter" data-level="4.3.5" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-3-explore-variation-within-a-single-individual-associated-with-diet"><i class="fa fa-check"></i><b>4.3.5</b> Activity 3 – Explore variation within a single individual associated with diet</a></li>
<li class="chapter" data-level="4.3.6" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-4-try-it-out"><i class="fa fa-check"></i><b>4.3.6</b> Activity 4 – Try it Out!</a></li>
<li class="chapter" data-level="4.3.7" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#grading-criteria-2"><i class="fa fa-check"></i><b>4.3.7</b> Grading criteria</a></li>
<li class="chapter" data-level="4.3.8" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#footnotes-3"><i class="fa fa-check"></i><b>4.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="exploring-16s-rrna-data-with-phyloseq.html"><a href="exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5</b> Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="5.1" data-path="exploring-16s-rrna-data-with-phyloseq-1.html"><a href="exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>5.1</b> Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="5.2" data-path="lecture---exploring-16s-rrna-data-with-phyloseq.html"><a href="lecture---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5.2</b> Lecture - Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="5.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#learning-objectives-2"><i class="fa fa-check"></i><b>5.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="5.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#introduction-3"><i class="fa fa-check"></i><b>5.3.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-1-explore-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>5.3.3</b> Activity 1 – Explore 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="5.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-2-try-it-out"><i class="fa fa-check"></i><b>5.3.4</b> Activity 2 – Try it out!</a></li>
<li class="chapter" data-level="5.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#grading-criteria-3"><i class="fa fa-check"></i><b>5.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="5.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#footnotes-4"><i class="fa fa-check"></i><b>5.3.6</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="analyzing-16s-rrna-data-with-phyloseq.html"><a href="analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>6</b> Analyzing 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="6.1" data-path="analyzing-16s-rrna-data-with-phyloseq-1.html"><a href="analyzing-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>6.1</b> Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="6.2" data-path="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><a href="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>6.2</b> Lecture - Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="6.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>6.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#learning-objectives-3"><i class="fa fa-check"></i><b>6.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="6.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#introduction-4"><i class="fa fa-check"></i><b>6.3.2</b> Introduction</a></li>
<li class="chapter" data-level="6.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-1-analyze-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>6.3.3</b> Activity 1 – Analyze 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="6.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-2-try-it-out-1"><i class="fa fa-check"></i><b>6.3.4</b> Activity 2 – Try it out!</a></li>
<li class="chapter" data-level="6.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#grading-criteria-4"><i class="fa fa-check"></i><b>6.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="6.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#footnotes-5"><i class="fa fa-check"></i><b>6.3.6</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Course Name</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=10.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/C-MOOR_Template/dasl_thin_main_image.png">
</div>
<div id="activity---exploring-16s-rrna-data-with-phyloseq-1" class="section level2 hasAnchor" number="6.3">
<h2><span class="header-section-number">6.3</span> Activity - Exploring 16S rRNA Data with phyloseq<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity---exploring-16s-rrna-data-with-phyloseq-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>To analyze bacterial diversity based on 16S rRNA gene sequencing using data from the “Impact of a 7-day homogeneous diet on interpersonal variation in human gut microbiomes and metabolomes” by Guthrie et al., 2023. This study is also referred to as the MISO study for “Microbiome Individuality and Stability Over Time” because the study aims to understand variation (or stability) of microbiomes across individuals.</p>
<p>This activity aims to analyze metagenomic diversity using the following R packages:</p>
<ul>
<li><strong>phyloseq</strong>: A popular package for taxonomy profiling</li>
<li><strong>DESeq2</strong>: A package originally designed for differential expression which we will use for differential abundance</li>
<li><strong>ggplot2</strong>: A tidyverse package that produces high-quality figures</li>
</ul>
<div id="learning-objectives-3" class="section level3 hasAnchor" number="6.3.1">
<h3><span class="header-section-number">6.3.1</span> Learning objectives<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#learning-objectives-3" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li>Use phyloseq to cluster samples based on the similarity of their microbial compositions using multidimensional scaling methods (NMDS or PCoA)</li>
<li>Use phyloseq to profile alpha diversity (Shannon and Simpson’s indices)</li>
<li>Use DESeq2 to perform differential abundance analysis</li>
</ol>
</div>
<div id="introduction-4" class="section level3 hasAnchor" number="6.3.2">
<h3><span class="header-section-number">6.3.2</span> Introduction<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#introduction-4" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>16S data can be manipulated and visualized in a variety of ways. In Google Sheets, we explored the data manually to gain an understanding at the level of 1-10 ASVs. Through phyloseq we collapsed ASVs with the same taxonomic annotation to survey microbial diversity and perform the same exploration across the entire dataset. In this activity, we will use the functions of phyloseq and DESeq2 that are more advanced. These analyses would be extremely difficult and tedious to do manually and will allow us deeper insights into our dataset. We will cluster samples based on similarity of ASV counts, survey alpha diversity, and search for differentially abundant ASVs between different groups.</p>
</div>
<div id="activity-1-analyze-16s-rrna-data-with-phyloseq-tutorial" class="section level3 hasAnchor" number="6.3.3">
<h3><span class="header-section-number">6.3.3</span> Activity 1 – Analyze 16S rRNA Data with phyloseq tutorial<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-1-analyze-16s-rrna-data-with-phyloseq-tutorial" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 35 min</em></p>
<p><strong>Activity 1. Explore a phyloseq object through the “Analyze 16S rRNA Data with phyloseq” tutorial on SciServer.</strong></p>
<ol style="list-style-type: decimal">
<li>Log into SciServer, click on compute, and create a new C-MOOR LearnR container. When creating a container, remember to:</li>
</ol>
<ul>
<li>Use the “C-MOOR LearnR (Bioconductor 3.17)” image (not the Bioconductor 3.16)</li>
<li>Check the box next to Data volume “C-MOOR Data”</li>
</ul>
<ol style="list-style-type: decimal">
<li>Start the “Analyze 16S rRNA Data with phyloseq” tutorial. Visit SciServer Guides and FAQs. If you need assistance accessing the tutorial.</li>
<li>To move through the activities click “Continue” at the bottom of the screen. When you are done with a topic, click “Next Topic” to move on.</li>
<li>This tutorial has small boxes in which you can enter and run short lines of code to analyze the data.</li>
<li>As you work through the tutorial, take snapshots of your work and paste your answers in the grey boxes below:</li>
</ol>
<p><strong>1-1. Take a snapshot and paste the code for an alpha diversity plot (Simpson) from the quiz question:</strong> What male subject has the data point for the LOWEST alpha diversity? HINT: Use subset_samples() to subset males, and specify individuals (subject) on x-axis of the alpha diversity plot
||
|:–|
| <br> |
<br></p>
<p><strong>1-2. Take a snapshot and paste the code for a PCoA plot from the quiz question:</strong>
In a PCoA with only data from ASVs with the class Bacteroidia, what is the percent of variance in the dataset explained by principal coordinate 1? HINT: You will need to change the code subsetting phylum and Firmicutes
- Subsets the Bacteroidetes phylum
- Color is by subject
- Shape is by timepointgroup
||
|:–|
| <br> |
<br></p>
<p><strong>1-3. Take a snapshot and paste the code for the differential abundance plot for the quiz question:</strong>
Which of the following Phylum have ASVs that are differentially abundant between the subject S02 and subject S03?
||
|:–|
| <br> |
<br></p>
</div>
<div id="activity-2-try-it-out-1" class="section level3 hasAnchor" number="6.3.4">
<h3><span class="header-section-number">6.3.4</span> Activity 2 – Try it out!<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-2-try-it-out-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 60 min</em></p>
<p>With your group, perform some exploratory data analysis selecting from one of the four questions below or coming up with your own question. When you are done, copy and paste your results into the <a href="https://docs.google.com/presentation/d/1eE5AltbkW6km0r3uQO8KKnUVRLCLx70SDNuBHp6aASU/edit?slide=id.g343cb6dd56c_2_18#slide=id.g343cb6dd56c_2_18">class poster</a></p>
<div id="question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender" class="section level4 hasAnchor" number="6.3.4.1">
<h4><span class="header-section-number">6.3.4.1</span> Question 1 - How sensitive is microbial diversity to variables like diet, age and gender?<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Approach: Plot Simpson alpha diversity using plot_richness() command in phyloseq and assess the impact of different study variables on changes in microbial diversity. Identify the variables that impact alpha diversity. Visible shifts in alpha diversity measure suggests a shift in microbial diversity, and higher alpha diversity value indicates an increase in alpha diversity.</p>
<p><strong>Note, when plotting this data you may get a warning below. That is ok!</strong></p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb22-1" tabindex="-1"></a>Warning<span class="sc">:</span> The data you have provided does not have any singletons. This is highly suspicious. Results of richness <span class="fu">estimates</span> (<span class="cf">for</span> example) are probably unreliable, or wrong, <span class="cf">if</span> you have already trimmed low<span class="sc">-</span>abundance taxa from the data.We recommended that you find the un<span class="sc">-</span>trimmed data and retry.</span></code></pre></div>
<p>Refer to the “alpha diversity” section of the “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_richness() function.</p>
<p><strong>Step 1A. Plot alpha diversity of the full MISO dataset by subject (individuals).</strong></p>
<ul>
<li>subject is on the x-axis</li>
<li>Color is by subject</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb23-1" tabindex="-1"></a><span class="fu">plot_richness</span>(miso_counts, <span class="at">x=</span><span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb23-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb23-2" tabindex="-1"></a>              <span class="at">color=</span><span class="st">&quot;subject&quot;</span>, </span>
<span id="cb23-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb23-3" tabindex="-1"></a>              <span class="at">measures=</span> <span class="st">&quot;Simpson&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">1-A. Paste the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>1A-2. Based on the alpha plot above, is individuality a strong determinant of microbial alpha diversity (richness and evenness) within an individual? Explain.</strong>
How close are the 5 points for an individual? Are individual data points closer to each other or to the 5 points of another individual? Give examples of individuals in your explanation (e.g. compare S26 and S30, or S02 and S09).
| |
|:–|
| <br> |
<br></p>
<p>Some individual alpha diversity profiles appear to have “outliers”, such that e.g. one of the sample points appears separate from the rest. E.g., individuals S03, S10 and S31 seem to have 1 outlier sample. <strong>Speculate in general on what such singleton “outliers” could mean including recalling what the 5 points represent, evoking concepts of replicates, thinking about sampling and real biology.</strong>
| |
|:–|
| <br> |
<br></p>
<p><strong>Step 1B. Plot alpha diversity of the full MISO dataset, by timepointgroup.</strong>
- timepoint group is on the x-axis
- Color is by subject
- Use the following code as a template:</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb24-1" tabindex="-1"></a><span class="fu">plot_richness</span>(miso_counts, <span class="at">x=</span><span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb24-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb24-2" tabindex="-1"></a>              <span class="at">color=</span><span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb24-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb24-3" tabindex="-1"></a>              <span class="at">measures=</span> <span class="st">&quot;Simpson&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">1B-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1B-2. Based on the alpha plot above, does diet have an impact on microbial diversity (richness and evenness) of individuals? Explain - do you observe a shift up or down in data distribution?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 1C. Plot alpha diversity of the full MISO dataset by age.</strong>
- age is on the x-axis
- Color is by subject
- Use the following code as a template:</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb25-1" tabindex="-1"></a><span class="fu">plot_richness</span>(miso_counts, <span class="at">x=</span><span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb25-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb25-2" tabindex="-1"></a>              <span class="at">color=</span><span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb25-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb25-3" tabindex="-1"></a>              <span class="at">measures=</span> <span class="st">&quot;Simpson&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">1C-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1C-2. Based on the alpha plot above, does age have an impact on microbial diversity of individuals? Explain - do you observe a shift up or down in data during ageing? Do you observe a tighter data distribution in younger or older individuals? Which group, younger (&lt; 50 yo) or older (&gt;50 yo) has higher and tiger overall distribution?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 1D. Plot alpha diversity of the full MISO dataset by gender.</strong></p>
<ul>
<li>gender is on the x-axis</li>
<li>Color is by subject</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb26-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb26-1" tabindex="-1"></a><span class="fu">plot_richness</span>(miso_counts, <span class="at">x=</span><span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb26-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb26-2" tabindex="-1"></a>              <span class="at">color=</span><span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb26-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb26-3" tabindex="-1"></a>              <span class="at">measures=</span> <span class="st">&quot;Simpson&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">1D-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1D-2. Based on the alpha plot above, does gender have an impact on microbial diversity? Explain. Which group has a tighter distribution? Which group overall has a more diverse microbiome, males or females? Which microbiomes are more similar to each other, male or female microbiomes?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2. Plot alpha diversity of the full MISO dataset, grouping based on levels for 5 different metabolites: Creatinine, PCS, IS, HIPP, PAG.</strong> Plot alpha diversity for the 5 metabolites independently then choose your favorite metabolite (e.g. one with most difference) and show plot below.</p>
<ul>
<li>metabolite on the x-axis (one at a time: Creatinine, PCS, IS, HIPP, PAG)</li>
<li>Color is by subject</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb27-1" tabindex="-1"></a><span class="fu">plot_richness</span>(miso_counts, <span class="at">x=</span><span class="st">&quot;fill in the blank metabolite name&quot;</span>, </span>
<span id="cb27-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb27-2" tabindex="-1"></a>              <span class="at">color=</span><span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb27-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb27-3" tabindex="-1"></a>              <span class="at">measures=</span> <span class="st">&quot;Simpson&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2A-1. Paste your metabolite plot(s) below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2A-2. Which metabolite did you choose to show and why? Any difference between microbiomes for individuals with low versus high metabolite levels?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 3. Select data for poster and include the code used to generate the figure.</strong>
1. Prepare your final figure and associated code for the poster.
1. Figure suggestion: A 2-part figure with Figure A showing alpha diversity based on your favorite variable (subject, timepointgroup, age, or gender), and Figure B, showing alpha diversity with your favorite metabolite (Creatinine, PCS, IS, HIPP, or PAG).
- Ensure you have a figure legend that explains your final figure.
- Ensure you submit the code you used to generate your final figure.</p>
<table>
<thead>
<tr class="header">
<th align="left">3A-1. Paste final poster figure and code below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals" class="section level4 hasAnchor" number="6.3.4.2">
<h4><span class="header-section-number">6.3.4.2</span> - Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals?<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Approach: Perform multidimensional scaling (also known as principal component analysis) to establish a relationship between the samples given multivariate data (metadata variables). Using a PcOA plot (via commands ordinate() and plot_ordination() in phyloseq), you will condense the original high-dimensional data into a low-dimensional one by converting data to distance map (matrix) with 2 dimensions, x and y, that best explain variability in your data. From your PCoA plot you will assess the contribution of different study variables to sample diversity and identify variables that help explain sample diversity. In a PcOA plot, samples with similar microbial profiles will be plotted close and may appear as “clusters”.</p>
<p><strong>Step 1. Make a PcOA plot, ordinating on the entire miso dataset, and coloring by individual. Investigate resulting plot shape.</strong>
- Ordination is a term used to summarize a multidimensional dataset when projected onto a low-dimensional space (like X &amp; Y axes) and then observing any pattern the data may possess with a visual inspection.
- Subsequent coloring of the pattern with metadata variables can reveal underlying relationships between data and experiment variables.</p>
<ul>
<li>color - by subject</li>
<li>Include a title for your PCoA plot</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb28-1" tabindex="-1"></a><span class="co">#Ordinate </span></span>
<span id="cb28-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb28-2" tabindex="-1"></a>miso.pcoa <span class="ot">&lt;-</span> <span class="fu">ordinate</span>(miso, <span class="at">method=</span><span class="st">&quot;PCoA&quot;</span>, <span class="at">distance=</span><span class="st">&quot;bray&quot;</span>) </span>
<span id="cb28-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb28-3" tabindex="-1"></a></span>
<span id="cb28-4"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb28-4" tabindex="-1"></a><span class="co">#Plot PCoA</span></span>
<span id="cb28-5"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb28-5" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa, </span>
<span id="cb28-6"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb28-6" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb28-7"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb28-7" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">1A-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1A-2. Describe the shape of your PCoA plot. Does the PCoA organize the into groups? Is there any pattern?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1A-3. Do you observe sample clustering by individual? Provide examples and an explanation.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">1A-4. Offer one question you want to ask about this PCoA plot?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2. Using the PCoA plot from Step 1 above, look for any correlations between the shape of your PCoA plot and diet, gender and age.</strong></p>
<p>Step 2A. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and diet? How well is your data explained by the variation in the diet - look for signs of correlation between PCoA shape and variable “timepointgroup” representing BD, HD and WO diets. Note that since we are using the same plot, we do not need to re-ordinate (no ordinate() function).</p>
<ul>
<li>Color - by timepointgroup</li>
<li>Include a title for your PcOA plot</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb29-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb29-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa, </span>
<span id="cb29-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb29-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb29-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb29-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2A-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2A-2 Did coloring the ‘miso’ PCoA plot by timepointgroup explain any variation in your data? E.g. did each of the 3 data groups/clusters correspond to each of the 3 timepointgroups or not? E.g. does the coloring appear randomly distributed or not?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2B. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and gender? </strong> How well is your data explained by the variation in the gender - look for signs of correlation between PcOA shape and variable “gender”.</p>
<ul>
<li>Color - by gender</li>
<li>Include a title for your PcOA plot</li>
<li>Use the following code as a template.</li>
</ul>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb30-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb30-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa, </span>
<span id="cb30-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb30-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb30-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb30-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2B-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2B-2 Did coloring of the ‘miso’ PCoA plot by gender explain any variation in your data? E.g. does the coloring appear randomly distributed or not?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>Step 2C. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and age? How well is your data explained by the variation in the age - look for signs of correlation between PcOA shape and variable “age”.</p>
<ul>
<li>Color - age</li>
<li>Include title name for your PcOA plot</li>
<li>Use the following code as a template. To enhance the color range of the age variable, additionally included below (in blue) is the command to specify the color gradient with yellow–to-blue gradient.</li>
<li>Try running the code with and without the scale_colour_gradient() command to appreciate the difference.</li>
</ul>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb31-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb31-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa, </span>
<span id="cb31-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb31-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb31-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb31-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb31-4"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb31-4" tabindex="-1"></a><span class="fu">scale_colour_gradient</span>(<span class="at">low =</span> <span class="st">&quot;red&quot;</span>, <span class="at">high =</span> <span class="st">&quot;green&quot;</span>) </span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2C-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2C-2 Did coloring of the ‘miso’ PCoA plot by age explain any variation in your data? E.g. Does the coloring appear randomly distributed or not? Is there any hint of a correlation between age and groups of data on the left or the right?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>Step 3. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and the level of metabolites. How well is your data correlated with the variation in the 5 metabolites - Creatinine, PCS, IS, HIPP, or PAG? Test each metabolite by coloring each metabolite at a time, then, choose your favorite metabolite (e.g. one with most difference) and show the plot below.</p>
<ul>
<li>Color - metabolite one at a time: Creatinine, PCS, IS, HIPP, PAG</li>
<li>Include a title for your PcOA plot</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb32-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa fill <span class="cf">in</span> the blank metabolite name, </span>
<span id="cb32-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb32-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb32-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb32-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb32-4"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb32-4" tabindex="-1"></a><span class="fu">scale_colour_gradient</span>(<span class="at">low =</span> <span class="st">&quot;yellow&quot;</span>, <span class="at">high =</span> <span class="st">&quot;blue&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">3A-1. Paste your metabolite plot(s) below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3A-2. Which metabolite did you choose to show and why? Any difference between microbiomes for individuals with low versus high metabolite levels? Any association between metabolite level and microbiome diversity?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>Step 4. One way of checking if the metabolite levels indeed correlate with your data, is to see if subsetting smaller chunks or specific chunks of the data will still maintain the metabolite-data relationship or break it. Using subset() command, subset out e.g. HD diet specifically, and then BD diet. Then, generate new PcOA plots and see if the correlation with your metabolite of interest still holds.</p>
<p>Step 4A. Subset “HD”, ordinate on “HD”, and make a new PCoA plot.</p>
<ul>
<li>Subset data - by the timepointgroup “HD”</li>
<li>Ordinate - misoHD</li>
<li>Color - by your selected metabolite from part 3</li>
<li>Include a title for your PcOA plot</li>
<li>Use the following code as a template:</li>
</ul>
<ol style="list-style-type: lower-alpha">
<li>Subset only “HD” timepoint from your miso data, creating a new phyloseq object called “misoHD”</li>
</ol>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb33-1" tabindex="-1"></a>misoHD <span class="ot">=</span> <span class="fu">subset_samples</span>(miso, timepointgroup <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>) </span></code></pre></div>
<ol start="2" style="list-style-type: lower-alpha">
<li>Ordinate using only “HD” subset, creating a new ordination matrix called “pcoa.misoHD”</li>
</ol>
<div class="sourceCode" id="cb34"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb34-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb34-1" tabindex="-1"></a>pcoa.misoHD <span class="ot">&lt;-</span> <span class="fu">ordinate</span>(misoHD, <span class="at">method=</span><span class="st">&quot;PCoA&quot;</span>, <span class="at">distance=</span><span class="st">&quot;bray&quot;</span>) </span></code></pre></div>
<ol start="3" style="list-style-type: lower-alpha">
<li>Make a PCoA plot using your “HD” subset.</li>
</ol>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb35-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(misoHD, pcoa.misoHD, </span>
<span id="cb35-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb35-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb35-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb35-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">4A-1. Paste your plot below for HD subset:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">4A-2 Did subsetting by the timpointgroup “HD” maintain your data’s relationship to your chosen metabolite? Did the shape of the plot change or stay the same with “HD” subsetting? Can you conclude if your metabolite of choice correlates with HD diet?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>Step 4B. Repeat the analysis on only the BD timepoint group samples. Subset “BD”, ordinate on “BD”, and make a new PCoA plot.</p>
<ul>
<li>Subset data - by the timepointgroup “BD”</li>
<li>Ordinate - misoBD</li>
<li>Color - by your selected metabolite from part 3</li>
<li>Include a title for your PcOA plot</li>
<li>Use the code above as a template.</li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">4A-1. Paste your plot below for BD subset:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Did subsetting “BD” maintain your data’s relationship to your chosen metabolitelevels? Did the shape of the plot change or stay the same with “BD” subsetting? Explain your results. Can you conclude if your metabolite of choice correlates with sample groups when only BD timepointgroup data is plotted?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>Step 5. Select data for poster and include the code used to generate the figure.
- Prepare your final figure and associated code for the poster.
- Figure suggestion: A 2-part figure with Figure A showing PCoA plot based on subject and Figure B, showing PCoA plot with your favorite metabolite (Creatinine, PCS, IS, HIPP, or PAG).
- Ensure you have a figure legend that explains your final figure.
- Ensure you submit the code you used to generate your final figure.</p>
<table>
<thead>
<tr class="header">
<th align="left">5A-1. Paste final poster figure and code below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact" class="section level4 hasAnchor" number="6.3.4.3">
<h4><span class="header-section-number">6.3.4.3</span> Question 3 - What microbes (ASVs) differ between males and females, and does age have an impact?<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Approach: Perform DESeq analysis between females and males and determine how many differentially abundant microbes are there between the sexes. Then examine if age has a further impact on the abundance of the microbes between the sexes.</p>
<p><strong>Step 3A. Perform differential abundance analysis between genders. </strong></p>
<ul>
<li><p>Design - is based on gender (no double quotes)</p></li>
<li><p>Groups to compare - females, F and males, M (baseline)</p></li>
<li><p>Plot Phylum on the X-axis and color by Class</p></li>
<li><p>Make sure your graph has a title that includes the comparison you are making (ex. WO vs HD)</p></li>
<li><p>Refer to the “Differential abundance” section of the “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_ordination() function.</p></li>
<li><p>Use the code below as a template:</p></li>
</ul>
<p>STEP 1: Convert the phyloseq object to a DESeq2 object and specify experimental design</p>
<div class="sourceCode" id="cb36"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb36-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb36-1" tabindex="-1"></a>DESeq2 <span class="ot">&lt;-</span> <span class="fu">phyloseq_to_deseq2</span>(miso_counts, <span class="at">design =</span> <span class="sc">~</span> fill <span class="cf">in</span> the blank)</span></code></pre></div>
<p>STEP 2: Select the groups to compare, where the latter group is your baseline</p>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb37-1" tabindex="-1"></a>my_comparision <span class="ot">&lt;-</span><span class="fu">c</span>(<span class="st">&quot;gender&quot;</span>, <span class="st">&quot;fill in the blank&quot;</span>, <span class="st">&quot;fill in the blank (baseline)&quot;</span>)</span></code></pre></div>
<p>STEP 3: Run the differential abundance analysis</p>
<div class="sourceCode" id="cb38"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb38-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb38-1" tabindex="-1"></a>Significant_DEseq2_ASVs<span class="ot">&lt;-</span><span class="fu">Differential_Abundance</span>(DESeq2, my_comparision, <span class="fl">0.05</span>)</span></code></pre></div>
<p>STEP 4: Retrieve the list of ASVs with a significant difference in abundance between the chosen groups</p>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb39-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb39-1" tabindex="-1"></a>Significant_DEseq2_ASVs</span></code></pre></div>
<p>STEP 5: Plot the results with your chosen x axis and legend</p>
<div class="sourceCode" id="cb40"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb40-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb40-1" tabindex="-1"></a><span class="fu">ggplot</span>(significant_ASVs, <span class="fu">aes</span>(<span class="at">x =</span> fill <span class="cf">in</span> the blank, <span class="at">y=</span>log2FoldChange, <span class="at">color=</span> fill <span class="cf">in</span> the blank)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">size=</span><span class="dv">4</span>, <span class="at">position=</span><span class="st">&quot;jitter&quot;</span>) <span class="sc">+</span></span>
<span id="cb40-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb40-2" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="sc">-</span><span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">0</span>, <span class="at">vjust=</span><span class="fl">0.5</span>))<span class="sc">+</span></span>
<span id="cb40-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb40-3" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">3A-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">3A-2. How many differentially abundant ASVs were identified?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3A-3. Give an example of a differentially abundant ASVs that was significantly higher in females than males. Include ASV ID and taxa. HINT: If male was your baseline, then an example of an ASV that is higher in females will have a positive log2FC.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3A-4. Give an example of a differentially abundant ASVs that was significantly higher in males than females. Include ASV ID and taxa. HINT: If male was your baseline, then an example of an ASV that is higher in males will have a negative log2FC.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 3B. Determine the impact of younger age on differential abundance in males and females, by subsetting age &lt; 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</strong></p>
<ul>
<li>Subset phyloseq object to only include younger individuals (age &lt; 50)</li>
<li>For DESeq2 analysis, keep the design the same as in part A - based on gender</li>
<li>Groups to compare: females (F) and males (M, baseline)</li>
<li>Plot Phylum on the X-axis and color by Class</li>
<li>Use template code below:</li>
</ul>
<p>STEP 1: subset by age &lt; 50</p>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb41-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb41-1" tabindex="-1"></a>ageA <span class="ot">=</span> <span class="fu">subset_samples</span>(miso_counts, age <span class="sc">&lt;</span><span class="dv">50</span>) </span></code></pre></div>
<p>STEP 2: Convert the phyloseq object to a DESeq2 object and specify experimental design</p>
<div class="sourceCode" id="cb42"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb42-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb42-1" tabindex="-1"></a>DESeq2 <span class="ot">&lt;-</span> <span class="fu">phyloseq_to_deseq2</span>(ageA, <span class="at">design =</span> <span class="sc">~</span> fill <span class="cf">in</span> the blank)</span></code></pre></div>
<p>STEP 3: Select the groups to compare, where log2FoldChange reported will correspond to y/x.</p>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb43-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb43-1" tabindex="-1"></a>my_comparision <span class="ot">&lt;-</span><span class="fu">c</span>(<span class="st">&quot;gender&quot;</span>, <span class="st">&quot;fill in the blank&quot;</span>, <span class="st">&quot;fill in the blank (baseline)&quot;</span>)</span></code></pre></div>
<p>STEP 4: Run the differential abundance analysis</p>
<div class="sourceCode" id="cb44"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb44-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb44-1" tabindex="-1"></a>significant_ASVs<span class="ot">&lt;-</span><span class="fu">Differential_Abundance</span>(DESeq2, my_comparision, <span class="fl">0.05</span>)</span></code></pre></div>
<p>STEP 5: Retrieve the list of ASVs with a significant difference in abundance between the chosen groups</p>
<div class="sourceCode" id="cb45"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb45-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb45-1" tabindex="-1"></a>significant_DEseq2_ASVs</span></code></pre></div>
<p>STEP 6: Plot the results with your chosen x axis and legend</p>
<div class="sourceCode" id="cb46"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb46-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb46-1" tabindex="-1"></a><span class="fu">ggplot</span>(significant_ASVs, <span class="fu">aes</span>(<span class="at">x =</span> fill <span class="cf">in</span> the blank, <span class="at">y=</span>log2FoldChange, <span class="at">color=</span> fill <span class="cf">in</span> the blank)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">size=</span><span class="dv">4</span>, <span class="at">position =</span> <span class="st">&quot;jitter&quot;</span>) <span class="sc">+</span></span>
<span id="cb46-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb46-2" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="sc">-</span><span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">0</span>, <span class="at">vjust=</span><span class="fl">0.5</span>))<span class="sc">+</span></span>
<span id="cb46-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb46-3" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">3B-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">3B-2. How many differentially abundant ASVs were identified?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3B-3. Give an example of a differentially abundant ASVs that was significantly higher in younger females than younger males. Include ASV ID and taxa. HINT: If male was your baseline, then an example of an ASV that is higher in females will have a positive log2FC.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3B-4. Give an example of a differentially abundant ASVs that was significantly higher in younger males than younger females. Include ASV ID and taxa. HINT: If male was your baseline, then an example of an ASV that is higher in males will have a negative log2FC.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3B-5. How does younger age impact microbe abundance between females and males in general?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 3C. Determine the impact of older age on differential abundance in males and females, by subsetting age &gt;= 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</strong></p>
<table>
<thead>
<tr class="header">
<th align="left">3C-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">3C-2. How many differentially abundant ASVs were identified?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3C-3. How does older age impact microbe abundance between females and males in general?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3C-4. Comparing fold change plots between younger and older females and males, what is one difference between Phyla distribution?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 4. Select data for poster and include the code used to generate the figure.</strong></p>
<p>Prepare your final figure and associated code for the poster.
- Figure suggestion: A 3-part figure with Figure A showing fold change differences between males and females in general, Figure B showing gender differences in younger individuals, and Figure C showing gender differences in older individuals.</p>
<ul>
<li>Ensure you have a figure legend that explains your final figure.</li>
<li>Ensure you submit the code you used to generate your final figure.</li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">4A-1. Paste final poster figure and code below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction" class="section level4 hasAnchor" number="6.3.4.4">
<h4><span class="header-section-number">6.3.4.4</span> Question 4 - Is there diet and age interaction and what microbes (ASVs) correlate with changes in diet-age interaction?<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Approach: Use alpha diversity measure and DESeq2 tools to answer this question. Using alpha diversity, determine if there is an interaction between diet and age. Then use DESeq2 to see if any ASVs are associated with changes in die-age interaction. Using Simpson alpha diversity measure (or specifically, Gini-Simpson) evaluate how microbial diversity changes with age and diet in general, or age and BD, HD and WO diet specifically. Remember, shifts in alpha diversity measure suggest a shift in microbial diversity, with high alpha diversity suggesting high microbial diversity. Then, use DESeq2 to evaluate if ageing changes ASVs associated with diet.</p>
<p>Note, when plotting this data you may get a warning below. That is ok!</p>
<div class="sourceCode" id="cb47"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb47-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb47-1" tabindex="-1"></a>Warning<span class="sc">:</span> The data you have provided does not have any singletons. This is highly suspicious. Results of richness <span class="fu">estimates</span> (<span class="cf">for</span> example) are probably unreliable, or wrong, <span class="cf">if</span> you have already trimmed low<span class="sc">-</span>abundance taxa from the data.We recommended that you find the un<span class="sc">-</span>trimmed data and retry.</span></code></pre></div>
<p>Refer to the “alpha diversity” section of the “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_richness() function.</p>
<p><strong>Step 1. Plot alpha diversity based on age for all dietary groups, and then for BD, HD and WO.</strong></p>
<p>1A-1. Plot alpha diversity based on age for all dietary groups.</p>
<ul>
<li>Subset data - None</li>
<li>age is on the x-axis</li>
<li>Color is by subject</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb48"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb48-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb48-1" tabindex="-1"></a><span class="fu">plot_richness</span>(miso_counts, <span class="at">x=</span><span class="st">&quot;age&quot;</span>,</span>
<span id="cb48-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb48-2" tabindex="-1"></a>            <span class="at">color=</span><span class="st">&quot;subject&quot;</span>,</span>
<span id="cb48-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb48-3" tabindex="-1"></a>            <span class="at">title =</span> <span class="st">&quot;miso&quot;</span>,</span>
<span id="cb48-4"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb48-4" tabindex="-1"></a>            <span class="at">measures=</span> <span class="fu">c</span>(<span class="st">&quot;Simpson&quot;</span>))</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1A-2. Based on the alpha plot above, is age a good determinant of microbial alpha diversity? Explain. Is there an age-based trend in alpha diversity?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>1B-1. Plot alpha diversity based on age for BD diet only.</p>
<ul>
<li>Subset data - timepointgroup BD</li>
<li>age is on the x-axis</li>
<li>Color is by subject</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb49"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb49-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb49-1" tabindex="-1"></a><span class="co"># Subset only BD samples</span></span>
<span id="cb49-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb49-2" tabindex="-1"></a>miso_counts_BD <span class="ot">=</span> <span class="fu">subset_samples</span>(miso_counts, timepointgroup <span class="sc">==</span> <span class="st">&quot;BD&quot;</span>)</span>
<span id="cb49-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb49-3" tabindex="-1"></a></span>
<span id="cb49-4"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb49-4" tabindex="-1"></a><span class="fu">plot_richness</span>(miso_counts_BD, <span class="at">x=</span><span class="st">&quot;fill in the blank&quot;</span>,</span>
<span id="cb49-5"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb49-5" tabindex="-1"></a>            <span class="at">color=</span><span class="st">&quot;fill in the blank&quot;</span>,</span>
<span id="cb49-6"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb49-6" tabindex="-1"></a>            <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>,</span>
<span id="cb49-7"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb49-7" tabindex="-1"></a>            <span class="at">measures=</span> <span class="fu">c</span>(<span class="st">&quot;Simpson&quot;</span>))</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1B-2. Based on the alpha plot above, how did subsetting for BD alter the age-based alpha diversity plot? Did the trend in age-based alpha diversity change? Explain if this makes sense.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>1C-1. Plot alpha diversity based on age for HD diet only.</p>
<ul>
<li>Subset data - timepointgroup HD</li>
<li>age is on the x-axis</li>
<li>Color is by subject</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb50"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb50-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb50-1" tabindex="-1"></a><span class="co"># Subset only HD samples</span></span>
<span id="cb50-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb50-2" tabindex="-1"></a>miso_counts_BD <span class="ot">=</span> <span class="fu">subset_samples</span>(miso_counts, timepointgroup <span class="sc">==</span> <span class="st">&quot;HD&quot;</span>)</span>
<span id="cb50-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb50-3" tabindex="-1"></a></span>
<span id="cb50-4"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb50-4" tabindex="-1"></a><span class="fu">plot_richness</span>(miso_counts_BD, <span class="at">x=</span><span class="st">&quot;fill in the blank&quot;</span>,</span>
<span id="cb50-5"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb50-5" tabindex="-1"></a>            <span class="at">color=</span><span class="st">&quot;fill in the blank&quot;</span>,</span>
<span id="cb50-6"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb50-6" tabindex="-1"></a>            <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>,</span>
<span id="cb50-7"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb50-7" tabindex="-1"></a>            <span class="at">measures=</span> <span class="fu">c</span>(<span class="st">&quot;Simpson&quot;</span>))</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1C-2. Based on the alpha plot above, how did subsetting for HD alter the age-based alpha diversity plot? Did the trend in age-based alpha diversity change? Explain if this makes sense.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>1D-1. Plot alpha diversity based on age for WO diet only.</p>
<ul>
<li>Subset data - timepointgroup WO</li>
<li>age is on the x-axis</li>
<li>Color is by subject</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb51"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb51-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb51-1" tabindex="-1"></a><span class="co"># Subset only WO samples</span></span>
<span id="cb51-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb51-2" tabindex="-1"></a>miso_counts_BD <span class="ot">=</span> <span class="fu">subset_samples</span>(miso_counts, timepointgroup <span class="sc">==</span> <span class="st">&quot;WO&quot;</span>)</span>
<span id="cb51-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb51-3" tabindex="-1"></a></span>
<span id="cb51-4"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb51-4" tabindex="-1"></a><span class="fu">plot_richness</span>(miso_counts_BD, <span class="at">x=</span><span class="st">&quot;fill in the blank&quot;</span>,</span>
<span id="cb51-5"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb51-5" tabindex="-1"></a>            <span class="at">color=</span><span class="st">&quot;fill in the blank&quot;</span>,</span>
<span id="cb51-6"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb51-6" tabindex="-1"></a>            <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>,</span>
<span id="cb51-7"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb51-7" tabindex="-1"></a>            <span class="at">measures=</span> <span class="fu">c</span>(<span class="st">&quot;Simpson&quot;</span>))</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1D-2. Based on the alpha plot above, how did subsetting for WO alter the age-based alpha diversity plot? Did the trend in age-based alpha diversity change? Explain if this makes sense. Remember, WO has twice as few datapoints as BD and HD since there is only 1 WO timepoint (versus 2 for BD and HD).</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2. Perform differential abundance analysis comparing HD to BD for the population in general, and then for older individuals specifically (because older individuals produced the biggest diet–age interaction based on alpha diversity).</strong></p>
<p>Step 2A. Perform differential abundance analysis between HD and BD.
- Design - is based on timepointgroup (no double quotes)
- Groups to compare - HD and BD (baseline)
- Plot Phylum on the X-axis and color by Class</p>
<ul>
<li>Make sure your graph has a title that includes the comparison you are making (ex. WO vs HD)</li>
<li>Refer to the “Differential abundance” section of the “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_ordination() function.</li>
<li>Use the code below as a template:</li>
</ul>
<p>STEP 1: Convert the phyloseq object to a DESeq2 object and specify experimental design</p>
<div class="sourceCode" id="cb52"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb52-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb52-1" tabindex="-1"></a>DESeq2 <span class="ot">&lt;-</span> <span class="fu">phyloseq_to_deseq2</span>(miso_counts, <span class="at">design =</span> <span class="sc">~</span> timepointgroup)</span></code></pre></div>
<p>STEP 2: Select the groups to compare, where the latter group is your baseline</p>
<div class="sourceCode" id="cb53"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb53-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb53-1" tabindex="-1"></a>comparision <span class="ot">&lt;-</span><span class="fu">c</span>(<span class="st">&quot;timepointgroup&quot;</span>, <span class="st">&quot;HD&quot;</span>, <span class="st">&quot;fill in the blank (baseline)&quot;</span>)</span></code></pre></div>
<p>STEP 3: Run the differential abundance analysis</p>
<div class="sourceCode" id="cb54"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb54-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb54-1" tabindex="-1"></a>Significant_DEseq2_ASVs<span class="ot">&lt;-</span><span class="fu">Differential_Abundance</span>(DESeq2, comparision, <span class="fl">0.05</span>)</span></code></pre></div>
<p>STEP 4: Retrieve the list of ASVs with a significant difference in abundance between the chosen groups</p>
<div class="sourceCode" id="cb55"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb55-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb55-1" tabindex="-1"></a>Significant_DEseq2_ASVs</span></code></pre></div>
<p>STEP 5: Plot the results with your chosen x axis and legend</p>
<div class="sourceCode" id="cb56"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb56-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb56-1" tabindex="-1"></a><span class="fu">ggplot</span>(significant_ASVs, <span class="fu">aes</span>(<span class="at">x =</span> Phylum, <span class="at">y=</span>log2FoldChange, <span class="at">color=</span> fill <span class="cf">in</span> the blank)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">size=</span><span class="dv">4</span>) <span class="sc">+</span></span>
<span id="cb56-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb56-2" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="sc">-</span><span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">0</span>, <span class="at">vjust=</span><span class="fl">0.5</span>))<span class="sc">+</span></span>
<span id="cb56-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb56-3" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2A-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">2A-2. How many differentially abundant ASVs were identified?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2A-3. Discuss the meaning for your differential abundance plot above. Did you observe few or many differences between BD and HD for the general population (all individuals)? What does that indicate in terms of diet-induced microbiome changes?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2B. Perform differential abundance analysis between HD and BD for older individual (&gt; 50 yo.) </strong></p>
<ul>
<li><p>Subset data - older age (&gt; 50 yo)</p></li>
<li><p>Design - is based on timepointgroup (no double quotes)</p></li>
<li><p>Groups to compare - HD and BD (baseline)</p></li>
<li><p>Plot Phylum on the X-axis and color by Class</p></li>
<li><p>Make sure your graph has a title that includes the comparison you are making (ex. WO vs HD)</p></li>
<li><p>Refer to the “Differential abundance” section of the “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_ordination() function.</p></li>
<li><p>Use the code below as a template:</p></li>
</ul>
<p>STEP 1: Subset miso_counts object to only include older individuals over age 50</p>
<div class="sourceCode" id="cb57"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb57-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb57-1" tabindex="-1"></a>subset <span class="ot">=</span> <span class="fu">subset_samples</span>(miso_counts, age <span class="sc">&gt;</span><span class="dv">50</span>)</span></code></pre></div>
<p>STEP 2: Convert the phyloseq object to a DESeq2 object and specify experimental design</p>
<div class="sourceCode" id="cb58"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb58-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb58-1" tabindex="-1"></a>DESeq2 <span class="ot">&lt;-</span> <span class="fu">phyloseq_to_deseq2</span>(subset, <span class="at">design =</span> <span class="sc">~</span> timepointgroup)</span></code></pre></div>
<p>STEP 3: Select the groups to compare, where the latter group is your baseline</p>
<div class="sourceCode" id="cb59"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb59-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb59-1" tabindex="-1"></a>comparision_subset <span class="ot">&lt;-</span><span class="fu">c</span>(<span class="st">&quot;timepointgroup&quot;</span>, <span class="st">&quot;HD&quot;</span>, <span class="st">&quot;fill in the blank (baseline)&quot;</span>)</span></code></pre></div>
<p>STEP 4: Run the differential abundance analysis</p>
<div class="sourceCode" id="cb60"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb60-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb60-1" tabindex="-1"></a>Significant_ASVs_subset <span class="ot">&lt;-</span> <span class="fu">Differential_Abundance</span>(DESeq2, comparision_subset, <span class="fl">0.05</span>)</span></code></pre></div>
<p>STEP 5: Retrieve the list of ASVs with a significant difference in abundance between the chosen groups</p>
<div class="sourceCode" id="cb61"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb61-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb61-1" tabindex="-1"></a>Significant_ASVs_subset</span></code></pre></div>
<p>STEP 6: Plot the results with your chosen x axis and legend</p>
<div class="sourceCode" id="cb62"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb62-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb62-1" tabindex="-1"></a><span class="fu">ggplot</span>(significant_ASVs_subset, <span class="fu">aes</span>(<span class="at">x =</span> Phylum, <span class="at">y=</span>log2FoldChange, <span class="at">color=</span> fill <span class="cf">in</span> the blank)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">size=</span><span class="dv">4</span>) <span class="sc">+</span></span>
<span id="cb62-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb62-2" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="sc">-</span><span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">0</span>, <span class="at">vjust=</span><span class="fl">0.5</span>))<span class="sc">+</span></span>
<span id="cb62-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#cb62-3" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2B-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">2B-2. How many differentially abundant ASVs were identified?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2B-3. Discuss the meaning for your differential abundance plot above. Did you observe any differences between BD and HD for the older individuals? How does that compare to general population? Note, by comparison, when subsetting the younger population (&lt;= 50yo), the number of differential ASVs produced is 3. What does that indicate in terms of diet-age interaction?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 3. Select data for poster and include the code used to generate the figure.</strong></p>
<ul>
<li>Prepare your final figure and associated code for the poster.</li>
<li>Figure suggestion: A 4-part figure with top panel: Figure A showing alpha diversity in general (includes all 3 dietary interventions), and Figure B showing alpha diversity for HD diet only. The bottom panel: Figure C showing differentially abundant ASVs between HD and BD for all ages, and Figure D showing differentially abundant ASVs for older individuals.</li>
<li>Ensure you have a figure legend that explains your final figure.</li>
<li>Ensure you submit the code you used to generate your final figure.</li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">Paste final poster figure and code below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
</div>
<div id="grading-criteria-4" class="section level3 hasAnchor" number="6.3.5">
<h3><span class="header-section-number">6.3.5</span> Grading criteria<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#grading-criteria-4" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Download this assignment as Microsoft Word (.docx) and upload on Canvas</li>
</ul>
</div>
<div id="footnotes-5" class="section level3 hasAnchor" number="6.3.6">
<h3><span class="header-section-number">6.3.6</span> Footnotes<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#footnotes-5" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="resources-5" class="section level4 hasAnchor" number="6.3.6.1">
<h4><span class="header-section-number">6.3.6.1</span> Resources<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#resources-5" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><a href="https://docs.google.com/document/d/1fiMHAB-BtHwYBKQ_pnjZrehIK7cooaVS_UT9bDNomrw/edit?usp=sharing">Google Doc</a></p>
</div>
<div id="contributions-and-affiliations-5" class="section level4 hasAnchor" number="6.3.6.2">
<h4><span class="header-section-number">6.3.6.2</span> Contributions and affiliations<a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#contributions-and-affiliations-5" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>Valeriya Gaysinskaya, Johns Hopkins University</li>
<li>Gauri Paul, Clovis Community College</li>
<li>Frederick Tan, Johns Hopkins University</li>
<li>Sayumi York, Notre Dame of Maryland University</li>
</ul>

</div>
</div>
</div>
<!-- </div> -->
<hr>
<center> 
  <div class="footer">
      All materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="lecture---analyzing-16s-rrna-data-with-phyloseq.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="about-the-authors.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
