<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>8.3 Activity - Exploring 16S rRNA Data with phyloseq | miniCURE: Human Gut 16S rRNA</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.46 and GitBook 2.6.7" />

  <meta property="og:title" content="8.3 Activity - Exploring 16S rRNA Data with phyloseq | miniCURE: Human Gut 16S rRNA" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="8.3 Activity - Exploring 16S rRNA Data with phyloseq | miniCURE: Human Gut 16S rRNA" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  



<meta name="date" content="2026-01-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/dasl_favicon.ico" type="image/x-icon" />
<link rel="prev" href="lecture---exploring-16s-rrna-data-with-phyloseq.html"/>
<link rel="next" href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<base target="_blank">


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="assets/style.css" type="text/css" />
<link rel="stylesheet" href="assets/C-MOOR_Template/c-moor-ottr.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.c-moor.org/" target="_blank"><img src="assets/C-MOOR_Template/c-moor-logo-vertical.png" style="width: 70%; padding-left: 10px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#learning-goals"><i class="fa fa-check"></i>Learning Goals</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="c-moor-overview.html"><a href="c-moor-overview.html"><i class="fa fa-check"></i><b>1</b> C-MOOR Overview</a>
<ul>
<li class="chapter" data-level="1.0.1" data-path="c-moor-overview.html"><a href="c-moor-overview.html#about-c-moor"><i class="fa fa-check"></i><b>1.0.1</b> About C-MOOR</a></li>
<li class="chapter" data-level="1.0.2" data-path="c-moor-overview.html"><a href="c-moor-overview.html#why-data-science"><i class="fa fa-check"></i><b>1.0.2</b> Why data science?</a></li>
<li class="chapter" data-level="1.0.3" data-path="c-moor-overview.html"><a href="c-moor-overview.html#learning-goals-1"><i class="fa fa-check"></i><b>1.0.3</b> Learning Goals</a></li>
</ul></li>
<li class="part"><span><b>INSTRUCTOR GUIDE</b></span></li>
<li class="chapter" data-level="2" data-path="for-new-instructors.html"><a href="for-new-instructors.html"><i class="fa fa-check"></i><b>2</b> For new instructors</a>
<ul>
<li class="chapter" data-level="2.0.1" data-path="for-new-instructors.html"><a href="for-new-instructors.html#course-based-undergraduate-research-experiences"><i class="fa fa-check"></i><b>2.0.1</b> Course-based undergraduate research experiences</a></li>
<li class="chapter" data-level="2.0.2" data-path="for-new-instructors.html"><a href="for-new-instructors.html#how-c-moor-supports-you"><i class="fa fa-check"></i><b>2.0.2</b> How C-MOOR supports you</a></li>
<li class="chapter" data-level="2.0.3" data-path="for-new-instructors.html"><a href="for-new-instructors.html#new-instructor-to-do-list"><i class="fa fa-check"></i><b>2.0.3</b> New instructor to-do list</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html"><i class="fa fa-check"></i><b>3</b> Human gut 16S rRNA</a>
<ul>
<li class="chapter" data-level="3.0.1" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#content-overview"><i class="fa fa-check"></i><b>3.0.1</b> Content overview</a></li>
<li class="chapter" data-level="3.0.2" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#sample-schedules"><i class="fa fa-check"></i><b>3.0.2</b> Sample Schedules</a></li>
<li class="chapter" data-level="3.0.3" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#what-to-know"><i class="fa fa-check"></i><b>3.0.3</b> What to know</a></li>
<li class="chapter" data-level="3.0.4" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#where-to-get-help"><i class="fa fa-check"></i><b>3.0.4</b> Where to get help</a></li>
<li class="chapter" data-level="3.0.5" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#grades"><i class="fa fa-check"></i><b>3.0.5</b> Grades</a></li>
<li class="chapter" data-level="3.0.6" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#footnotes"><i class="fa fa-check"></i><b>3.0.6</b> Footnotes</a></li>
</ul></li>
<li class="part"><span><b>Setting up on a cloud platform</b></span></li>
<li class="chapter" data-level="4" data-path="anvil.html"><a href="anvil.html"><i class="fa fa-check"></i><b>4</b> AnVIL</a>
<ul>
<li class="chapter" data-level="4.1" data-path="about-anvil.html"><a href="about-anvil.html"><i class="fa fa-check"></i><b>4.1</b> About AnVIL</a></li>
<li class="chapter" data-level="4.2" data-path="sign-up-for-an-anvil-account.html"><a href="sign-up-for-an-anvil-account.html"><i class="fa fa-check"></i><b>4.2</b> Sign up for an AnVIL account</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="sign-up-for-an-anvil-account.html"><a href="sign-up-for-an-anvil-account.html#create-an-anvil-account"><i class="fa fa-check"></i><b>4.2.1</b> Create an AnVIL account</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="setting-up-workspaces-on-anvil.html"><a href="setting-up-workspaces-on-anvil.html"><i class="fa fa-check"></i><b>4.3</b> Setting up workspaces on AnVIL</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="setting-up-workspaces-on-anvil.html"><a href="setting-up-workspaces-on-anvil.html#a-note-to-instructors-on-billing"><i class="fa fa-check"></i><b>4.3.1</b> A note to instructors on billing</a></li>
<li class="chapter" data-level="4.3.2" data-path="setting-up-workspaces-on-anvil.html"><a href="setting-up-workspaces-on-anvil.html#using-groups-to-manage-classes"><i class="fa fa-check"></i><b>4.3.2</b> Using groups to manage classes</a></li>
<li class="chapter" data-level="4.3.3" data-path="setting-up-workspaces-on-anvil.html"><a href="setting-up-workspaces-on-anvil.html#clone-a-workspace-on-anvil"><i class="fa fa-check"></i><b>4.3.3</b> Clone a workspace on AnVIL</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html"><i class="fa fa-check"></i><b>4.4</b> Running a module on AnVIL</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#what-is-a-workspace"><i class="fa fa-check"></i><b>4.4.1</b> What is a workspace?</a></li>
<li class="chapter" data-level="4.4.2" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#running-modules-on-anvil"><i class="fa fa-check"></i><b>4.4.2</b> Running modules on AnVIL</a></li>
<li class="chapter" data-level="4.4.3" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#closing-out-a-session-on-anvil"><i class="fa fa-check"></i><b>4.4.3</b> Closing out a session on AnVIL</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Scientific Literature</b></span></li>
<li class="chapter" data-level="5" data-path="scientific-literature.html"><a href="scientific-literature.html"><i class="fa fa-check"></i><b>5</b> Scientific Literature</a>
<ul>
<li class="chapter" data-level="5.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html"><i class="fa fa-check"></i><b>5.1</b> Pre-lab - Scientific literature</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#purpose-3"><i class="fa fa-check"></i><b>5.1.1</b> Purpose</a></li>
<li class="chapter" data-level="5.1.2" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#grading-criteria"><i class="fa fa-check"></i><b>5.1.2</b> Grading criteria</a></li>
<li class="chapter" data-level="5.1.3" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#activity"><i class="fa fa-check"></i><b>5.1.3</b> Activity</a></li>
<li class="chapter" data-level="5.1.4" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#footnotes-1"><i class="fa fa-check"></i><b>5.1.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="lecture---reading-a-paper.html"><a href="lecture---reading-a-paper.html"><i class="fa fa-check"></i><b>5.2</b> Lecture - Reading a paper</a></li>
<li class="chapter" data-level="5.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html"><i class="fa fa-check"></i><b>5.3</b> Activity - Scientific Literature</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#purpose-4"><i class="fa fa-check"></i><b>5.3.1</b> Purpose</a></li>
<li class="chapter" data-level="5.3.2" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#learning-objectives-3"><i class="fa fa-check"></i><b>5.3.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.3.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#introduction"><i class="fa fa-check"></i><b>5.3.3</b> Introduction</a></li>
<li class="chapter" data-level="5.3.4" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#instructions-1"><i class="fa fa-check"></i><b>5.3.4</b> Instructions</a></li>
<li class="chapter" data-level="5.3.5" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-1---overview-of-the-paper"><i class="fa fa-check"></i><b>5.3.5</b> Activity 1 - Overview of the paper</a></li>
<li class="chapter" data-level="5.3.6" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-2---methods-and-figure-analysis-in-class-and-homework"><i class="fa fa-check"></i><b>5.3.6</b> Activity 2 - Methods and Figure Analysis (in class and homework)</a></li>
<li class="chapter" data-level="5.3.7" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#grading-criteria-1"><i class="fa fa-check"></i><b>5.3.7</b> Grading criteria</a></li>
<li class="chapter" data-level="5.3.8" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#footnotes-2"><i class="fa fa-check"></i><b>5.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Project: 16S rRNA Research</b></span></li>
<li class="chapter" data-level="6" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html"><i class="fa fa-check"></i><b>6</b> Reference - Meet the MISO study</a>
<ul>
<li class="chapter" data-level="6.0.1" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#miso-study-diet-and-sampling-schedule"><i class="fa fa-check"></i><b>6.0.1</b> MISO study diet and sampling schedule</a></li>
<li class="chapter" data-level="6.0.2" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#miso-study-variables-and-factors"><i class="fa fa-check"></i><b>6.0.2</b> MISO study variables and factors</a></li>
<li class="chapter" data-level="6.0.3" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#amplicon-sequence-variants-asvs-data"><i class="fa fa-check"></i><b>6.0.3</b> Amplicon sequence variants (ASVs) data</a></li>
<li class="chapter" data-level="6.0.4" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#footnotes-3"><i class="fa fa-check"></i><b>6.0.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="s-rrna-data-in-spreadsheets.html"><a href="s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>7</b> 16S rRNA Data in spreadsheets</a>
<ul>
<li class="chapter" data-level="7.1" data-path="exploring-16s-rrna-data-in-spreadsheets.html"><a href="exploring-16s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>7.1</b> Exploring 16S rRNA data in spreadsheets</a></li>
<li class="chapter" data-level="7.2" data-path="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><a href="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><i class="fa fa-check"></i><b>7.2</b> Lecture - Introducing 16S rRNA Data Using Spreadsheets</a></li>
<li class="chapter" data-level="7.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html"><i class="fa fa-check"></i><b>7.3</b> Activity - Introducing 16S rRNA Data Using Spreadsheets</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#purpose-5"><i class="fa fa-check"></i><b>7.3.1</b> Purpose</a></li>
<li class="chapter" data-level="7.3.2" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#learning-objectives-4"><i class="fa fa-check"></i><b>7.3.2</b> Learning objectives</a></li>
<li class="chapter" data-level="7.3.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#introduction-1"><i class="fa fa-check"></i><b>7.3.3</b> Introduction</a></li>
<li class="chapter" data-level="7.3.4" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-1-quick-data-overview-based-on-1-sample"><i class="fa fa-check"></i><b>7.3.4</b> Activity 1 – Quick data overview based on 1 sample</a></li>
<li class="chapter" data-level="7.3.5" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-2-explore-variation-between-individuals"><i class="fa fa-check"></i><b>7.3.5</b> Activity 2 – Explore variation between individuals</a></li>
<li class="chapter" data-level="7.3.6" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-2-3.-plot-individual-variation-in-abundance-of-the-most-common-asv."><i class="fa fa-check"></i><b>7.3.6</b> Activity 2-3. Plot individual variation in abundance of the most common ASV.</a></li>
<li class="chapter" data-level="7.3.7" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-3-explore-variation-within-a-single-individual-associated-with-diet"><i class="fa fa-check"></i><b>7.3.7</b> Activity 3 – Explore variation within a single individual associated with diet</a></li>
<li class="chapter" data-level="7.3.8" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#footnotes-4"><i class="fa fa-check"></i><b>7.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="exploring-16s-rrna-data-with-phyloseq.html"><a href="exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>8</b> Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="8.1" data-path="exploring-16s-rrna-data-with-phyloseq-1.html"><a href="exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>8.1</b> Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="8.2" data-path="lecture---exploring-16s-rrna-data-with-phyloseq.html"><a href="lecture---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>8.2</b> Lecture - Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="8.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>8.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="8.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#learning-objectives-5"><i class="fa fa-check"></i><b>8.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="8.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#introduction-2"><i class="fa fa-check"></i><b>8.3.2</b> Introduction</a></li>
<li class="chapter" data-level="8.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-1-explore-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>8.3.3</b> Activity 1 – Explore 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="8.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-2-try-it-out-questions"><i class="fa fa-check"></i><b>8.3.4</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="8.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#grading-criteria-2"><i class="fa fa-check"></i><b>8.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="8.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#footnotes-5"><i class="fa fa-check"></i><b>8.3.6</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><i class="fa fa-check"></i><b>8.4</b> Try it Question 1 - What has more impact on human microbial variation, diet or individuality</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1a-plot-high-level-taxonomy-phylum-level-for-each-timepoint-using-the-plot_bar-function.-specifically-your-plot-should-show-phyla-diversity-across-all-5-timepoints."><i class="fa fa-check"></i><b>8.4.1</b> Step 1A: Plot high-level taxonomy (Phylum level) for each timepoint using the plot_bar() function.** Specifically, your plot should show Phyla diversity across all 5 timepoints.</a></li>
<li class="chapter" data-level="8.4.2" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1b-plot-high-level-taxonomy-phylum-level-for-each-individual-using-only-normalized-homogenized-diet-hd-timepoints-to-minimize-contribution-of-diet-to-the-analysis-using-the-plot_bar-function."><i class="fa fa-check"></i><b>8.4.2</b> Step 1B: Plot high-level taxonomy (Phylum level) for each individual, using only normalized homogenized diet “HD” timepoints to minimize contribution of diet to the analysis using the plot_bar() function.</a></li>
</ul></li>
<li class="chapter" data-level="8.5" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><i class="fa fa-check"></i><b>8.5</b> Try it Question 2 - What are some of the most abundant microbes when viewed at different taxonomic resolutions</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2a.-plot-phyla-across-all-subjects-and-identify-the-most-abundant-phyla.-specifically-your-plot-should-show-phyla-diversity-across-all-21-individuals."><i class="fa fa-check"></i><b>8.5.1</b> Step 2A. Plot Phyla across all subjects and identify the most abundant Phyla. Specifically, your plot should show Phyla diversity across all 21 individuals.</a></li>
<li class="chapter" data-level="8.5.2" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2b.-subset-the-most-abundant-phyla-and-plot-order-skip-class"><i class="fa fa-check"></i><b>8.5.2</b> Step 2B. Subset the most abundant Phyla, and plot Order (Skip Class)</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>8.6</b> Try it Question 3 - Does gender contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="8.6.1" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3c.-test-gender-based-difference-in-species-for-the-low-abundant-phylum-you-selected.-specifically-your-plot-should-show-species-diversity-within-male-and-female-groups-for-only-your-phylum-of-choice."><i class="fa fa-check"></i><b>8.6.1</b> Step 3C. Test gender-based difference in Species for the low abundant phylum you selected.** Specifically, your plot should show species diversity within male and female groups for only your phylum of choice.</a></li>
<li class="chapter" data-level="8.6.2" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3d-plot-the-individual-abundance-of-2-candidate-microbes-for-gender-differences-based-on-your-analysis-above-andor-the-curated-menu-of-microbes-below-previously-associated-with-gender-variation-with-color-denoting-gender."><i class="fa fa-check"></i><b>8.6.2</b> Step 3D: Plot the individual abundance of 2 candidate microbes for gender differences based on your analysis above, and/or the curated menu of microbes below (previously associated with gender variation), with color denoting gender.</a></li>
</ul></li>
<li class="chapter" data-level="8.7" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>8.7</b> Try it Question 4 - Does age contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="8.7.1" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4a.-perform-an-initial-survey-to-see-if-there-are-any-differences-in-phyla-related-to-age."><i class="fa fa-check"></i><b>8.7.1</b> Step 4A. Perform an initial survey to see if there are any differences in phyla related to age.</a></li>
<li class="chapter" data-level="8.7.2" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4b.-merge-samples-by-age-then-re-normalize-and-finally-re-plot-phyla-by-age."><i class="fa fa-check"></i><b>8.7.2</b> Step 4B. Merge samples by age, then re-normalize and finally re-plot Phyla by age.</a></li>
<li class="chapter" data-level="8.7.3" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4c.-candidate-phyla-implicated-in-age-variation-include-firmicutes-and-bacteroidetes.-plot-these-specific-phyla-on-their-own-separately-to-observe-any-shift-that-occurs-with-age."><i class="fa fa-check"></i><b>8.7.3</b> Step 4C. Candidate Phyla implicated in age variation include Firmicutes and Bacteroidetes. Plot these specific phyla on their own, separately, to observe any shift that occurs with age.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="analyzing-16s-rrna-data-with-phyloseq.html"><a href="analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>9</b> Analyzing 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="9.1" data-path="analyzing-16s-rrna-data-with-phyloseq-1.html"><a href="analyzing-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>9.1</b> Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="9.2" data-path="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><a href="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>9.2</b> Lecture - Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="9.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>9.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#purpose-7"><i class="fa fa-check"></i><b>9.3.1</b> Purpose</a></li>
<li class="chapter" data-level="9.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#learning-objectives-6"><i class="fa fa-check"></i><b>9.3.2</b> Learning objectives</a></li>
<li class="chapter" data-level="9.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#introduction-3"><i class="fa fa-check"></i><b>9.3.3</b> Introduction</a></li>
<li class="chapter" data-level="9.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-1-analyze-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>9.3.4</b> Activity 1 – Analyze 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="9.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-2-try-it-out-questions-1"><i class="fa fa-check"></i><b>9.3.5</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="9.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#grading-criteria-3"><i class="fa fa-check"></i><b>9.3.6</b> Grading criteria</a></li>
<li class="chapter" data-level="9.3.7" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#footnotes-6"><i class="fa fa-check"></i><b>9.3.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><i class="fa fa-check"></i><b>9.4</b> Try it Question 1 - How sensitive is microbial diversity to variables like diet, age and gender</a>
<ul>
<li class="chapter" data-level="9.4.1" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1a.-plot-alpha-diversity-of-the-full-miso-dataset-by-subject-individuals."><i class="fa fa-check"></i><b>9.4.1</b> Step 1A. Plot alpha diversity of the full MISO dataset by subject (individuals).</a></li>
<li class="chapter" data-level="9.4.2" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1b.-plot-alpha-diversity-of-the-full-miso-dataset-by-diet"><i class="fa fa-check"></i><b>9.4.2</b> Step 1B. Plot alpha diversity of the full MISO dataset by diet</a></li>
<li class="chapter" data-level="9.4.3" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1c.-plot-alpha-diversity-of-the-full-miso-dataset-by-age."><i class="fa fa-check"></i><b>9.4.3</b> Step 1C. Plot alpha diversity of the full MISO dataset by age.</a></li>
<li class="chapter" data-level="9.4.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1d.-plot-alpha-diversity-of-the-full-miso-dataset-by-gender."><i class="fa fa-check"></i><b>9.4.4</b> Step 1D. Plot alpha diversity of the full MISO dataset by gender.</a></li>
<li class="chapter" data-level="9.4.5" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-2.-plot-alpha-diversity-of-the-full-miso-dataset-grouping-based-on-levels-for-5-different-metabolites-creatinine-pcs-is-hipp-pag."><i class="fa fa-check"></i><b>9.4.5</b> Step 2. Plot alpha diversity of the full MISO dataset, grouping based on levels for 5 different metabolites: Creatinine, PCS, IS, HIPP, PAG.</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><i class="fa fa-check"></i><b>9.5</b> Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-1.-make-a-pcoa-plot-ordinating-on-the-entire-miso-dataset-and-coloring-by-individual.-investigate-resulting-plot-shape."><i class="fa fa-check"></i><b>9.5.1</b> Step 1. Make a PcOA plot, ordinating on the entire miso dataset, and coloring by individual. Investigate resulting plot shape.</a></li>
<li class="chapter" data-level="9.5.2" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-2.-using-the-pcoa-plot-from-step-1-above-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-diet-gender-and-age"><i class="fa fa-check"></i><b>9.5.2</b> Step 2. Using the PCoA plot from Step 1 above, look for any correlations between the shape of your PCoA plot and diet, gender and age</a></li>
<li class="chapter" data-level="9.5.3" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-3.-using-the-pcoa-plot-from-step-1-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-the-level-of-metabolites."><i class="fa fa-check"></i><b>9.5.3</b> Step 3. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and the level of metabolites.</a></li>
<li class="chapter" data-level="9.5.4" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-4.-one-way-of-checking-if-the-metabolite-levels-indeed-correlate-with-your-data-is-to-see-if-subsetting-smaller-chunks-or-specific-chunks-of-the-data-will-still-maintain-the-metabolite-data-relationship-or-break-it."><i class="fa fa-check"></i><b>9.5.4</b> Step 4. One way of checking if the metabolite levels indeed correlate with your data, is to see if subsetting smaller chunks or specific chunks of the data will still maintain the metabolite-data relationship or break it.</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><i class="fa fa-check"></i><b>9.6</b> Try it Question 3 - What microbes (ASVs) differ between males and females, and does age have an impact</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3a.-perform-differential-abundance-analysis-between-genders."><i class="fa fa-check"></i><b>9.6.1</b> Step 3A. Perform differential abundance analysis between genders.</a></li>
<li class="chapter" data-level="9.6.2" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3b.-determine-the-impact-of-younger-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>9.6.2</b> Step 3B. Determine the impact of younger age on differential abundance in males and females, by subsetting age &lt; 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
<li class="chapter" data-level="9.6.3" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3c.-determine-the-impact-of-older-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>9.6.3</b> Step 3C. Determine the impact of older age on differential abundance in males and females, by subsetting age &gt;= 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
</ul></li>
<li class="chapter" data-level="9.7" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><i class="fa fa-check"></i><b>9.7</b> Try it Question 4 - Is there an interaction between diet and age and the microbiome</a>
<ul>
<li class="chapter" data-level="9.7.1" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-1.-plot-alpha-diversity-based-on-age-for-all-dietary-groups-and-then-for-bd-hd-and-wo."><i class="fa fa-check"></i><b>9.7.1</b> Step 1. Plot alpha diversity based on age for all dietary groups, and then for BD, HD and WO.</a></li>
<li class="chapter" data-level="9.7.2" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-2.-perform-differential-abundance-analysis-comparing-hd-to-bd-for-the-population-in-general-and-then-for-older-individuals-specifically."><i class="fa fa-check"></i><b>9.7.2</b> Step 2. Perform differential abundance analysis comparing HD to BD for the population in general, and then for older individuals specifically.</a></li>
<li class="chapter" data-level="9.7.3" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-2b.-perform-differential-abundance-analysis-between-hd-and-bd-for-older-individual-50-yo."><i class="fa fa-check"></i><b>9.7.3</b> Step 2B. Perform differential abundance analysis between HD and BD for older individual (&gt; 50 yo.)</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Community analysis and feedback</b></span></li>
<li class="chapter" data-level="10" data-path="community-analysis-and-feedback.html"><a href="community-analysis-and-feedback.html"><i class="fa fa-check"></i><b>10</b> Community analysis and feedback</a>
<ul>
<li class="chapter" data-level="10.1" data-path="lecture---scientific-posters.html"><a href="lecture---scientific-posters.html"><i class="fa fa-check"></i><b>10.1</b> Lecture - Scientific Posters</a></li>
<li class="chapter" data-level="10.2" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html"><i class="fa fa-check"></i><b>10.2</b> Activity - Scientific Posters</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html#activity-1"><i class="fa fa-check"></i><b>10.2.1</b> Activity</a></li>
<li class="chapter" data-level="10.2.2" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html#grading-criteria-4"><i class="fa fa-check"></i><b>10.2.2</b> Grading Criteria</a></li>
<li class="chapter" data-level="10.2.3" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html#footnotes-7"><i class="fa fa-check"></i><b>10.2.3</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html"><i class="fa fa-check"></i><b>10.3</b> Activity - Share Your Poster</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#introduction-4"><i class="fa fa-check"></i><b>10.3.1</b> Introduction</a></li>
<li class="chapter" data-level="10.3.2" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#activity-2-present-your-poster"><i class="fa fa-check"></i><b>10.3.2</b> Activity 2 – Present Your Poster</a></li>
<li class="chapter" data-level="10.3.3" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#grading-criteria-5"><i class="fa fa-check"></i><b>10.3.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="10.3.4" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#footnotes-8"><i class="fa fa-check"></i><b>10.3.4</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="11" data-path="professional-development.html"><a href="professional-development.html"><i class="fa fa-check"></i><b>11</b> Professional Development</a>
<ul>
<li class="chapter" data-level="11.1" data-path="lecture---next-steps.html"><a href="lecture---next-steps.html"><i class="fa fa-check"></i><b>11.1</b> Lecture - Next Steps</a></li>
<li class="chapter" data-level="11.2" data-path="activity---next-steps.html"><a href="activity---next-steps.html"><i class="fa fa-check"></i><b>11.2</b> Activity - Next Steps</a>
<ul>
<li class="chapter" data-level="11.2.1" data-path="activity---next-steps.html"><a href="activity---next-steps.html#introduction-5"><i class="fa fa-check"></i><b>11.2.1</b> Introduction</a></li>
<li class="chapter" data-level="11.2.2" data-path="activity---next-steps.html"><a href="activity---next-steps.html#activity-1-complete-feedback"><i class="fa fa-check"></i><b>11.2.2</b> Activity 1 – Complete Feedback</a></li>
<li class="chapter" data-level="11.2.3" data-path="activity---next-steps.html"><a href="activity---next-steps.html#activity-2-research-opportunities"><i class="fa fa-check"></i><b>11.2.3</b> Activity 2 – Research Opportunities</a></li>
<li class="chapter" data-level="11.2.4" data-path="activity---next-steps.html"><a href="activity---next-steps.html#activity-3-explore-communities"><i class="fa fa-check"></i><b>11.2.4</b> Activity 3 – Explore Communities</a></li>
<li class="chapter" data-level="11.2.5" data-path="activity---next-steps.html"><a href="activity---next-steps.html#optional-activity-4-share-work"><i class="fa fa-check"></i><b>11.2.5</b> (Optional) Activity 4 – Share Work</a></li>
<li class="chapter" data-level="11.2.6" data-path="activity---next-steps.html"><a href="activity---next-steps.html#grading-criteria-6"><i class="fa fa-check"></i><b>11.2.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="11.2.7" data-path="activity---next-steps.html"><a href="activity---next-steps.html#footnotes-9"><i class="fa fa-check"></i><b>11.2.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="11.3" data-path="science-talks.html"><a href="science-talks.html"><i class="fa fa-check"></i><b>11.3</b> Science Talks</a>
<ul>
<li class="chapter" data-level="11.3.1" data-path="science-talks.html"><a href="science-talks.html#science-talks---activity-1"><i class="fa fa-check"></i><b>11.3.1</b> Science Talks - Activity 1</a></li>
<li class="chapter" data-level="11.3.2" data-path="science-talks.html"><a href="science-talks.html#science-talks---activity-2"><i class="fa fa-check"></i><b>11.3.2</b> Science Talks - Activity 2</a></li>
</ul></li>
<li class="chapter" data-level="11.4" data-path="create-your-cv.html"><a href="create-your-cv.html"><i class="fa fa-check"></i><b>11.4</b> Create your CV</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="c-moor-scholars.html"><a href="c-moor-scholars.html"><i class="fa fa-check"></i><b>12</b> C-MOOR Scholars</a>
<ul>
<li class="chapter" data-level="12.1" data-path="c-moor-scholars-1.html"><a href="c-moor-scholars-1.html"><i class="fa fa-check"></i><b>12.1</b> C-MOOR Scholars</a>
<ul>
<li class="chapter" data-level="12.1.1" data-path="c-moor-scholars-1.html"><a href="c-moor-scholars-1.html#c-moor-scholars-links"><i class="fa fa-check"></i><b>12.1.1</b> C-MOOR Scholars links</a></li>
</ul></li>
<li class="chapter" data-level="12.2" data-path="pursue-further-research.html"><a href="pursue-further-research.html"><i class="fa fa-check"></i><b>12.2</b> Pursue further research</a>
<ul>
<li class="chapter" data-level="12.2.1" data-path="pursue-further-research.html"><a href="pursue-further-research.html#research-opportunities-at-your-institution"><i class="fa fa-check"></i><b>12.2.1</b> Research opportunities at your institution</a></li>
<li class="chapter" data-level="12.2.2" data-path="pursue-further-research.html"><a href="pursue-further-research.html#find-a-research-lab-to-join"><i class="fa fa-check"></i><b>12.2.2</b> Find a research lab to join</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="biodigs.html"><a href="biodigs.html"><i class="fa fa-check"></i><b>12.3</b> BioDIGS</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="online-community.html"><a href="online-community.html"><i class="fa fa-check"></i><b>13</b> Online Community</a></li>
<li class="part"><span><b>Resources</b></span></li>
<li class="chapter" data-level="14" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html"><i class="fa fa-check"></i><b>14</b> Spreadsheets 101</a>
<ul>
<li class="chapter" data-level="14.0.1" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#getting-a-copy-of-the-spreadsheet"><i class="fa fa-check"></i><b>14.0.1</b> Getting a copy of the spreadsheet</a></li>
<li class="chapter" data-level="14.0.2" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#basic-spreadsheet-terms"><i class="fa fa-check"></i><b>14.0.2</b> Basic spreadsheet terms</a></li>
<li class="chapter" data-level="14.0.3" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#summary-statistics-by-row-or-column"><i class="fa fa-check"></i><b>14.0.3</b> Summary statistics by row or column</a></li>
<li class="chapter" data-level="14.0.4" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#adding-data-to-cells"><i class="fa fa-check"></i><b>14.0.4</b> Adding data to cells</a></li>
<li class="chapter" data-level="14.0.5" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#applying-a-calculation-to-many-cells"><i class="fa fa-check"></i><b>14.0.5</b> Applying a calculation to many cells</a></li>
<li class="chapter" data-level="14.0.6" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#manipulating-the-spreadsheet"><i class="fa fa-check"></i><b>14.0.6</b> Manipulating the spreadsheet</a></li>
<li class="chapter" data-level="14.0.7" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#useful-formulas-in-google-sheets"><i class="fa fa-check"></i><b>14.0.7</b> Useful formulas in Google Sheets</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="figure-and-tables-guide.html"><a href="figure-and-tables-guide.html"><i class="fa fa-check"></i><b>15</b> Figure and tables guide</a>
<ul>
<li class="chapter" data-level="15.0.1" data-path="figure-and-tables-guide.html"><a href="figure-and-tables-guide.html#figures"><i class="fa fa-check"></i><b>15.0.1</b> Figures</a></li>
<li class="chapter" data-level="15.0.2" data-path="figure-and-tables-guide.html"><a href="figure-and-tables-guide.html#tables"><i class="fa fa-check"></i><b>15.0.2</b> Tables</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="s-minicure-guide.html"><a href="s-minicure-guide.html"><i class="fa fa-check"></i><b>16</b> 16S miniCURE Guide</a>
<ul>
<li class="chapter" data-level="16.0.1" data-path="s-minicure-guide.html"><a href="s-minicure-guide.html#footnotes-10"><i class="fa fa-check"></i><b>16.0.1</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">miniCURE: Human Gut 16S rRNA</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=10.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/C-MOOR_Template/dasl_thin_main_image.png">
</div>
<div id="activity---exploring-16s-rrna-data-with-phyloseq" class="section level2 hasAnchor" number="8.3">
<h2><span class="header-section-number">8.3</span> Activity - Exploring 16S rRNA Data with phyloseq<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity---exploring-16s-rrna-data-with-phyloseq" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="purpose-6" class="section level4 hasAnchor" number="8.3.0.1">
<h4><span class="header-section-number">8.3.0.1</span> Purpose<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#purpose-6" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>To explore bacterial diversity based on 16S rRNA gene sequencing using data from the “Impact of a 7-day homogeneous diet on interpersonal variation in human gut microbiomes and metabolomes” by Guthrie et al., 2023. This study is also referred to as the MISO study for “Microbiome Individuality and Stability Over Time” because the study aims to understand variation (or stability) of microbiomes across individuals.</p>
<p>This activity aims to explore and profile the metagenomic diversity using the R/Bioconductor package phyloseq which leverages various R-based tools to produce publication-quality taxonomy profiling and analysis graphics.</p>
</div>
<div id="learning-objectives-5" class="section level3 hasAnchor" number="8.3.1">
<h3><span class="header-section-number">8.3.1</span> Learning objectives<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#learning-objectives-5" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li>Use phyloseq to explore data associated with Amplicon Sequence Variants (ASVs).</li>
<li>In phyloseq, subset data based on metadata and taxonomy.</li>
<li>In phyloseq, profile and plot taxonomy.</li>
</ol>
</div>
<div id="introduction-2" class="section level3 hasAnchor" number="8.3.2">
<h3><span class="header-section-number">8.3.2</span> Introduction<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#introduction-2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The most popular sequencing technique for the analysis of bacterial diversity is targeted sequencing, or sequencing of a specific gene (or region of a gene, e.g. a hypervariable region of the bacterial 16S ribosomal rRNA gene) using polymerase chain reaction (PCR) to create sequences called amplicons. Sequence variation in the resulting amplicons creates amplicon sequence variants (ASVs). ASVs varying from as little as one single nucleotide can be defined as separate ASVs, which can be further clustered into OTUs (Operational Taxonomic Units) based on sequence similarity; e.g. as ASVs within 1% sequence difference can clustered into the same species/OTU.</p>
</div>
<div id="activity-1-explore-16s-rrna-data-with-phyloseq-tutorial" class="section level3 hasAnchor" number="8.3.3">
<h3><span class="header-section-number">8.3.3</span> Activity 1 – Explore 16S rRNA Data with phyloseq tutorial<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-1-explore-16s-rrna-data-with-phyloseq-tutorial" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 25 min</em></p>
<p>Explore a phyloseq object through the “Explore 16S rRNA Data with phyloseq” tutorial on SciServer.</p>
<p>Log into SciServer, click on compute, and create a new C-MOOR LearnR container. When creating a container, remember to:</p>
<ol style="list-style-type: decimal">
<li>Access the C-MOOR Tutorials</li>
</ol>
<ul>
<li><p>If you are using SciServer, log into SciServer, click on compute and open your “C-MOOR LearnR” container. Visit SciServer Guides and FAQs if you need to jog your memory on how to do this.</p></li>
<li><p>If you are using AnVIL, log into AnVIL, navigate to your class Workspace, start up an RStudio Cloud Environment, and open RStudio. Visit the AnVIL Guides and FAQs if you need to jog your memory on how to do this. This module can be found in the “1-explore-ps” folder of the “16s” curriculum folder.</p></li>
<li><p>If you are using an alternative setup, follow the instructions provided by your instructor.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Start the “Explore 16S rRNA Data with phyloseq” tutorial. Visit SciServer Guides and FAQs. If you need assistance accessing the tutorial.</li>
<li>To move through the activities click “Continue” at the bottom of the screen. When you are done with a topic, click “Next Topic” to move on.</li>
<li>This tutorial has small boxes in which you can enter and run short lines of code to analyze the data.</li>
<li>As you work through the tutorial, take snapshots of your work and paste your answers in the grey boxes below:</li>
</ol>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>1-1. OTU_table section – Take a snapshot and paste your code and the output for the following question in the tutorial: “What is the normalized count for ASV115 in sample 10?”</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>1-2. Visualizing Taxonomy section – Take a snapshot and paste your code and the output for the following question in the tutorial: “What timepoint has the lowest proportion of phylum Verrucomicrobia?”</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Using subset_taxa() section – Take a snapshot and paste your code and the output for the following question in the tutorial: “What are the 2 most abundant orders in the class Gammaproteobacteria?”</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="activity-2-try-it-out-questions" class="section level3 hasAnchor" number="8.3.4">
<h3><span class="header-section-number">8.3.4</span> Activity 2 – Try it out questions<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-2-try-it-out-questions" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 60 min</em></p>
<p>With your group, perform some exploratory data analysis selecting from one of the four questions below or coming up with your own question. Each question has its own section with code templates for you to use below. Select a question here, then go to the page specific to your question.</p>
<div id="question-1.-what-has-more-impact-on-human-microbial-variation-diet-or-individuality" class="section level4 hasAnchor" number="8.3.4.1">
<h4><span class="header-section-number">8.3.4.1</span> Question 1. What has more impact on human microbial variation, diet or individuality?<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#question-1.-what-has-more-impact-on-human-microbial-variation-diet-or-individuality" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Diet is suggested to play an important role in shaping the human gut microbiome. However, other factors that are specific to an individual such as their physical fitness, metabolism, and genotype are also suspected to play a strong role. Here we will explore the impact of diet and individuality on human gut microbiome.</p>
<p>Approach: Examine high-level (Phylum) taxonomy for 5 timepoints and 21 individuals and evaluate if the change of diet or inter-individual differences have more impact on microbe variation.</p>
<ol style="list-style-type: decimal">
<li>Evaluate the effect of diet – Plot the three dietary groups BD, HD, and WO corresponding to 5 experimental timepoints (where timepoints 1 and 2 are BD, timepoints 3 and 4 are HD and timepoint 5 is WO).</li>
<li>Evaluate the effect of individuality – Plot the microbiome for all 21 individuals but focus on just the HD timepointgroup (where diet is the same).</li>
</ol>
</div>
<div id="question-2.-what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions" class="section level4 hasAnchor" number="8.3.4.2">
<h4><span class="header-section-number">8.3.4.2</span> Question 2. What are some of the most abundant microbes when viewed at different taxonomic resolutions?<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#question-2.-what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Microbes within the core microbiome - the most common and abundant species across samples in a given group - are likely to be involved in key functions of the holobiont. Here we will explore some of the most common microbes at increasing levels of taxonomic resolution.</p>
<p>Approach: Examine microbial composition for most abundant taxonomic ranks in the human gut of the 21 individuals. To do so, you will use progressive sub-setting of the most abundant taxonomic ranks ranging from Phylum down to Species.</p>
<ol style="list-style-type: decimal">
<li>Plot all the Phyla and identify the most abundant Phylum</li>
<li>Subset the most abundant Phylum, and plot all the Orders</li>
<li>Subset the most abundant Order, and plot all the Families</li>
<li>Subset the most abundant Family, and plot all the Genera</li>
<li>Subset the most abundant Genus, and plot all the Species</li>
</ol>
</div>
<div id="question-3.-does-gender-have-an-impact-on-the-human-gut-microbiome" class="section level4 hasAnchor" number="8.3.4.3">
<h4><span class="header-section-number">8.3.4.3</span> Question 3. Does gender have an impact on the human gut microbiome?<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#question-3.-does-gender-have-an-impact-on-the-human-gut-microbiome" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The biological sex of the host has been suggested to help shape or influence its gut microbiome. Some candidate microbes have been implicated in sex differences. Here we will explore the potential gender-based differences in the human gut microbiome.</p>
<p>Approach: Test if Phyla and/or species vary based on gender. First you will survey the differences between male and female subjects in the composition of their Phylum and species. Then you will select some candidate microbes of interest to examine in further detail.</p>
<ol style="list-style-type: decimal">
<li>Survey differences in phylum and species composition by gender.</li>
<li>Plot differences in species for a high abundance phylum by gender.</li>
<li>Plot differences in species for a low abundance phylum by gender.</li>
<li>Plot two candidate microbes reported in the literature to vary with gender.</li>
</ol>
</div>
<div id="question-4.-does-age-have-an-impact-on-human-gut-microbiomes" class="section level4 hasAnchor" number="8.3.4.4">
<h4><span class="header-section-number">8.3.4.4</span> Question 4. Does age have an impact on human gut microbiomes?<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#question-4.-does-age-have-an-impact-on-human-gut-microbiomes" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The human gut microbiome has been associated with age-related disease states, immune-system changes, and metabolic function. Here we will explore the potential microbiome changes associated with aging.</p>
<p>Approach: Test if Phylum and/or species composition differs by age across the dataset. First you will plot the relationship between age and phylum; we will see that age is a more challenging variable to work with than our categorical variables. You will collapse and transform the data to fix this issue to create a final plot for your interpretation.</p>
<ol style="list-style-type: decimal">
<li>Plot all Phyla based on age.</li>
<li>Normalize age that includes multiple individuals to avoid overestimation (e.g., 2 individuals are of age 27, 46 and 58, and 3 individuals are 54).</li>
<li>Profile candidate Phyla Firmicutes based on age.</li>
<li>Profile candidate Phyla Bacteroidetes based on age.</li>
</ol>
</div>
</div>
<div id="grading-criteria-2" class="section level3 hasAnchor" number="8.3.5">
<h3><span class="header-section-number">8.3.5</span> Grading criteria<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#grading-criteria-2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Download the assignment to your local computer as a .docx, complete it, and upload the assignment to your LMS (Blackboard, Canvas, Google Classroom).</li>
</ul>
</div>
<div id="footnotes-5" class="section level3 hasAnchor" number="8.3.6">
<h3><span class="header-section-number">8.3.6</span> Footnotes<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#footnotes-5" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="resources-4" class="section level4 hasAnchor" number="8.3.6.1">
<h4><span class="header-section-number">8.3.6.1</span> Resources<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#resources-4" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li><p><a href="https://docs.google.com/document/d/1d1co_BVzZ1P5RuUYnPWrkB-Lvo7f4lYvahXhoCOjvIg/edit?usp=sharing">Google Doc</a></p></li>
<li><p><a href="https://docs.google.com/document/d/1pYTq_U-e3Fath5yB6N9b5ic-cMynUtclkEgdojkAXwE/edit?usp=sharing">16S rRNA R Quick Reference</a></p></li>
</ul>
</div>
<div id="contributions-and-affiliations-4" class="section level4 hasAnchor" number="8.3.6.2">
<h4><span class="header-section-number">8.3.6.2</span> Contributions and affiliations<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#contributions-and-affiliations-4" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>Valeriya Gaysinskaya, Johns Hopkins University</li>
<li>Gauri Paul, Clovis Community College</li>
<li>Frederick Tan, Johns Hopkins University</li>
<li>Sayumi York, Notre Dame of Maryland University</li>
</ul>
</div>
</div>
</div>
<hr>
<center> 
  <div class="footer">
      All materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="lecture---exploring-16s-rrna-data-with-phyloseq.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "section"
  }
});
});
</script>

</body>

</html>
