<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>7.3 Activity - Introducing 16S rRNA Data Using Spreadsheets | miniCURE: Human Gut 16S rRNA</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.46 and GitBook 2.6.7" />

  <meta property="og:title" content="7.3 Activity - Introducing 16S rRNA Data Using Spreadsheets | miniCURE: Human Gut 16S rRNA" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="7.3 Activity - Introducing 16S rRNA Data Using Spreadsheets | miniCURE: Human Gut 16S rRNA" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  



<meta name="date" content="2026-01-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/dasl_favicon.ico" type="image/x-icon" />
<link rel="prev" href="lecture---introducing-16s-rrna-data-using-spreadsheets.html"/>
<link rel="next" href="exploring-16s-rrna-data-with-phyloseq.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<base target="_blank">


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="assets/style.css" type="text/css" />
<link rel="stylesheet" href="assets/C-MOOR_Template/c-moor-ottr.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.c-moor.org/" target="_blank"><img src="assets/C-MOOR_Template/c-moor-logo-vertical.png" style="width: 70%; padding-left: 10px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#learning-goals"><i class="fa fa-check"></i>Learning Goals</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="c-moor-overview.html"><a href="c-moor-overview.html"><i class="fa fa-check"></i><b>1</b> C-MOOR Overview</a>
<ul>
<li class="chapter" data-level="1.0.1" data-path="c-moor-overview.html"><a href="c-moor-overview.html#about-c-moor"><i class="fa fa-check"></i><b>1.0.1</b> About C-MOOR</a></li>
<li class="chapter" data-level="1.0.2" data-path="c-moor-overview.html"><a href="c-moor-overview.html#why-data-science"><i class="fa fa-check"></i><b>1.0.2</b> Why data science?</a></li>
<li class="chapter" data-level="1.0.3" data-path="c-moor-overview.html"><a href="c-moor-overview.html#learning-goals-1"><i class="fa fa-check"></i><b>1.0.3</b> Learning Goals</a></li>
</ul></li>
<li class="part"><span><b>INSTRUCTOR GUIDE</b></span></li>
<li class="chapter" data-level="2" data-path="for-new-instructors.html"><a href="for-new-instructors.html"><i class="fa fa-check"></i><b>2</b> For new instructors</a>
<ul>
<li class="chapter" data-level="2.0.1" data-path="for-new-instructors.html"><a href="for-new-instructors.html#course-based-undergraduate-research-experiences"><i class="fa fa-check"></i><b>2.0.1</b> Course-based undergraduate research experiences</a></li>
<li class="chapter" data-level="2.0.2" data-path="for-new-instructors.html"><a href="for-new-instructors.html#how-c-moor-supports-you"><i class="fa fa-check"></i><b>2.0.2</b> How C-MOOR supports you</a></li>
<li class="chapter" data-level="2.0.3" data-path="for-new-instructors.html"><a href="for-new-instructors.html#new-instructor-to-do-list"><i class="fa fa-check"></i><b>2.0.3</b> New instructor to-do list</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html"><i class="fa fa-check"></i><b>3</b> Human gut 16S rRNA</a>
<ul>
<li class="chapter" data-level="3.0.1" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#content-overview"><i class="fa fa-check"></i><b>3.0.1</b> Content overview</a></li>
<li class="chapter" data-level="3.0.2" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#sample-schedules"><i class="fa fa-check"></i><b>3.0.2</b> Sample Schedules</a></li>
<li class="chapter" data-level="3.0.3" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#what-to-know"><i class="fa fa-check"></i><b>3.0.3</b> What to know</a></li>
<li class="chapter" data-level="3.0.4" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#where-to-get-help"><i class="fa fa-check"></i><b>3.0.4</b> Where to get help</a></li>
<li class="chapter" data-level="3.0.5" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#grades"><i class="fa fa-check"></i><b>3.0.5</b> Grades</a></li>
<li class="chapter" data-level="3.0.6" data-path="human-gut-16s-rrna.html"><a href="human-gut-16s-rrna.html#footnotes"><i class="fa fa-check"></i><b>3.0.6</b> Footnotes</a></li>
</ul></li>
<li class="part"><span><b>Setting up on a cloud platform</b></span></li>
<li class="chapter" data-level="4" data-path="anvil.html"><a href="anvil.html"><i class="fa fa-check"></i><b>4</b> AnVIL</a>
<ul>
<li class="chapter" data-level="4.1" data-path="about-anvil.html"><a href="about-anvil.html"><i class="fa fa-check"></i><b>4.1</b> About AnVIL</a></li>
<li class="chapter" data-level="4.2" data-path="sign-up-for-an-anvil-account.html"><a href="sign-up-for-an-anvil-account.html"><i class="fa fa-check"></i><b>4.2</b> Sign up for an AnVIL account</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="sign-up-for-an-anvil-account.html"><a href="sign-up-for-an-anvil-account.html#create-an-anvil-account"><i class="fa fa-check"></i><b>4.2.1</b> Create an AnVIL account</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="setting-up-workspaces-on-anvil.html"><a href="setting-up-workspaces-on-anvil.html"><i class="fa fa-check"></i><b>4.3</b> Setting up workspaces on AnVIL</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="setting-up-workspaces-on-anvil.html"><a href="setting-up-workspaces-on-anvil.html#a-note-to-instructors-on-billing"><i class="fa fa-check"></i><b>4.3.1</b> A note to instructors on billing</a></li>
<li class="chapter" data-level="4.3.2" data-path="setting-up-workspaces-on-anvil.html"><a href="setting-up-workspaces-on-anvil.html#using-groups-to-manage-classes"><i class="fa fa-check"></i><b>4.3.2</b> Using groups to manage classes</a></li>
<li class="chapter" data-level="4.3.3" data-path="setting-up-workspaces-on-anvil.html"><a href="setting-up-workspaces-on-anvil.html#clone-a-workspace-on-anvil"><i class="fa fa-check"></i><b>4.3.3</b> Clone a workspace on AnVIL</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html"><i class="fa fa-check"></i><b>4.4</b> Running a module on AnVIL</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#what-is-a-workspace"><i class="fa fa-check"></i><b>4.4.1</b> What is a workspace?</a></li>
<li class="chapter" data-level="4.4.2" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#running-modules-on-anvil"><i class="fa fa-check"></i><b>4.4.2</b> Running modules on AnVIL</a></li>
<li class="chapter" data-level="4.4.3" data-path="running-a-module-on-anvil.html"><a href="running-a-module-on-anvil.html#closing-out-a-session-on-anvil"><i class="fa fa-check"></i><b>4.4.3</b> Closing out a session on AnVIL</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Scientific Literature</b></span></li>
<li class="chapter" data-level="5" data-path="scientific-literature.html"><a href="scientific-literature.html"><i class="fa fa-check"></i><b>5</b> Scientific Literature</a>
<ul>
<li class="chapter" data-level="5.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html"><i class="fa fa-check"></i><b>5.1</b> Pre-lab - Scientific literature</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#purpose-3"><i class="fa fa-check"></i><b>5.1.1</b> Purpose</a></li>
<li class="chapter" data-level="5.1.2" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#grading-criteria"><i class="fa fa-check"></i><b>5.1.2</b> Grading criteria</a></li>
<li class="chapter" data-level="5.1.3" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#activity"><i class="fa fa-check"></i><b>5.1.3</b> Activity</a></li>
<li class="chapter" data-level="5.1.4" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#footnotes-1"><i class="fa fa-check"></i><b>5.1.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="lecture---reading-a-paper.html"><a href="lecture---reading-a-paper.html"><i class="fa fa-check"></i><b>5.2</b> Lecture - Reading a paper</a></li>
<li class="chapter" data-level="5.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html"><i class="fa fa-check"></i><b>5.3</b> Activity - Scientific Literature</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#purpose-4"><i class="fa fa-check"></i><b>5.3.1</b> Purpose</a></li>
<li class="chapter" data-level="5.3.2" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#learning-objectives-3"><i class="fa fa-check"></i><b>5.3.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.3.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#introduction"><i class="fa fa-check"></i><b>5.3.3</b> Introduction</a></li>
<li class="chapter" data-level="5.3.4" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#instructions-1"><i class="fa fa-check"></i><b>5.3.4</b> Instructions</a></li>
<li class="chapter" data-level="5.3.5" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-1---overview-of-the-paper"><i class="fa fa-check"></i><b>5.3.5</b> Activity 1 - Overview of the paper</a></li>
<li class="chapter" data-level="5.3.6" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-2---methods-and-figure-analysis-in-class-and-homework"><i class="fa fa-check"></i><b>5.3.6</b> Activity 2 - Methods and Figure Analysis (in class and homework)</a></li>
<li class="chapter" data-level="5.3.7" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#grading-criteria-1"><i class="fa fa-check"></i><b>5.3.7</b> Grading criteria</a></li>
<li class="chapter" data-level="5.3.8" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#footnotes-2"><i class="fa fa-check"></i><b>5.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Project: 16S rRNA Research</b></span></li>
<li class="chapter" data-level="6" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html"><i class="fa fa-check"></i><b>6</b> Reference - Meet the MISO study</a>
<ul>
<li class="chapter" data-level="6.0.1" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#miso-study-diet-and-sampling-schedule"><i class="fa fa-check"></i><b>6.0.1</b> MISO study diet and sampling schedule</a></li>
<li class="chapter" data-level="6.0.2" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#miso-study-variables-and-factors"><i class="fa fa-check"></i><b>6.0.2</b> MISO study variables and factors</a></li>
<li class="chapter" data-level="6.0.3" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#amplicon-sequence-variants-asvs-data"><i class="fa fa-check"></i><b>6.0.3</b> Amplicon sequence variants (ASVs) data</a></li>
<li class="chapter" data-level="6.0.4" data-path="reference---meet-the-miso-study.html"><a href="reference---meet-the-miso-study.html#footnotes-3"><i class="fa fa-check"></i><b>6.0.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="s-rrna-data-in-spreadsheets.html"><a href="s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>7</b> 16S rRNA Data in spreadsheets</a>
<ul>
<li class="chapter" data-level="7.1" data-path="exploring-16s-rrna-data-in-spreadsheets.html"><a href="exploring-16s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>7.1</b> Exploring 16S rRNA data in spreadsheets</a></li>
<li class="chapter" data-level="7.2" data-path="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><a href="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><i class="fa fa-check"></i><b>7.2</b> Lecture - Introducing 16S rRNA Data Using Spreadsheets</a></li>
<li class="chapter" data-level="7.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html"><i class="fa fa-check"></i><b>7.3</b> Activity - Introducing 16S rRNA Data Using Spreadsheets</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#purpose-5"><i class="fa fa-check"></i><b>7.3.1</b> Purpose</a></li>
<li class="chapter" data-level="7.3.2" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#learning-objectives-4"><i class="fa fa-check"></i><b>7.3.2</b> Learning objectives</a></li>
<li class="chapter" data-level="7.3.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#introduction-1"><i class="fa fa-check"></i><b>7.3.3</b> Introduction</a></li>
<li class="chapter" data-level="7.3.4" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-1-quick-data-overview-based-on-1-sample"><i class="fa fa-check"></i><b>7.3.4</b> Activity 1 – Quick data overview based on 1 sample</a></li>
<li class="chapter" data-level="7.3.5" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-2-explore-variation-between-individuals"><i class="fa fa-check"></i><b>7.3.5</b> Activity 2 – Explore variation between individuals</a></li>
<li class="chapter" data-level="7.3.6" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-2-3.-plot-individual-variation-in-abundance-of-the-most-common-asv."><i class="fa fa-check"></i><b>7.3.6</b> Activity 2-3. Plot individual variation in abundance of the most common ASV.</a></li>
<li class="chapter" data-level="7.3.7" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-3-explore-variation-within-a-single-individual-associated-with-diet"><i class="fa fa-check"></i><b>7.3.7</b> Activity 3 – Explore variation within a single individual associated with diet</a></li>
<li class="chapter" data-level="7.3.8" data-path="activity---introducing-16s-rrna-data-using-spreadsheets.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#footnotes-4"><i class="fa fa-check"></i><b>7.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="exploring-16s-rrna-data-with-phyloseq.html"><a href="exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>8</b> Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="8.1" data-path="exploring-16s-rrna-data-with-phyloseq-1.html"><a href="exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>8.1</b> Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="8.2" data-path="lecture---exploring-16s-rrna-data-with-phyloseq.html"><a href="lecture---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>8.2</b> Lecture - Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="8.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>8.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="8.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#learning-objectives-5"><i class="fa fa-check"></i><b>8.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="8.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#introduction-2"><i class="fa fa-check"></i><b>8.3.2</b> Introduction</a></li>
<li class="chapter" data-level="8.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-1-explore-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>8.3.3</b> Activity 1 – Explore 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="8.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-2-try-it-out-questions"><i class="fa fa-check"></i><b>8.3.4</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="8.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#grading-criteria-2"><i class="fa fa-check"></i><b>8.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="8.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#footnotes-5"><i class="fa fa-check"></i><b>8.3.6</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><i class="fa fa-check"></i><b>8.4</b> Try it Question 1 - What has more impact on human microbial variation, diet or individuality</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1a-plot-high-level-taxonomy-phylum-level-for-each-timepoint-using-the-plot_bar-function.-specifically-your-plot-should-show-phyla-diversity-across-all-5-timepoints."><i class="fa fa-check"></i><b>8.4.1</b> Step 1A: Plot high-level taxonomy (Phylum level) for each timepoint using the plot_bar() function.** Specifically, your plot should show Phyla diversity across all 5 timepoints.</a></li>
<li class="chapter" data-level="8.4.2" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1b-plot-high-level-taxonomy-phylum-level-for-each-individual-using-only-normalized-homogenized-diet-hd-timepoints-to-minimize-contribution-of-diet-to-the-analysis-using-the-plot_bar-function."><i class="fa fa-check"></i><b>8.4.2</b> Step 1B: Plot high-level taxonomy (Phylum level) for each individual, using only normalized homogenized diet “HD” timepoints to minimize contribution of diet to the analysis using the plot_bar() function.</a></li>
</ul></li>
<li class="chapter" data-level="8.5" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><i class="fa fa-check"></i><b>8.5</b> Try it Question 2 - What are some of the most abundant microbes when viewed at different taxonomic resolutions</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2a.-plot-phyla-across-all-subjects-and-identify-the-most-abundant-phyla.-specifically-your-plot-should-show-phyla-diversity-across-all-21-individuals."><i class="fa fa-check"></i><b>8.5.1</b> Step 2A. Plot Phyla across all subjects and identify the most abundant Phyla. Specifically, your plot should show Phyla diversity across all 21 individuals.</a></li>
<li class="chapter" data-level="8.5.2" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2b.-subset-the-most-abundant-phyla-and-plot-order-skip-class"><i class="fa fa-check"></i><b>8.5.2</b> Step 2B. Subset the most abundant Phyla, and plot Order (Skip Class)</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>8.6</b> Try it Question 3 - Does gender contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="8.6.1" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3c.-test-gender-based-difference-in-species-for-the-low-abundant-phylum-you-selected.-specifically-your-plot-should-show-species-diversity-within-male-and-female-groups-for-only-your-phylum-of-choice."><i class="fa fa-check"></i><b>8.6.1</b> Step 3C. Test gender-based difference in Species for the low abundant phylum you selected.** Specifically, your plot should show species diversity within male and female groups for only your phylum of choice.</a></li>
<li class="chapter" data-level="8.6.2" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3d-plot-the-individual-abundance-of-2-candidate-microbes-for-gender-differences-based-on-your-analysis-above-andor-the-curated-menu-of-microbes-below-previously-associated-with-gender-variation-with-color-denoting-gender."><i class="fa fa-check"></i><b>8.6.2</b> Step 3D: Plot the individual abundance of 2 candidate microbes for gender differences based on your analysis above, and/or the curated menu of microbes below (previously associated with gender variation), with color denoting gender.</a></li>
</ul></li>
<li class="chapter" data-level="8.7" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>8.7</b> Try it Question 4 - Does age contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="8.7.1" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4a.-perform-an-initial-survey-to-see-if-there-are-any-differences-in-phyla-related-to-age."><i class="fa fa-check"></i><b>8.7.1</b> Step 4A. Perform an initial survey to see if there are any differences in phyla related to age.</a></li>
<li class="chapter" data-level="8.7.2" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4b.-merge-samples-by-age-then-re-normalize-and-finally-re-plot-phyla-by-age."><i class="fa fa-check"></i><b>8.7.2</b> Step 4B. Merge samples by age, then re-normalize and finally re-plot Phyla by age.</a></li>
<li class="chapter" data-level="8.7.3" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4c.-candidate-phyla-implicated-in-age-variation-include-firmicutes-and-bacteroidetes.-plot-these-specific-phyla-on-their-own-separately-to-observe-any-shift-that-occurs-with-age."><i class="fa fa-check"></i><b>8.7.3</b> Step 4C. Candidate Phyla implicated in age variation include Firmicutes and Bacteroidetes. Plot these specific phyla on their own, separately, to observe any shift that occurs with age.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="analyzing-16s-rrna-data-with-phyloseq.html"><a href="analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>9</b> Analyzing 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="9.1" data-path="analyzing-16s-rrna-data-with-phyloseq-1.html"><a href="analyzing-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>9.1</b> Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="9.2" data-path="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><a href="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>9.2</b> Lecture - Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="9.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>9.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#purpose-7"><i class="fa fa-check"></i><b>9.3.1</b> Purpose</a></li>
<li class="chapter" data-level="9.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#learning-objectives-6"><i class="fa fa-check"></i><b>9.3.2</b> Learning objectives</a></li>
<li class="chapter" data-level="9.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#introduction-3"><i class="fa fa-check"></i><b>9.3.3</b> Introduction</a></li>
<li class="chapter" data-level="9.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-1-analyze-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>9.3.4</b> Activity 1 – Analyze 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="9.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-2-try-it-out-questions-1"><i class="fa fa-check"></i><b>9.3.5</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="9.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#grading-criteria-3"><i class="fa fa-check"></i><b>9.3.6</b> Grading criteria</a></li>
<li class="chapter" data-level="9.3.7" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#footnotes-6"><i class="fa fa-check"></i><b>9.3.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><i class="fa fa-check"></i><b>9.4</b> Try it Question 1 - How sensitive is microbial diversity to variables like diet, age and gender</a>
<ul>
<li class="chapter" data-level="9.4.1" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1a.-plot-alpha-diversity-of-the-full-miso-dataset-by-subject-individuals."><i class="fa fa-check"></i><b>9.4.1</b> Step 1A. Plot alpha diversity of the full MISO dataset by subject (individuals).</a></li>
<li class="chapter" data-level="9.4.2" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1b.-plot-alpha-diversity-of-the-full-miso-dataset-by-diet"><i class="fa fa-check"></i><b>9.4.2</b> Step 1B. Plot alpha diversity of the full MISO dataset by diet</a></li>
<li class="chapter" data-level="9.4.3" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1c.-plot-alpha-diversity-of-the-full-miso-dataset-by-age."><i class="fa fa-check"></i><b>9.4.3</b> Step 1C. Plot alpha diversity of the full MISO dataset by age.</a></li>
<li class="chapter" data-level="9.4.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1d.-plot-alpha-diversity-of-the-full-miso-dataset-by-gender."><i class="fa fa-check"></i><b>9.4.4</b> Step 1D. Plot alpha diversity of the full MISO dataset by gender.</a></li>
<li class="chapter" data-level="9.4.5" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-2.-plot-alpha-diversity-of-the-full-miso-dataset-grouping-based-on-levels-for-5-different-metabolites-creatinine-pcs-is-hipp-pag."><i class="fa fa-check"></i><b>9.4.5</b> Step 2. Plot alpha diversity of the full MISO dataset, grouping based on levels for 5 different metabolites: Creatinine, PCS, IS, HIPP, PAG.</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><i class="fa fa-check"></i><b>9.5</b> Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-1.-make-a-pcoa-plot-ordinating-on-the-entire-miso-dataset-and-coloring-by-individual.-investigate-resulting-plot-shape."><i class="fa fa-check"></i><b>9.5.1</b> Step 1. Make a PcOA plot, ordinating on the entire miso dataset, and coloring by individual. Investigate resulting plot shape.</a></li>
<li class="chapter" data-level="9.5.2" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-2.-using-the-pcoa-plot-from-step-1-above-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-diet-gender-and-age"><i class="fa fa-check"></i><b>9.5.2</b> Step 2. Using the PCoA plot from Step 1 above, look for any correlations between the shape of your PCoA plot and diet, gender and age</a></li>
<li class="chapter" data-level="9.5.3" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-3.-using-the-pcoa-plot-from-step-1-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-the-level-of-metabolites."><i class="fa fa-check"></i><b>9.5.3</b> Step 3. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and the level of metabolites.</a></li>
<li class="chapter" data-level="9.5.4" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-4.-one-way-of-checking-if-the-metabolite-levels-indeed-correlate-with-your-data-is-to-see-if-subsetting-smaller-chunks-or-specific-chunks-of-the-data-will-still-maintain-the-metabolite-data-relationship-or-break-it."><i class="fa fa-check"></i><b>9.5.4</b> Step 4. One way of checking if the metabolite levels indeed correlate with your data, is to see if subsetting smaller chunks or specific chunks of the data will still maintain the metabolite-data relationship or break it.</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><i class="fa fa-check"></i><b>9.6</b> Try it Question 3 - What microbes (ASVs) differ between males and females, and does age have an impact</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3a.-perform-differential-abundance-analysis-between-genders."><i class="fa fa-check"></i><b>9.6.1</b> Step 3A. Perform differential abundance analysis between genders.</a></li>
<li class="chapter" data-level="9.6.2" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3b.-determine-the-impact-of-younger-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>9.6.2</b> Step 3B. Determine the impact of younger age on differential abundance in males and females, by subsetting age &lt; 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
<li class="chapter" data-level="9.6.3" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3c.-determine-the-impact-of-older-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>9.6.3</b> Step 3C. Determine the impact of older age on differential abundance in males and females, by subsetting age &gt;= 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
</ul></li>
<li class="chapter" data-level="9.7" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><i class="fa fa-check"></i><b>9.7</b> Try it Question 4 - Is there an interaction between diet and age and the microbiome</a>
<ul>
<li class="chapter" data-level="9.7.1" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-1.-plot-alpha-diversity-based-on-age-for-all-dietary-groups-and-then-for-bd-hd-and-wo."><i class="fa fa-check"></i><b>9.7.1</b> Step 1. Plot alpha diversity based on age for all dietary groups, and then for BD, HD and WO.</a></li>
<li class="chapter" data-level="9.7.2" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-2.-perform-differential-abundance-analysis-comparing-hd-to-bd-for-the-population-in-general-and-then-for-older-individuals-specifically."><i class="fa fa-check"></i><b>9.7.2</b> Step 2. Perform differential abundance analysis comparing HD to BD for the population in general, and then for older individuals specifically.</a></li>
<li class="chapter" data-level="9.7.3" data-path="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html"><a href="try-it-question-4---is-there-an-interaction-between-diet-and-age-and-the-microbiome.html#step-2b.-perform-differential-abundance-analysis-between-hd-and-bd-for-older-individual-50-yo."><i class="fa fa-check"></i><b>9.7.3</b> Step 2B. Perform differential abundance analysis between HD and BD for older individual (&gt; 50 yo.)</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Community analysis and feedback</b></span></li>
<li class="chapter" data-level="10" data-path="community-analysis-and-feedback.html"><a href="community-analysis-and-feedback.html"><i class="fa fa-check"></i><b>10</b> Community analysis and feedback</a>
<ul>
<li class="chapter" data-level="10.1" data-path="lecture---scientific-posters.html"><a href="lecture---scientific-posters.html"><i class="fa fa-check"></i><b>10.1</b> Lecture - Scientific Posters</a></li>
<li class="chapter" data-level="10.2" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html"><i class="fa fa-check"></i><b>10.2</b> Activity - Scientific Posters</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html#activity-1"><i class="fa fa-check"></i><b>10.2.1</b> Activity</a></li>
<li class="chapter" data-level="10.2.2" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html#grading-criteria-4"><i class="fa fa-check"></i><b>10.2.2</b> Grading Criteria</a></li>
<li class="chapter" data-level="10.2.3" data-path="activity---scientific-posters.html"><a href="activity---scientific-posters.html#footnotes-7"><i class="fa fa-check"></i><b>10.2.3</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html"><i class="fa fa-check"></i><b>10.3</b> Activity - Share Your Poster</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#introduction-4"><i class="fa fa-check"></i><b>10.3.1</b> Introduction</a></li>
<li class="chapter" data-level="10.3.2" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#activity-2-present-your-poster"><i class="fa fa-check"></i><b>10.3.2</b> Activity 2 – Present Your Poster</a></li>
<li class="chapter" data-level="10.3.3" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#grading-criteria-5"><i class="fa fa-check"></i><b>10.3.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="10.3.4" data-path="activity---share-your-poster.html"><a href="activity---share-your-poster.html#footnotes-8"><i class="fa fa-check"></i><b>10.3.4</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="11" data-path="professional-development.html"><a href="professional-development.html"><i class="fa fa-check"></i><b>11</b> Professional Development</a>
<ul>
<li class="chapter" data-level="11.1" data-path="lecture---next-steps.html"><a href="lecture---next-steps.html"><i class="fa fa-check"></i><b>11.1</b> Lecture - Next Steps</a></li>
<li class="chapter" data-level="11.2" data-path="activity---next-steps.html"><a href="activity---next-steps.html"><i class="fa fa-check"></i><b>11.2</b> Activity - Next Steps</a>
<ul>
<li class="chapter" data-level="11.2.1" data-path="activity---next-steps.html"><a href="activity---next-steps.html#introduction-5"><i class="fa fa-check"></i><b>11.2.1</b> Introduction</a></li>
<li class="chapter" data-level="11.2.2" data-path="activity---next-steps.html"><a href="activity---next-steps.html#activity-1-complete-feedback"><i class="fa fa-check"></i><b>11.2.2</b> Activity 1 – Complete Feedback</a></li>
<li class="chapter" data-level="11.2.3" data-path="activity---next-steps.html"><a href="activity---next-steps.html#activity-2-research-opportunities"><i class="fa fa-check"></i><b>11.2.3</b> Activity 2 – Research Opportunities</a></li>
<li class="chapter" data-level="11.2.4" data-path="activity---next-steps.html"><a href="activity---next-steps.html#activity-3-explore-communities"><i class="fa fa-check"></i><b>11.2.4</b> Activity 3 – Explore Communities</a></li>
<li class="chapter" data-level="11.2.5" data-path="activity---next-steps.html"><a href="activity---next-steps.html#optional-activity-4-share-work"><i class="fa fa-check"></i><b>11.2.5</b> (Optional) Activity 4 – Share Work</a></li>
<li class="chapter" data-level="11.2.6" data-path="activity---next-steps.html"><a href="activity---next-steps.html#grading-criteria-6"><i class="fa fa-check"></i><b>11.2.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="11.2.7" data-path="activity---next-steps.html"><a href="activity---next-steps.html#footnotes-9"><i class="fa fa-check"></i><b>11.2.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="11.3" data-path="science-talks.html"><a href="science-talks.html"><i class="fa fa-check"></i><b>11.3</b> Science Talks</a>
<ul>
<li class="chapter" data-level="11.3.1" data-path="science-talks.html"><a href="science-talks.html#science-talks---activity-1"><i class="fa fa-check"></i><b>11.3.1</b> Science Talks - Activity 1</a></li>
<li class="chapter" data-level="11.3.2" data-path="science-talks.html"><a href="science-talks.html#science-talks---activity-2"><i class="fa fa-check"></i><b>11.3.2</b> Science Talks - Activity 2</a></li>
</ul></li>
<li class="chapter" data-level="11.4" data-path="create-your-cv.html"><a href="create-your-cv.html"><i class="fa fa-check"></i><b>11.4</b> Create your CV</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="c-moor-scholars.html"><a href="c-moor-scholars.html"><i class="fa fa-check"></i><b>12</b> C-MOOR Scholars</a>
<ul>
<li class="chapter" data-level="12.1" data-path="c-moor-scholars-1.html"><a href="c-moor-scholars-1.html"><i class="fa fa-check"></i><b>12.1</b> C-MOOR Scholars</a>
<ul>
<li class="chapter" data-level="12.1.1" data-path="c-moor-scholars-1.html"><a href="c-moor-scholars-1.html#c-moor-scholars-links"><i class="fa fa-check"></i><b>12.1.1</b> C-MOOR Scholars links</a></li>
</ul></li>
<li class="chapter" data-level="12.2" data-path="pursue-further-research.html"><a href="pursue-further-research.html"><i class="fa fa-check"></i><b>12.2</b> Pursue further research</a>
<ul>
<li class="chapter" data-level="12.2.1" data-path="pursue-further-research.html"><a href="pursue-further-research.html#research-opportunities-at-your-institution"><i class="fa fa-check"></i><b>12.2.1</b> Research opportunities at your institution</a></li>
<li class="chapter" data-level="12.2.2" data-path="pursue-further-research.html"><a href="pursue-further-research.html#find-a-research-lab-to-join"><i class="fa fa-check"></i><b>12.2.2</b> Find a research lab to join</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="biodigs.html"><a href="biodigs.html"><i class="fa fa-check"></i><b>12.3</b> BioDIGS</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="online-community.html"><a href="online-community.html"><i class="fa fa-check"></i><b>13</b> Online Community</a></li>
<li class="part"><span><b>Resources</b></span></li>
<li class="chapter" data-level="14" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html"><i class="fa fa-check"></i><b>14</b> Spreadsheets 101</a>
<ul>
<li class="chapter" data-level="14.0.1" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#getting-a-copy-of-the-spreadsheet"><i class="fa fa-check"></i><b>14.0.1</b> Getting a copy of the spreadsheet</a></li>
<li class="chapter" data-level="14.0.2" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#basic-spreadsheet-terms"><i class="fa fa-check"></i><b>14.0.2</b> Basic spreadsheet terms</a></li>
<li class="chapter" data-level="14.0.3" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#summary-statistics-by-row-or-column"><i class="fa fa-check"></i><b>14.0.3</b> Summary statistics by row or column</a></li>
<li class="chapter" data-level="14.0.4" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#adding-data-to-cells"><i class="fa fa-check"></i><b>14.0.4</b> Adding data to cells</a></li>
<li class="chapter" data-level="14.0.5" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#applying-a-calculation-to-many-cells"><i class="fa fa-check"></i><b>14.0.5</b> Applying a calculation to many cells</a></li>
<li class="chapter" data-level="14.0.6" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#manipulating-the-spreadsheet"><i class="fa fa-check"></i><b>14.0.6</b> Manipulating the spreadsheet</a></li>
<li class="chapter" data-level="14.0.7" data-path="spreadsheets-101.html"><a href="spreadsheets-101.html#useful-formulas-in-google-sheets"><i class="fa fa-check"></i><b>14.0.7</b> Useful formulas in Google Sheets</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="figure-and-tables-guide.html"><a href="figure-and-tables-guide.html"><i class="fa fa-check"></i><b>15</b> Figure and tables guide</a>
<ul>
<li class="chapter" data-level="15.0.1" data-path="figure-and-tables-guide.html"><a href="figure-and-tables-guide.html#figures"><i class="fa fa-check"></i><b>15.0.1</b> Figures</a></li>
<li class="chapter" data-level="15.0.2" data-path="figure-and-tables-guide.html"><a href="figure-and-tables-guide.html#tables"><i class="fa fa-check"></i><b>15.0.2</b> Tables</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="s-minicure-guide.html"><a href="s-minicure-guide.html"><i class="fa fa-check"></i><b>16</b> 16S miniCURE Guide</a>
<ul>
<li class="chapter" data-level="16.0.1" data-path="s-minicure-guide.html"><a href="s-minicure-guide.html#footnotes-10"><i class="fa fa-check"></i><b>16.0.1</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">miniCURE: Human Gut 16S rRNA</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=10.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/C-MOOR_Template/dasl_thin_main_image.png">
</div>
<div id="activity---introducing-16s-rrna-data-using-spreadsheets" class="section level2 hasAnchor" number="7.3">
<h2><span class="header-section-number">7.3</span> Activity - Introducing 16S rRNA Data Using Spreadsheets<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity---introducing-16s-rrna-data-using-spreadsheets" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="purpose-5" class="section level3 hasAnchor" number="7.3.1">
<h3><span class="header-section-number">7.3.1</span> Purpose<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#purpose-5" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To explore bacterial diversity based on 16S rRNA gene sequencing using data from the “Impact of a 7-day homogeneous diet on interpersonal variation in human gut microbiomes and metabolomes” by Guthrie et al., 2023. This study is also referred to as the MISO study for “Microbiome Individuality and Stability Over Time” because the study aims to understand variation (or stability) of microbiomes in individuals.</p>
<p>This spreadsheet activity aims to orient you towards understanding the data and metadata content of this study. In subsequent activities, you will explore and analyze this data using the R/Bioconductor package phyloseq. Google Sheets and R are both useful for doing different analyses, so keep in mind the tools at your disposal and what will be the most efficient way to answer your research questions.</p>
<p><strong>Please note that this activity takes place in Google Sheets.</strong> It is possible to do the same analysis in Microsoft Excel although the individual steps you will take are different. To help make things easier for everyone, we strongly recommend everyone uses Google Sheets; all our instructions will be specific to Sheets. Students who do not have a Google-associated email address can sign up for a free Gmail account and gain access to Google Sheets this way.</p>
<p><strong>A Google Sheets cheatsheet that covers the necessary steps for this activity is available.</strong> Students who are not experienced using spreadsheet software should note that the instructions build on prior steps.</p>
</div>
<div id="learning-objectives-4" class="section level3 hasAnchor" number="7.3.2">
<h3><span class="header-section-number">7.3.2</span> Learning objectives<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#learning-objectives-4" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li>Explore Amplicon Sequence Variants (ASVs) through the taxonomy profile of select ASVs, and sample metadata for a subset of ASVs.</li>
<li>Ask research questions based on data and metadata</li>
</ol>
<ul>
<li>How many different microbes (ASVs) are there in a given sample?</li>
<li>Is there an even abundance of microbes (ASVs) in a sample or are there a few that dominate?</li>
<li>How varied are ASVs and their abundance across subjects?</li>
<li>How varied are ASVs and their abundance across diets?</li>
</ul>
</div>
<div id="introduction-1" class="section level3 hasAnchor" number="7.3.3">
<h3><span class="header-section-number">7.3.3</span> Introduction<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#introduction-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The most popular sequencing technique for the analysis of bacterial diversity is targeted sequencing, or sequencing of a specific gene (or region of a gene, e.g. a hypervariable region of the bacterial 16S ribosomal rRNA gene) using Polymerase Chain Reaction (PCR) to create sequences called amplicons. Sequence variation in the resulting amplicons creates Amplicon Sequence Variants (ASVs). ASVs varying from as little as one single nucleotide are defined as separate ASVs and as little as 1% difference in ASV sequence can be associated with different species. In this activity you will be exploring ASVs generated with sequencing 250 nucleotides (nt) Illumina sequencing, where Amplicon sequence variants (ASVs) were identified.</p>
</div>
<div id="activity-1-quick-data-overview-based-on-1-sample" class="section level3 hasAnchor" number="7.3.4">
<h3><span class="header-section-number">7.3.4</span> Activity 1 – Quick data overview based on 1 sample<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-1-quick-data-overview-based-on-1-sample" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 10 min</em></p>
<p>Although the full dataset contains 105 samples from 21 subjects and 5 timepoints (representing 3 dietary conditions), in this activity we will familiarize ourselves with the data by exploring data from 1 sample only.</p>
<div id="activity-1-1.-explore-data-contents-using-one-sample-only-namely-miso1-s02-1-subject-2-timepoint-1." class="section level4 hasAnchor" number="7.3.4.1">
<h4><span class="header-section-number">7.3.4.1</span> Activity 1-1. Explore data contents using one sample only, namely MISO1-S02-1 (subject 2, timepoint 1).<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-1-1.-explore-data-contents-using-one-sample-only-namely-miso1-s02-1-subject-2-timepoint-1." class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Access <a href="https://docs.google.com/spreadsheets/d/1hpSyjO0H8sXq6C1xM_oGMyv9vCY2qTvfYHVdqO9ZVhk/edit?usp=sharing">‘MISO1-S02-1’</a> file and open it with Google Sheets.</p>
<ul>
<li>Rows = ASVs</li>
<li>Columns = Samples and taxonomy:
<ul>
<li>Col A = ASV id</li>
<li>Cols B-H = Taxonomic information</li>
<li>Col I = ASV counts (for sample MISO1-SO2-1)</li>
</ul></li>
</ul>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>1-1.1 How many microbes (ASVs) are there?</strong> This is the # of rows there are in the spreadsheet. Scroll down to the bottom of the sheet and look at the row number or select column A and see the count of all the cells in a row at the bottom right of the window. Remember to not count the header row.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>1-1.2 How many microbes are there in this sample?</strong> In other words, what is the total number of ASVs per sample?</th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<ul>
<li><p>We can count cells with a specific condition using the following formula: =COUNTIF(CELLS, “&lt;&gt;0”), where CELLS is replaced by the range of cells you want to use in the calculation.</p></li>
<li><p>For example, for this sample (MISO1-S02-1) in cell J1 enter the formula =COUNTIF(I2:I2185, “&lt;&gt;0”). This formula says, count cells in the range between I2 and I2185 that are not equal to 0. Note that we do not include I1 in the calculation as that is the name of the sample name.
<br></p></li>
</ul>
<table>
<tbody>
<tr class="odd">
<td align="left"># Microbes:</td>
</tr>
<tr class="even">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>1-1.3 What taxonomy is associated with ASV1? Include all taxonomic ranks, even those that are listed as NA.</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Kingdom:</td>
</tr>
<tr class="even">
<td align="left">Phylum:</td>
</tr>
<tr class="odd">
<td align="left">Class:</td>
</tr>
<tr class="even">
<td align="left">Order:</td>
</tr>
<tr class="odd">
<td align="left">Family:</td>
</tr>
<tr class="even">
<td align="left">Genus:</td>
</tr>
<tr class="odd">
<td align="left">Species:</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>1-1.4 What taxonomy is associated with ASV2? Include all taxonomic ranks, even those that are listed as NA.</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Kingdom:</td>
</tr>
<tr class="even">
<td align="left">Phylum:</td>
</tr>
<tr class="odd">
<td align="left">Class:</td>
</tr>
<tr class="even">
<td align="left">Order:</td>
</tr>
<tr class="odd">
<td align="left">Family:</td>
</tr>
<tr class="even">
<td align="left">Genus:</td>
</tr>
<tr class="odd">
<td align="left">Species:</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left"><strong>1-1.5 For sample 1 (MISO1-S02-1) what is the count of ASV 1?</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left"><strong>1-1.6 For sample 1 (MISO1-S02-1) what is the count of ASV 2?</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>1-1.7 Are ASVs equally abundant throughout the sample? Plot the ASV count distribution (range of ASV abundance) for sample MISO1-S02-1.</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>A) Make a bar graph of ASV distribution.</strong> Your chart should have the same format as the example.</th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<ol style="list-style-type: decimal">
<li><p>To make a bar graph with this data, hold the Command key on a Mac or Ctrl on Windows to select data that are not immediately next to each other and then click on columns A (“ASV”, which contains ASV ids) and I ( “MISO1-S02-1”, which contains counts)</p></li>
<li><p>In the menu at the top of the screen, select Insert &gt; Chart.</p></li>
<li><p>Make sure your graph is a bar plot: click on the 3 dots in the top right corner of the plot → edit chart and in the Chart editor, under Chart type, selecting Column Chart.</p></li>
<li><p>Copy the bar graph into a box below: click on the 3 dots in the top right corner of the plot, select copy chart, then, paste it below.</p></li>
<li><p>Your graph should look similar to the example graph with the ASVs on the x-axis and counts on the y-axis (although the Y axis is titled the same name of the MISO sample, since this is the name of the column).</p></li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left">Insert graph here:</td>
</tr>
<tr class="even">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>B) What is the ASV with the highest abundance (count) in sample 1?</strong> To find this answer hover over the tallest bar in your ASV distribution plot from activity at its highest point.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>C) Describe the pattern of abundance in ASVs for this sample.</strong> Is the abundance of ASVs evenly distributed (left in the example graph) where each ASV is approximately the same abundance (count) or is the abundance of ASVs unevenly distributed (right in the example graph), where some ASVs have very high and/or very low counts?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="activity-2-explore-variation-between-individuals" class="section level3 hasAnchor" number="7.3.5">
<h3><span class="header-section-number">7.3.5</span> Activity 2 – Explore variation between individuals<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-2-explore-variation-between-individuals" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 10 min</em></p>
<p>One aspect of the Guthrie et al. study is the extent to which diet contributes to interpersonal microbiome variation (variation in microbiome between individuals). <strong>How does what you eat affect your gut microbiome?</strong></p>
<p><strong>To this end, authors standardized the diet for all individuals with the same diet to evaluate how much of an effect the diet has on the microbiome.</strong> If people all eat the same diet, will their gut microbiome become more similar to each other? The study period was 28 days long and had 3 dietary conditions:</p>
<ol style="list-style-type: decimal">
<li><strong>BD (Baseline diet)</strong>: which lasted 14 days (2 time points collected) and has participants eating what they usually eat.</li>
<li><strong>HD (Homogenized diet)</strong>: which lasted 7 days (2 time points collected), and has all participants on the same diet</li>
<li><strong>WO (washout diet)</strong>: which lasted for 7 days where participants choose their own diet again</li>
</ol>
<p><strong>To explore variation between individuals, here we simplify the dietary variation to a single condition, WO.</strong></p>
<ul>
<li><p>For this activity, the data file <a href="https://docs.google.com/spreadsheets/d/1vD0wbxgYuDlfKA0DF2Km-N4GHL2gRoUaEN_oCrSb2jU/edit?usp=sharing">MISO_WO</a> contains samples pre-filtered for WO condition only (and excludes samples corresponding to BD and HD).</p></li>
<li><p>Since there is only 1 WO datapoint per individual, each WO sample corresponds to a different individual, so you should see 21 samples.</p></li>
</ul>
<div id="activity-2-1---what-are-the-most-common-asvs-between-individuals" class="section level4 hasAnchor" number="7.3.5.1">
<h4><span class="header-section-number">7.3.5.1</span> Activity 2-1 - What are the most common ASVs between individuals?<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-2-1---what-are-the-most-common-asvs-between-individuals" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>IMPORTANT: An ASV is associated with an individual if it is NOT a zero. Zero means that particular ASV was not found in the individual, so you need to exclude zeroes from your calculations.</p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>A) Calculate the number of samples associated with each ASV. Record first 5 values below.</strong></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<ol style="list-style-type: decimal">
<li>Create an empty new first column for temporary calculations by right clicking column A and selecting “Insert 1 column left”.</li>
<li>In cell A1, label this new column “Occurrence”.</li>
<li>We will use the formula =COUNTIF(range, “&lt;&gt;0”) again to count how many cells have a non-0 value. For example, for ASV1 enter the formula = COUNTIF(C3:W3, “&lt;&gt;0”) into cell A3.</li>
<li>If the formula and its placement are correct, cell A3 should calculate to 17 (so 17 of 21 WO samples have ASV1). Confirm your calculation is working correctly.</li>
<li>Apply the calculation to all cells in the column by copy pasting the formula. For a reminder on how to do this, see the Google Sheets cheatsheet.</li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left">ASV1:</td>
</tr>
<tr class="even">
<td align="left">ASV2:</td>
</tr>
<tr class="odd">
<td align="left">ASV3:</td>
</tr>
<tr class="even">
<td align="left">ASV4:</td>
</tr>
<tr class="odd">
<td align="left">ASV5:</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>B) Calculate % of samples associated with each ASV and record the first top 5. record the first top 5.</strong></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<ol style="list-style-type: decimal">
<li>Create another new column by right clicking on column A and selecting “Insert 1 column left”.</li>
<li>In cell A1, give the new column a name, “% of samples with this ASV”.</li>
<li>Calculate % by dividing a value you obtained in part A) by 21 (samples) and multiplying by 100. To do this, starting with cell A3 (corresponding to ASV1), use the formula =100*(B3/21).</li>
<li>Apply the calculation to all cells in the column by copy pasting the formula. For a reminder on how to do this, see the Google Sheets cheatsheet.</li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left">ASV1:</td>
</tr>
<tr class="even">
<td align="left">ASV2:</td>
</tr>
<tr class="odd">
<td align="left">ASV3:</td>
</tr>
<tr class="even">
<td align="left">ASV4:</td>
</tr>
<tr class="odd">
<td align="left">ASV5:</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>C) How many ASVs are present in all 21 individuals and what are their ids?</strong></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<p>To do so, find the number of ASVs that have a % of samples with this ASV = 100%.</p>
<ol style="list-style-type: decimal">
<li>In cell A2 use the formula = COUNTIF(A3:A2186, “=100”). This formula says, count a cell if it is equal to 100.</li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
<div id="activity-2-2.-explore-the-taxonomy-of-the-most-common-asvs" class="section level4 hasAnchor" number="7.3.5.2">
<h4><span class="header-section-number">7.3.5.2</span> Activity 2-2. Explore the taxonomy of the most common ASVs<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-2-2.-explore-the-taxonomy-of-the-most-common-asvs" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>2-2.1 Which ASV is found in 100% of samples?</strong> This ASV has a % of samples with this ASV = 100%. Hint: This ASV is found within the first 20 ASVs.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>2-2.2 What is the taxonomy assigned to the most common ASV you found in Activity above?</strong></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<ol style="list-style-type: decimal">
<li>Return to your copy of <a href="https://docs.google.com/spreadsheets/d/1hpSyjO0H8sXq6C1xM_oGMyv9vCY2qTvfYHVdqO9ZVhk/edit?usp=sharing">‘MISO1-S02-1’</a>.</li>
<li>Columns B-H correspond to taxonomic information.</li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left">Kingdom:</td>
</tr>
<tr class="even">
<td align="left">Phylum:</td>
</tr>
<tr class="odd">
<td align="left">Class:</td>
</tr>
<tr class="even">
<td align="left">Order:</td>
</tr>
<tr class="odd">
<td align="left">Family:</td>
</tr>
<tr class="even">
<td align="left">Genus:</td>
</tr>
<tr class="odd">
<td align="left">Species:</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>2-2.3 This ASV was found in every subject. What might this mean about the function this ASV plays in the gut microbiome?</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="activity-2-3.-plot-individual-variation-in-abundance-of-the-most-common-asv." class="section level3 hasAnchor" number="7.3.6">
<h3><span class="header-section-number">7.3.6</span> Activity 2-3. Plot individual variation in abundance of the most common ASV.<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-2-3.-plot-individual-variation-in-abundance-of-the-most-common-asv." class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The most common ASV is represented in all samples, but does each sample have the same abundance of this ASV?</p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>A) Make a bar graph of most common ASV distribution across 21 individuals</strong></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<p>1.Return to your version of the <a href="https://docs.google.com/spreadsheets/d/1vD0wbxgYuDlfKA0DF2Km-N4GHL2gRoUaEN_oCrSb2jU/edit?usp=sharing">MISO_WO</a> spreadsheet.</p>
<ol style="list-style-type: decimal">
<li><p>Hold the Command key on a Mac or Ctrl on Windows to select data that are not immediately next to each other to select the header row (D2:X2, sample names) and the row corresponding to the most common ASV you identified above.</p></li>
<li><p>In the menu at the top of the screen, select Insert &gt; Chart.
Make sure your graph is a bar plot: click on the 3 dots in the top right corner of the plot → edit chart and in the Chart editor, under Chart type, selecting Column Chart.</p></li>
<li><p>Copy the bar graph into a box below: click on the 3 dots in the top right corner of the plot, select copy chart, then, paste it below. Your graph should look similar to the example graph below, with multicolored bars that each represent one sample’s count of the ASV. The example shows a different ASV plotted with 5 samples.</p></li>
<li><p>Your chart will have more bars and the legend may not be in the same place; do not worry if there is no legend or the legend text is not visible as in the example.</p></li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left">Insert chart here</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>B) Is the abundance of this ASV similar or dissimilar across samples?</strong> Each bar represents the count (abundance) of this ASV for a given subject. Ignore the axis title, which only has 1 sample name. Compare your graph to the example figure. Which is more like the graph you made?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>C) What might this mean for the function of this ASV in the gut microbiome?</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
<div id="activity-3-explore-variation-within-a-single-individual-associated-with-diet" class="section level3 hasAnchor" number="7.3.7">
<h3><span class="header-section-number">7.3.7</span> Activity 3 – Explore variation within a single individual associated with diet<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-3-explore-variation-within-a-single-individual-associated-with-diet" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 10 min</em></p>
<p>Although the full dataset contains 105 samples from 21 subjects and 5 timepoints (representing 3 dietary conditions), in this activity we will zoom into ASV counts data for 1 individual (comprised of 5 samples representing the 5 timepoints collected from that individual).</p>
<ul>
<li>Timepoints 1 and 2: <strong>BD</strong> (Baseline diet) prior to normalization with homogenized diet</li>
<li>Timepoints 3 and 4: <strong>HD</strong> (Homogenized diet): normalized to be same for all</li>
<li>Timepoint 5: <strong>WO</strong> (Washout diet), subjects can choose to eat what they want again</li>
</ul>
<div id="activity-3-1-explore-differences-between-dietary-changes-in-one-subject" class="section level4 hasAnchor" number="7.3.7.1">
<h4><span class="header-section-number">7.3.7.1</span> Activity 3-1 Explore differences between dietary changes in one subject<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-3-1-explore-differences-between-dietary-changes-in-one-subject" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>3-1.1 Calculate variation in ASV abundance within individual 1 (MISO1-S02) and record the first 5 values below.</strong></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<p>How varied is the ASV abundance across timepoints?</p>
<ul>
<li>This involves plotting counts for ASV 1-10 for MISO1-SO2 timepoints 1 through 5.</li>
<li>To calculate variation within a sample, we will use the VAR.S function in google sheets to calculate the variance of a sample. For more information on variance, see the Google Sheets cheatsheet.</li>
</ul>
<ol style="list-style-type: decimal">
<li>Access <a href="https://docs.google.com/spreadsheets/d/1SXPLeaHAbxjOBkdVsyldp-bdGG7fQgReKlh4sjG5AKI/edit?usp=sharing">‘MISO_Subject1’</a></li>
<li>Label an empty Column G by putting “Variance” in cell G1. Use this column to calculate variance.</li>
<li>Starting with the first ASV use the following formula: =VAR.S(range) to calculate ASV abundance variation in the 5 samples/timepoints collected for individual MISO1-SO2. For example, for ASV1 enter formula =VAR.S(B2:F2) in cell G2.</li>
<li>If imputed correctly, cell G2 should calculate to 14947547.8. Confirm this.</li>
<li>Apply the calculation to all cells in the column by copy pasting the formula. For a reminder on how to do this, see the Google Sheets cheatsheet.</li>
<li>Record the first 5 variance values below.</li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left">ASV1 Variance:</td>
</tr>
<tr class="even">
<td align="left">ASV2 Variance:</td>
</tr>
<tr class="odd">
<td align="left">ASV3 Variance:</td>
</tr>
<tr class="even">
<td align="left">ASV4 Variance:</td>
</tr>
<tr class="odd">
<td align="left">ASV5 Variance:</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left"><strong>3-1.2 Plot ASVs with highest variance.</strong></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<p>These are the ASVs whose abundance changed the most over the course of the study and could potentially depend heavily on diet. By plotting their counts, we will be able to see how the abundance of a given ASV changed throughout the experiment.</p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>A) Sort the ASVs by the variance column with the largest variance values at the top.</strong></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<ol style="list-style-type: decimal">
<li>Click on cell G2.</li>
<li>Go to Data &gt; Sort sheet &gt; Sort sheet by column G (Z to A)</li>
<li>Record the top 5 ASVs with the highest Variance and their Variance values below.</li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left">ASV# Variance:</td>
</tr>
<tr class="even">
<td align="left">ASV# Variance:</td>
</tr>
<tr class="odd">
<td align="left">ASV# Variance:</td>
</tr>
<tr class="even">
<td align="left">ASV# Variance:</td>
</tr>
<tr class="odd">
<td align="left">ASV# Variance:</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>B) After sorting ASVs based on high to low variance, make a plot of ASV counts corresponding to top 15 ASVs with the highest variance.</strong></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<ol style="list-style-type: decimal">
<li>To make a bar graph highlight top 16 rows corresponding to sample names and the 15 ASVs with highest variance, for samples MISO1-S02-1 through 5. Do not include the “Variance” column.</li>
<li>In the menu at the top of the screen, select Insert &gt; Chart.</li>
<li>Make sure your graph is a bar plot: click on the 3 dots in the top right corner of the plot → edit chart and in the Chart editor, under Chart type, selecting Column Chart.</li>
<li>Copy the bar graph into a box below: click on the 3 dots in the top right corner of the plot, select copy chart, then, paste it below. Your chart should look similar to the example chart with different ASVs</li>
</ol>
<table>
<tbody>
<tr class="odd">
<td align="left">Insert chart here</td>
</tr>
</tbody>
</table>
</div>
<div id="activity-3-2.-are-there-any-asvs-whose-abundance-changes-with-different-diets" class="section level4 hasAnchor" number="7.3.7.2">
<h4><span class="header-section-number">7.3.7.2</span> Activity 3-2. Are there any ASVs whose abundance changes with different diets?<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#activity-3-2.-are-there-any-asvs-whose-abundance-changes-with-different-diets" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Recall that there samples 1 and 2 are from the baseline diet (BD), samples 3 and 4 are on the uniform homogenized diet (HD) and sample 5 is from the washout diet.</p>
<ul>
<li>ASV_A in the example graph seems tightly linked to diet - timepoints from the same diet are similar to each other, and timepoints from different diets are different from each other.</li>
<li>ASV_B shows a pattern that doesn’t seem connected to diet. Samples from timepoints within the same diet are not similar, while samples from different diets are.</li>
</ul>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>A) List an ASV that changes with diet and one that changes seemingly without the influence of diet:</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">ASVs that seem to change in abundance based on diet:</td>
</tr>
<tr class="even">
<td align="left">ASVs that seem to change in abundance that do not appear connected to diet:</td>
</tr>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>B) What factors other than diet do you think might change the abundance of a microbe in the gut microbiome?</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="footnotes-4" class="section level3 hasAnchor" number="7.3.8">
<h3><span class="header-section-number">7.3.8</span> Footnotes<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#footnotes-4" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="resources-3" class="section level4 hasAnchor" number="7.3.8.1">
<h4><span class="header-section-number">7.3.8.1</span> Resources<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#resources-3" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li><a href="https://docs.google.com/document/d/1GC2cwwGAQEzMCta82RUQgQvqz9OOvFyCH4ht520BTGk/edit?usp=sharing">Google Doc</a></li>
<li><a href="https://docs.google.com/spreadsheets/d/1hpSyjO0H8sXq6C1xM_oGMyv9vCY2qTvfYHVdqO9ZVhk/edit?usp=sharing">‘MISO1-S02-1’</a> spreadsheet</li>
<li><a href="https://docs.google.com/spreadsheets/d/1vD0wbxgYuDlfKA0DF2Km-N4GHL2gRoUaEN_oCrSb2jU/edit?usp=sharing">MISO_WO</a> spreadsheet</li>
<li><a href="https://docs.google.com/spreadsheets/d/1SXPLeaHAbxjOBkdVsyldp-bdGG7fQgReKlh4sjG5AKI/edit?usp=sharing">‘MISO_Subject1’</a> spreadsheet</li>
</ul>
</div>
<div id="contributions-and-affiliations-3" class="section level4 hasAnchor" number="7.3.8.2">
<h4><span class="header-section-number">7.3.8.2</span> Contributions and affiliations<a href="activity---introducing-16s-rrna-data-using-spreadsheets.html#contributions-and-affiliations-3" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>Valeriya Gaysinskaya, Johns Hopkins University</li>
<li>Gauri Paul, Clovis Community College</li>
<li>Frederick Tan, Johns Hopkins University</li>
<li>Sayumi York, Notre Dame of Maryland University</li>
</ul>
<p>Last Revised: January 2026</p>

<!-- Set up code of OTTR Book-->
</div>
</div>
</div>
<!-- </div> -->
<hr>
<center> 
  <div class="footer">
      All materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="lecture---introducing-16s-rrna-data-using-spreadsheets.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="exploring-16s-rrna-data-with-phyloseq.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "section"
  }
});
});
</script>

</body>

</html>
