<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5.7 Try it Question 4 - Does age contribute to individual microbiome variation | Course Name</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="5.7 Try it Question 4 - Does age contribute to individual microbiome variation | Course Name" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5.7 Try it Question 4 - Does age contribute to individual microbiome variation | Course Name" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  



<meta name="date" content="2025-07-16" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/dasl_favicon.ico" type="image/x-icon" />
<link rel="prev" href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"/>
<link rel="next" href="analyzing-16s-rrna-data-with-phyloseq.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="assets/style.css" type="text/css" />
<link rel="stylesheet" href="assets/C-MOOR_Template/c-moor-ottr.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.c-moor.org/" target="_blank"><img src="assets/C-MOOR_Template/c-moor-logo-vertical.png" style="width: 70%; padding-left: 10px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#learning-goals"><i class="fa fa-check"></i>Learning Goals</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="motivation.html"><a href="motivation.html"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="target-audience.html"><a href="target-audience.html"><i class="fa fa-check"></i><b>1.2</b> Target Audience</a></li>
<li class="chapter" data-level="1.3" data-path="curriculum.html"><a href="curriculum.html"><i class="fa fa-check"></i><b>1.3</b> Curriculum</a></li>
</ul></li>
<li class="part"><span><b>Scientific Literature</b></span></li>
<li class="chapter" data-level="2" data-path="scientific-literature.html"><a href="scientific-literature.html"><i class="fa fa-check"></i><b>2</b> Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html"><i class="fa fa-check"></i><b>2.1</b> Pre-lab - Scientific literature</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#purpose"><i class="fa fa-check"></i><b>2.1.1</b> Purpose</a></li>
<li class="chapter" data-level="2.1.2" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#grading-criteria"><i class="fa fa-check"></i><b>2.1.2</b> Grading criteria</a></li>
<li class="chapter" data-level="2.1.3" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#activity"><i class="fa fa-check"></i><b>2.1.3</b> Activity</a></li>
<li class="chapter" data-level="2.1.4" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#footnotes"><i class="fa fa-check"></i><b>2.1.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="lecture---reading-a-paper.html"><a href="lecture---reading-a-paper.html"><i class="fa fa-check"></i><b>2.2</b> Lecture - Reading a paper</a></li>
<li class="chapter" data-level="2.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html"><i class="fa fa-check"></i><b>2.3</b> Activity - Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#purpose-1"><i class="fa fa-check"></i><b>2.3.1</b> Purpose</a></li>
<li class="chapter" data-level="2.3.2" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#learning-objectives"><i class="fa fa-check"></i><b>2.3.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.3.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#introduction-1"><i class="fa fa-check"></i><b>2.3.3</b> Introduction</a></li>
<li class="chapter" data-level="2.3.4" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#instructions-1"><i class="fa fa-check"></i><b>2.3.4</b> Instructions</a></li>
<li class="chapter" data-level="2.3.5" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-1---overview-of-the-paper"><i class="fa fa-check"></i><b>2.3.5</b> Activity 1 - Overview of the paper</a></li>
<li class="chapter" data-level="2.3.6" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-2---methods-and-figure-analysis-in-class-and-homework"><i class="fa fa-check"></i><b>2.3.6</b> Activity 2 - Methods and Figure Analysis (in class and homework)</a></li>
<li class="chapter" data-level="2.3.7" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#grading-criteria-1"><i class="fa fa-check"></i><b>2.3.7</b> Grading criteria</a></li>
<li class="chapter" data-level="2.3.8" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#footnotes-1"><i class="fa fa-check"></i><b>2.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Project: 16S rRNA Research</b></span></li>
<li class="chapter" data-level="3" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html"><i class="fa fa-check"></i><b>3</b> Meet the MISO study</a>
<ul>
<li class="chapter" data-level="3.0.1" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#miso-study-diet-and-sampling-schedule"><i class="fa fa-check"></i><b>3.0.1</b> MISO study diet and sampling schedule</a></li>
<li class="chapter" data-level="3.0.2" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#miso-study-variables-and-factors"><i class="fa fa-check"></i><b>3.0.2</b> MISO study variables and factors</a></li>
<li class="chapter" data-level="3.0.3" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#amplicon-sequence-variants-asvs-data"><i class="fa fa-check"></i><b>3.0.3</b> Amplicon sequence variants (ASVs) data</a></li>
<li class="chapter" data-level="3.0.4" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#footnotes-2"><i class="fa fa-check"></i><b>3.0.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="s-rrna-data-in-spreadsheets.html"><a href="s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>4</b> 16S rRNA Data in spreadsheets</a>
<ul>
<li class="chapter" data-level="4.1" data-path="exploring-16s-rrna-data-in-spreadsheets.html"><a href="exploring-16s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>4.1</b> Exploring 16S rRNA data in spreadsheets</a></li>
<li class="chapter" data-level="4.2" data-path="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><a href="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><i class="fa fa-check"></i><b>4.2</b> Lecture - Introducing 16S rRNA Data Using Spreadsheets</a></li>
<li class="chapter" data-level="4.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html"><i class="fa fa-check"></i><b>4.3</b> Activity - Introducing 16S rRNA Data Using Spreadsheet</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#learning-objectives-1"><i class="fa fa-check"></i><b>4.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="4.3.2" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#introduction-2"><i class="fa fa-check"></i><b>4.3.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-1-quick-data-overview-based-on-1-sample"><i class="fa fa-check"></i><b>4.3.3</b> Activity 1 – Quick data overview based on 1 sample</a></li>
<li class="chapter" data-level="4.3.4" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-explore-variation-between-individuals"><i class="fa fa-check"></i><b>4.3.4</b> Activity 2 – Explore variation between individuals</a></li>
<li class="chapter" data-level="4.3.5" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-3.-plot-individual-variation-in-abundance-of-the-most-common-asv"><i class="fa fa-check"></i><b>4.3.5</b> Activity 2-3. Plot individual variation in abundance of the most common ASV</a></li>
<li class="chapter" data-level="4.3.6" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-3-explore-variation-within-a-single-individual-associated-with-diet"><i class="fa fa-check"></i><b>4.3.6</b> Activity 3 – Explore variation within a single individual associated with diet</a></li>
<li class="chapter" data-level="4.3.7" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-4-try-it-out"><i class="fa fa-check"></i><b>4.3.7</b> Activity 4 – Try it Out!</a></li>
<li class="chapter" data-level="4.3.8" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#grading-criteria-2"><i class="fa fa-check"></i><b>4.3.8</b> Grading criteria</a></li>
<li class="chapter" data-level="4.3.9" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#footnotes-3"><i class="fa fa-check"></i><b>4.3.9</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="exploring-16s-rrna-data-with-phyloseq.html"><a href="exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5</b> Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="5.1" data-path="exploring-16s-rrna-data-with-phyloseq-1.html"><a href="exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>5.1</b> Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="5.2" data-path="lecture---exploring-16s-rrna-data-with-phyloseq.html"><a href="lecture---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5.2</b> Lecture - Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="5.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#learning-objectives-2"><i class="fa fa-check"></i><b>5.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="5.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#introduction-3"><i class="fa fa-check"></i><b>5.3.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-1-explore-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>5.3.3</b> Activity 1 – Explore 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="5.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-2-try-it-out-questions"><i class="fa fa-check"></i><b>5.3.4</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="5.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#grading-criteria-3"><i class="fa fa-check"></i><b>5.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="5.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#footnotes-4"><i class="fa fa-check"></i><b>5.3.6</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><i class="fa fa-check"></i><b>5.4</b> Try it Question 1 - What has more impact on human microbial variation, diet or individuality</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1a-plot-high-level-taxonomy-phylum-level-for-each-timepoint-using-the-plot_bar-function.-specifically-your-plot-should-show-phyla-diversity-across-all-5-timepoints."><i class="fa fa-check"></i><b>5.4.1</b> Step 1A: Plot high-level taxonomy (Phylum level) for each timepoint using the plot_bar() function.** Specifically, your plot should show Phyla diversity across all 5 timepoints.</a></li>
<li class="chapter" data-level="5.4.2" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1b-plot-high-level-taxonomy-phylum-level-for-each-individual-using-only-normalized-homogenized-diet-hd-timepoints-to-minimize-contribution-of-diet-to-the-analysis-using-the-plot_bar-function.-specifically-your-plot-should-show-phyla-diversity-across-all-21-individuals-at-hd-timepoints."><i class="fa fa-check"></i><b>5.4.2</b> Step 1B: Plot high-level taxonomy (Phylum level) for each individual, using only normalized homogenized diet “HD” timepoints to minimize contribution of diet to the analysis using the plot_bar() function. Specifically, your plot should show Phyla diversity across all 21 individuals at HD timepoints.</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><i class="fa fa-check"></i><b>5.5</b> Try it Question 2 - What are some of the most abundant microbes when viewed at different taxonomic resolutions</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2a.-plot-phyla-across-all-subjects-and-identify-the-most-abundant-phyla.-specifically-your-plot-should-show-phyla-diversity-across-all-21-individuals."><i class="fa fa-check"></i><b>5.5.1</b> Step 2A. Plot Phyla across all subjects and identify the most abundant Phyla.** Specifically, your plot should show Phyla diversity across all 21 individuals.</a></li>
<li class="chapter" data-level="5.5.2" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2b.-subset-the-most-abundant-phyla-and-plot-order-skip-class"><i class="fa fa-check"></i><b>5.5.2</b> Step 2B. Subset the most abundant Phyla, and plot Order (Skip Class)</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>5.6</b> Try it Question 3 - Does gender contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3c.-test-gender-based-difference-in-species-for-the-low-abundant-phylum-you-selected.-specifically-your-plot-should-show-species-diversity-within-male-and-female-groups-for-only-your-phylum-of-choice."><i class="fa fa-check"></i><b>5.6.1</b> Step 3C. Test gender-based difference in Species for the low abundant phylum you selected.** Specifically, your plot should show species diversity within male and female groups for only your phylum of choice.</a></li>
<li class="chapter" data-level="5.6.2" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3d-plot-the-individual-abundance-of-2-candidate-microbes-for-gender-differences-based-on-your-analysis-above-andor-the-curated-menu-of-microbes-below-previously-associated-with-gender-variation-with-color-denoting-gender."><i class="fa fa-check"></i><b>5.6.2</b> Step 3D: Plot the individual abundance of 2 candidate microbes for gender differences based on your analysis above, and/or the curated menu of microbes below (previously associated with gender variation), with color denoting gender.</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>5.7</b> Try it Question 4 - Does age contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="5.7.1" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4a.-perform-an-initial-survey-to-see-if-there-are-any-differences-in-phyla-related-to-age."><i class="fa fa-check"></i><b>5.7.1</b> Step 4A. Perform an initial survey to see if there are any differences in phyla related to age.</a></li>
<li class="chapter" data-level="5.7.2" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4b.-merge-samples-by-age-then-re-normalize-and-finally-re-plot-phyla-by-age."><i class="fa fa-check"></i><b>5.7.2</b> Step 4B. Merge samples by age, then re-normalize and finally re-plot Phyla by age.</a></li>
<li class="chapter" data-level="5.7.3" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4c.-candidate-phyla-implicated-in-age-variation-include-firmicutes-and-bacteroidetes.-plot-these-specific-phyla-on-their-own-separately-to-observe-any-shift-that-occurs-with-age.-use-your-new-merged-and-normalized-merge-phyloseq-object-as-input."><i class="fa fa-check"></i><b>5.7.3</b> Step 4C. Candidate Phyla implicated in age variation include Firmicutes and Bacteroidetes. Plot these specific phyla on their own, separately, to observe any shift that occurs with age.** Use your new merged and normalized ‘merge’ phyloseq object as input.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="analyzing-16s-rrna-data-with-phyloseq.html"><a href="analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>6</b> Analyzing 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="6.1" data-path="analyzing-16s-rrna-data-with-phyloseq-1.html"><a href="analyzing-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>6.1</b> Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="6.2" data-path="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><a href="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>6.2</b> Lecture - Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="6.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>6.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#learning-objectives-3"><i class="fa fa-check"></i><b>6.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="6.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#introduction-4"><i class="fa fa-check"></i><b>6.3.2</b> Introduction</a></li>
<li class="chapter" data-level="6.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-1-analyze-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>6.3.3</b> Activity 1 – Analyze 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="6.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-2-try-it-out-questions-1"><i class="fa fa-check"></i><b>6.3.4</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="6.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#grading-criteria-4"><i class="fa fa-check"></i><b>6.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="6.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#footnotes-5"><i class="fa fa-check"></i><b>6.3.6</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><i class="fa fa-check"></i><b>6.4</b> Try it Question 1 - How sensitive is microbial diversity to variables like diet, age and gender</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1a.-plot-alpha-diversity-of-the-full-miso-dataset-by-subject-individuals."><i class="fa fa-check"></i><b>6.4.1</b> Step 1A. Plot alpha diversity of the full MISO dataset by subject (individuals).</a></li>
<li class="chapter" data-level="6.4.2" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1b.-plot-alpha-diversity-of-the-full-miso-dataset-by-timepointgroup."><i class="fa fa-check"></i><b>6.4.2</b> Step 1B. Plot alpha diversity of the full MISO dataset, by timepointgroup.</a></li>
<li class="chapter" data-level="6.4.3" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1c.-plot-alpha-diversity-of-the-full-miso-dataset-by-age."><i class="fa fa-check"></i><b>6.4.3</b> Step 1C. Plot alpha diversity of the full MISO dataset by age.</a></li>
<li class="chapter" data-level="6.4.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1d.-plot-alpha-diversity-of-the-full-miso-dataset-by-gender."><i class="fa fa-check"></i><b>6.4.4</b> Step 1D. Plot alpha diversity of the full MISO dataset by gender.</a></li>
<li class="chapter" data-level="6.4.5" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-2.-plot-alpha-diversity-of-the-full-miso-dataset-grouping-based-on-levels-for-5-different-metabolites-creatinine-pcs-is-hipp-pag."><i class="fa fa-check"></i><b>6.4.5</b> Step 2. Plot alpha diversity of the full MISO dataset, grouping based on levels for 5 different metabolites: Creatinine, PCS, IS, HIPP, PAG.</a></li>
<li class="chapter" data-level="6.4.6" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-3.-select-data-for-poster-and-include-the-code-used-to-generate-the-figure."><i class="fa fa-check"></i><b>6.4.6</b> Step 3. Select data for poster and include the code used to generate the figure.</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><i class="fa fa-check"></i><b>6.5</b> Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals</a>
<ul>
<li class="chapter" data-level="6.5.1" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-1.-make-a-pcoa-plot-ordinating-on-the-entire-miso-dataset-and-coloring-by-individual.-investigate-resulting-plot-shape."><i class="fa fa-check"></i><b>6.5.1</b> Step 1. Make a PcOA plot, ordinating on the entire miso dataset, and coloring by individual. Investigate resulting plot shape.</a></li>
<li class="chapter" data-level="6.5.2" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-2.-using-the-pcoa-plot-from-step-1-above-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-diet-gender-and-age"><i class="fa fa-check"></i><b>6.5.2</b> Step 2. Using the PCoA plot from Step 1 above, look for any correlations between the shape of your PCoA plot and diet, gender and age</a></li>
<li class="chapter" data-level="6.5.3" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-3.-using-the-pcoa-plot-from-step-1-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-the-level-of-metabolites."><i class="fa fa-check"></i><b>6.5.3</b> Step 3. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and the level of metabolites.</a></li>
<li class="chapter" data-level="6.5.4" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-4.-one-way-of-checking-if-the-metabolite-levels-indeed-correlate-with-your-data-is-to-see-if-subsetting-smaller-chunks-or-specific-chunks-of-the-data-will-still-maintain-the-metabolite-data-relationship-or-break-it."><i class="fa fa-check"></i><b>6.5.4</b> Step 4. One way of checking if the metabolite levels indeed correlate with your data, is to see if subsetting smaller chunks or specific chunks of the data will still maintain the metabolite-data relationship or break it.</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><i class="fa fa-check"></i><b>6.6</b> Try it Question 3 - What microbes (ASVs) differ between males and females, and does age have an impact</a>
<ul>
<li class="chapter" data-level="6.6.1" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3a.-perform-differential-abundance-analysis-between-genders."><i class="fa fa-check"></i><b>6.6.1</b> Step 3A. Perform differential abundance analysis between genders.</a></li>
<li class="chapter" data-level="6.6.2" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3b.-determine-the-impact-of-younger-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>6.6.2</b> Step 3B. Determine the impact of younger age on differential abundance in males and females, by subsetting age &lt; 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
<li class="chapter" data-level="6.6.3" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3c.-determine-the-impact-of-older-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>6.6.3</b> Step 3C. Determine the impact of older age on differential abundance in males and females, by subsetting age &gt;= 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
<li class="chapter" data-level="6.6.4" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-4.-select-data-for-poster-and-include-the-code-used-to-generate-the-figure"><i class="fa fa-check"></i><b>6.6.4</b> Step 4. Select data for poster and include the code used to generate the figure</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><i class="fa fa-check"></i><b>6.7</b> Try it Question 4 - Is there diet and age interaction and what microbes (ASVs) correlate with changes in diet-age interaction</a>
<ul>
<li class="chapter" data-level="6.7.1" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html#step-1.-plot-alpha-diversity-based-on-age-for-all-dietary-groups-and-then-for-bd-hd-and-wo."><i class="fa fa-check"></i><b>6.7.1</b> Step 1. Plot alpha diversity based on age for all dietary groups, and then for BD, HD and WO.</a></li>
<li class="chapter" data-level="6.7.2" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html#step-2.-perform-differential-abundance-analysis-comparing-hd-to-bd-for-the-population-in-general-and-then-for-older-individuals-specifically-because-older-individuals-produced-the-biggest-dietage-interaction-based-on-alpha-diversity."><i class="fa fa-check"></i><b>6.7.2</b> Step 2. Perform differential abundance analysis comparing HD to BD for the population in general, and then for older individuals specifically (because older individuals produced the biggest diet–age interaction based on alpha diversity).</a></li>
<li class="chapter" data-level="6.7.3" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html#step-2b.-perform-differential-abundance-analysis-between-hd-and-bd-for-older-individual-50-yo."><i class="fa fa-check"></i><b>6.7.3</b> Step 2B. Perform differential abundance analysis between HD and BD for older individual (&gt; 50 yo.)</a></li>
<li class="chapter" data-level="6.7.4" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html#step-3.-select-data-for-poster-and-include-the-code-used-to-generate-the-figure.-1"><i class="fa fa-check"></i><b>6.7.4</b> Step 3. Select data for poster and include the code used to generate the figure.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Course Name</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=10.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/C-MOOR_Template/dasl_thin_main_image.png">
</div>
<div id="try-it-question-4---does-age-contribute-to-individual-microbiome-variation" class="section level2 hasAnchor" number="5.7">
<h2><span class="header-section-number">5.7</span> Try it Question 4 - Does age contribute to individual microbiome variation<a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#try-it-question-4---does-age-contribute-to-individual-microbiome-variation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Approach: Survey the differences in microbiome across age, refining the plot to accommodate a continuous variable (age) and unequal sample sizes (some individuals in the study are the same age). Then, test if there is a shift in microbiome abundance of phyla Firmicutes and Bacteroidetes, two phyla suggested previously to change with age.</p>
<div id="step-4a.-perform-an-initial-survey-to-see-if-there-are-any-differences-in-phyla-related-to-age." class="section level3 hasAnchor" number="5.7.1">
<h3><span class="header-section-number">5.7.1</span> Step 4A. Perform an initial survey to see if there are any differences in phyla related to age.<a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4a.-perform-an-initial-survey-to-see-if-there-are-any-differences-in-phyla-related-to-age." class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Specifically, your plot should show the relationship phyla diversity and age of an individual.</p>
<ul>
<li>Refer to the “Explore 16S rRNA Data with phyloseq” tutorial for help using the plot_bar() function.</li>
<li>Use the following code as a template
plot_bar(miso, “fill in the blank”, fill = “fill in the blank”, title = “fill in the blank”) +
geom_bar(aes(color = fill in the blank, fill = fill in the blank), stat = “identity”, position = “stack”)</li>
</ul>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb15-1" tabindex="-1"></a><span class="fu">plot_bar</span>(miso, <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb15-2"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb15-2" tabindex="-1"></a><span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">4A-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<ul>
<li>You probably found it difficult to discern any differences.</li>
<li>Abundance is additive, so having more samples of the same age will result in adding the abundances together, so if there are say 2 samples with age 27 (as is the case), the abundance of the bar will be much higher than other ages that have only one individual.</li>
</ul>
</div>
<div id="step-4b.-merge-samples-by-age-then-re-normalize-and-finally-re-plot-phyla-by-age." class="section level3 hasAnchor" number="5.7.2">
<h3><span class="header-section-number">5.7.2</span> Step 4B. Merge samples by age, then re-normalize and finally re-plot Phyla by age.<a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4b.-merge-samples-by-age-then-re-normalize-and-finally-re-plot-phyla-by-age." class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li>Merge samples of the same age, saving the resulting phyloseq object as ‘merge’. You will probably get a warning message letting you know “NAs were introduced by coercion.” That is OK.</li>
</ol>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb16-1" tabindex="-1"></a>merge <span class="ot">&lt;-</span> <span class="fu">merge_samples</span>(miso, <span class="st">&quot;age&quot;</span>)</span></code></pre></div>
<ol start="2" style="list-style-type: decimal">
<li>Check your merged phyloseq object. Your new ‘merge’ object should have 16 samples corresponding to 16 unique ages in the MISO dataset.</li>
</ol>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb17-1" tabindex="-1"></a>merge</span>
<span id="cb17-2"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb17-2" tabindex="-1"></a></span>
<span id="cb17-3"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb17-3" tabindex="-1"></a>phyloseq<span class="sc">-</span>class experiment<span class="sc">-</span>level object</span>
<span id="cb17-4"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb17-4" tabindex="-1"></a><span class="fu">otu_table</span>()   OTU Table<span class="sc">:</span>         [ <span class="dv">1702</span> taxa and <span class="dv">16</span> samples ]</span>
<span id="cb17-5"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb17-5" tabindex="-1"></a><span class="fu">sample_data</span>() Sample Data<span class="sc">:</span>       [ <span class="dv">16</span> samples by <span class="dv">32</span> sample variables ]</span>
<span id="cb17-6"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb17-6" tabindex="-1"></a><span class="fu">tax_table</span>()   Taxonomy Table<span class="sc">:</span>    [ <span class="dv">1702</span> taxa by <span class="dv">9</span> taxonomic ranks ]</span></code></pre></div>
<ol start="3" style="list-style-type: decimal">
<li>Normalize the sample counts using the function transform_sample_counts. Our function, (100* x/sum(x)) will transform all the samples into proportions.</li>
</ol>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb18-1" tabindex="-1"></a>merge <span class="ot">&lt;-</span> <span class="fu">transform_sample_counts</span>(merge, <span class="cf">function</span>(x) <span class="dv">100</span> <span class="sc">*</span> x<span class="sc">/</span><span class="fu">sum</span>(x))</span></code></pre></div>
<ol start="4" style="list-style-type: decimal">
<li>Re-plot Phyla by age.</li>
</ol>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb19-1" tabindex="-1"></a><span class="fu">plot_bar</span>(merge, <span class="at">x=</span> <span class="st">&quot;age&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span></span>
<span id="cb19-2"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb19-2" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<ol start="5" style="list-style-type: decimal">
<li>If you need to decrease the size of your legend key and text to fit, add code below to your plot after adding a + to the end of the geom_bar line:</li>
</ol>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb20-1" tabindex="-1"></a><span class="fu">theme</span>(<span class="at">legend.text=</span><span class="fu">element_text</span>(<span class="at">size=</span><span class="dv">6</span>)) <span class="sc">+</span> </span>
<span id="cb20-2"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb20-2" tabindex="-1"></a><span class="fu">theme</span>(<span class="at">legend.key.size =</span> <span class="fu">unit</span>(<span class="dv">6</span>, <span class="st">&quot;pt&quot;</span>))</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">4B-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">4B-2. Do you observe any candidate Phyla that changes with age?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="step-4c.-candidate-phyla-implicated-in-age-variation-include-firmicutes-and-bacteroidetes.-plot-these-specific-phyla-on-their-own-separately-to-observe-any-shift-that-occurs-with-age.-use-your-new-merged-and-normalized-merge-phyloseq-object-as-input." class="section level3 hasAnchor" number="5.7.3">
<h3><span class="header-section-number">5.7.3</span> Step 4C. Candidate Phyla implicated in age variation include Firmicutes and Bacteroidetes. Plot these specific phyla on their own, separately, to observe any shift that occurs with age.** Use your new merged and normalized ‘merge’ phyloseq object as input.<a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4c.-candidate-phyla-implicated-in-age-variation-include-firmicutes-and-bacteroidetes.-plot-these-specific-phyla-on-their-own-separately-to-observe-any-shift-that-occurs-with-age.-use-your-new-merged-and-normalized-merge-phyloseq-object-as-input." class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Refer to the “Subset taxonomy” section of the “Explore 16S rRNA Data with phyloseq” tutorial for help using the subset_taxa and plot_bar() functions.</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb21-1" tabindex="-1"></a>subset <span class="ot">=</span> <span class="fu">subset_taxa</span>(merge,  Phylum <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>)</span>
<span id="cb21-2"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb21-2" tabindex="-1"></a><span class="fu">plot_bar</span>(subset, <span class="st">&quot;age&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb21-3"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#cb21-3" tabindex="-1"></a> <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">4C-1. Insert the plot that graphs the relationship of age to the Phylum Firmicutes below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">4C-2. Do you observe any changes in Phyla based on age?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">4C-3. Insert the plot that graphs the relationship of age to the Phylum Bacteroidetes below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">4C-4 Do you observe any changes in Phyla based on age?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Step 4C-5: Based on the graphs you made, did you identify any candidate Phylum that may differ between genders?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>A suggestion for your poster:</strong>
- Combine 2-4 of your plots to make one figure summarizing Phylum and Species age-based differences you observed in the MISO study. Include titles and short figure description (figure legend).
- Paste the code you used to generate your final figure(s) below:</p>
<table>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>

<!-- Set up code of OTTR Book-->
</div>
</div>
<!-- </div> -->
<hr>
<center> 
  <div class="footer">
      All materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="analyzing-16s-rrna-data-with-phyloseq.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
