<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6.5 Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals | Course Name</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="6.5 Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals | Course Name" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6.5 Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals | Course Name" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  



<meta name="date" content="2025-07-16" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/dasl_favicon.ico" type="image/x-icon" />
<link rel="prev" href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"/>
<link rel="next" href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="assets/style.css" type="text/css" />
<link rel="stylesheet" href="assets/C-MOOR_Template/c-moor-ottr.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.c-moor.org/" target="_blank"><img src="assets/C-MOOR_Template/c-moor-logo-vertical.png" style="width: 70%; padding-left: 10px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#learning-goals"><i class="fa fa-check"></i>Learning Goals</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="motivation.html"><a href="motivation.html"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="target-audience.html"><a href="target-audience.html"><i class="fa fa-check"></i><b>1.2</b> Target Audience</a></li>
<li class="chapter" data-level="1.3" data-path="curriculum.html"><a href="curriculum.html"><i class="fa fa-check"></i><b>1.3</b> Curriculum</a></li>
</ul></li>
<li class="part"><span><b>Scientific Literature</b></span></li>
<li class="chapter" data-level="2" data-path="scientific-literature.html"><a href="scientific-literature.html"><i class="fa fa-check"></i><b>2</b> Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html"><i class="fa fa-check"></i><b>2.1</b> Pre-lab - Scientific literature</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#purpose"><i class="fa fa-check"></i><b>2.1.1</b> Purpose</a></li>
<li class="chapter" data-level="2.1.2" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#grading-criteria"><i class="fa fa-check"></i><b>2.1.2</b> Grading criteria</a></li>
<li class="chapter" data-level="2.1.3" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#activity"><i class="fa fa-check"></i><b>2.1.3</b> Activity</a></li>
<li class="chapter" data-level="2.1.4" data-path="pre-lab---scientific-literature.html"><a href="pre-lab---scientific-literature.html#footnotes"><i class="fa fa-check"></i><b>2.1.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="lecture---reading-a-paper.html"><a href="lecture---reading-a-paper.html"><i class="fa fa-check"></i><b>2.2</b> Lecture - Reading a paper</a></li>
<li class="chapter" data-level="2.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html"><i class="fa fa-check"></i><b>2.3</b> Activity - Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#purpose-1"><i class="fa fa-check"></i><b>2.3.1</b> Purpose</a></li>
<li class="chapter" data-level="2.3.2" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#learning-objectives"><i class="fa fa-check"></i><b>2.3.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.3.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#introduction-1"><i class="fa fa-check"></i><b>2.3.3</b> Introduction</a></li>
<li class="chapter" data-level="2.3.4" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#instructions-1"><i class="fa fa-check"></i><b>2.3.4</b> Instructions</a></li>
<li class="chapter" data-level="2.3.5" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-1---overview-of-the-paper"><i class="fa fa-check"></i><b>2.3.5</b> Activity 1 - Overview of the paper</a></li>
<li class="chapter" data-level="2.3.6" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-2---methods-and-figure-analysis-in-class-and-homework"><i class="fa fa-check"></i><b>2.3.6</b> Activity 2 - Methods and Figure Analysis (in class and homework)</a></li>
<li class="chapter" data-level="2.3.7" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#grading-criteria-1"><i class="fa fa-check"></i><b>2.3.7</b> Grading criteria</a></li>
<li class="chapter" data-level="2.3.8" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#footnotes-1"><i class="fa fa-check"></i><b>2.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Project: 16S rRNA Research</b></span></li>
<li class="chapter" data-level="3" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html"><i class="fa fa-check"></i><b>3</b> Meet the MISO study</a>
<ul>
<li class="chapter" data-level="3.0.1" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#miso-study-diet-and-sampling-schedule"><i class="fa fa-check"></i><b>3.0.1</b> MISO study diet and sampling schedule</a></li>
<li class="chapter" data-level="3.0.2" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#miso-study-variables-and-factors"><i class="fa fa-check"></i><b>3.0.2</b> MISO study variables and factors</a></li>
<li class="chapter" data-level="3.0.3" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#amplicon-sequence-variants-asvs-data"><i class="fa fa-check"></i><b>3.0.3</b> Amplicon sequence variants (ASVs) data</a></li>
<li class="chapter" data-level="3.0.4" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#footnotes-2"><i class="fa fa-check"></i><b>3.0.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="s-rrna-data-in-spreadsheets.html"><a href="s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>4</b> 16S rRNA Data in spreadsheets</a>
<ul>
<li class="chapter" data-level="4.1" data-path="exploring-16s-rrna-data-in-spreadsheets.html"><a href="exploring-16s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>4.1</b> Exploring 16S rRNA data in spreadsheets</a></li>
<li class="chapter" data-level="4.2" data-path="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><a href="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><i class="fa fa-check"></i><b>4.2</b> Lecture - Introducing 16S rRNA Data Using Spreadsheets</a></li>
<li class="chapter" data-level="4.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html"><i class="fa fa-check"></i><b>4.3</b> Activity - Introducing 16S rRNA Data Using Spreadsheet</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#learning-objectives-1"><i class="fa fa-check"></i><b>4.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="4.3.2" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#introduction-2"><i class="fa fa-check"></i><b>4.3.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-1-quick-data-overview-based-on-1-sample"><i class="fa fa-check"></i><b>4.3.3</b> Activity 1 – Quick data overview based on 1 sample</a></li>
<li class="chapter" data-level="4.3.4" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-explore-variation-between-individuals"><i class="fa fa-check"></i><b>4.3.4</b> Activity 2 – Explore variation between individuals</a></li>
<li class="chapter" data-level="4.3.5" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-3.-plot-individual-variation-in-abundance-of-the-most-common-asv"><i class="fa fa-check"></i><b>4.3.5</b> Activity 2-3. Plot individual variation in abundance of the most common ASV</a></li>
<li class="chapter" data-level="4.3.6" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-3-explore-variation-within-a-single-individual-associated-with-diet"><i class="fa fa-check"></i><b>4.3.6</b> Activity 3 – Explore variation within a single individual associated with diet</a></li>
<li class="chapter" data-level="4.3.7" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-4-try-it-out"><i class="fa fa-check"></i><b>4.3.7</b> Activity 4 – Try it Out!</a></li>
<li class="chapter" data-level="4.3.8" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#grading-criteria-2"><i class="fa fa-check"></i><b>4.3.8</b> Grading criteria</a></li>
<li class="chapter" data-level="4.3.9" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#footnotes-3"><i class="fa fa-check"></i><b>4.3.9</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="exploring-16s-rrna-data-with-phyloseq.html"><a href="exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5</b> Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="5.1" data-path="exploring-16s-rrna-data-with-phyloseq-1.html"><a href="exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>5.1</b> Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="5.2" data-path="lecture---exploring-16s-rrna-data-with-phyloseq.html"><a href="lecture---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5.2</b> Lecture - Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="5.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#learning-objectives-2"><i class="fa fa-check"></i><b>5.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="5.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#introduction-3"><i class="fa fa-check"></i><b>5.3.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-1-explore-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>5.3.3</b> Activity 1 – Explore 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="5.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-2-try-it-out-questions"><i class="fa fa-check"></i><b>5.3.4</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="5.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#grading-criteria-3"><i class="fa fa-check"></i><b>5.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="5.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#footnotes-4"><i class="fa fa-check"></i><b>5.3.6</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><i class="fa fa-check"></i><b>5.4</b> Try it Question 1 - What has more impact on human microbial variation, diet or individuality</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1a-plot-high-level-taxonomy-phylum-level-for-each-timepoint-using-the-plot_bar-function.-specifically-your-plot-should-show-phyla-diversity-across-all-5-timepoints."><i class="fa fa-check"></i><b>5.4.1</b> Step 1A: Plot high-level taxonomy (Phylum level) for each timepoint using the plot_bar() function.** Specifically, your plot should show Phyla diversity across all 5 timepoints.</a></li>
<li class="chapter" data-level="5.4.2" data-path="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html"><a href="try-it-question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality.html#step-1b-plot-high-level-taxonomy-phylum-level-for-each-individual-using-only-normalized-homogenized-diet-hd-timepoints-to-minimize-contribution-of-diet-to-the-analysis-using-the-plot_bar-function.-specifically-your-plot-should-show-phyla-diversity-across-all-21-individuals-at-hd-timepoints."><i class="fa fa-check"></i><b>5.4.2</b> Step 1B: Plot high-level taxonomy (Phylum level) for each individual, using only normalized homogenized diet “HD” timepoints to minimize contribution of diet to the analysis using the plot_bar() function. Specifically, your plot should show Phyla diversity across all 21 individuals at HD timepoints.</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><i class="fa fa-check"></i><b>5.5</b> Try it Question 2 - What are some of the most abundant microbes when viewed at different taxonomic resolutions</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2a.-plot-phyla-across-all-subjects-and-identify-the-most-abundant-phyla.-specifically-your-plot-should-show-phyla-diversity-across-all-21-individuals."><i class="fa fa-check"></i><b>5.5.1</b> Step 2A. Plot Phyla across all subjects and identify the most abundant Phyla.** Specifically, your plot should show Phyla diversity across all 21 individuals.</a></li>
<li class="chapter" data-level="5.5.2" data-path="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html"><a href="try-it-question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions.html#step-2b.-subset-the-most-abundant-phyla-and-plot-order-skip-class"><i class="fa fa-check"></i><b>5.5.2</b> Step 2B. Subset the most abundant Phyla, and plot Order (Skip Class)</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>5.6</b> Try it Question 3 - Does gender contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3c.-test-gender-based-difference-in-species-for-the-low-abundant-phylum-you-selected.-specifically-your-plot-should-show-species-diversity-within-male-and-female-groups-for-only-your-phylum-of-choice."><i class="fa fa-check"></i><b>5.6.1</b> Step 3C. Test gender-based difference in Species for the low abundant phylum you selected.** Specifically, your plot should show species diversity within male and female groups for only your phylum of choice.</a></li>
<li class="chapter" data-level="5.6.2" data-path="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-3---does-gender-contribute-to-individual-microbiome-variation.html#step-3d-plot-the-individual-abundance-of-2-candidate-microbes-for-gender-differences-based-on-your-analysis-above-andor-the-curated-menu-of-microbes-below-previously-associated-with-gender-variation-with-color-denoting-gender."><i class="fa fa-check"></i><b>5.6.2</b> Step 3D: Plot the individual abundance of 2 candidate microbes for gender differences based on your analysis above, and/or the curated menu of microbes below (previously associated with gender variation), with color denoting gender.</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><i class="fa fa-check"></i><b>5.7</b> Try it Question 4 - Does age contribute to individual microbiome variation</a>
<ul>
<li class="chapter" data-level="5.7.1" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4a.-perform-an-initial-survey-to-see-if-there-are-any-differences-in-phyla-related-to-age."><i class="fa fa-check"></i><b>5.7.1</b> Step 4A. Perform an initial survey to see if there are any differences in phyla related to age.</a></li>
<li class="chapter" data-level="5.7.2" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4b.-merge-samples-by-age-then-re-normalize-and-finally-re-plot-phyla-by-age."><i class="fa fa-check"></i><b>5.7.2</b> Step 4B. Merge samples by age, then re-normalize and finally re-plot Phyla by age.</a></li>
<li class="chapter" data-level="5.7.3" data-path="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html"><a href="try-it-question-4---does-age-contribute-to-individual-microbiome-variation.html#step-4c.-candidate-phyla-implicated-in-age-variation-include-firmicutes-and-bacteroidetes.-plot-these-specific-phyla-on-their-own-separately-to-observe-any-shift-that-occurs-with-age.-use-your-new-merged-and-normalized-merge-phyloseq-object-as-input."><i class="fa fa-check"></i><b>5.7.3</b> Step 4C. Candidate Phyla implicated in age variation include Firmicutes and Bacteroidetes. Plot these specific phyla on their own, separately, to observe any shift that occurs with age.** Use your new merged and normalized ‘merge’ phyloseq object as input.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="analyzing-16s-rrna-data-with-phyloseq.html"><a href="analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>6</b> Analyzing 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="6.1" data-path="analyzing-16s-rrna-data-with-phyloseq-1.html"><a href="analyzing-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>6.1</b> Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="6.2" data-path="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><a href="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>6.2</b> Lecture - Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="6.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>6.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#learning-objectives-3"><i class="fa fa-check"></i><b>6.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="6.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#introduction-4"><i class="fa fa-check"></i><b>6.3.2</b> Introduction</a></li>
<li class="chapter" data-level="6.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-1-analyze-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>6.3.3</b> Activity 1 – Analyze 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="6.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-2-try-it-out-questions-1"><i class="fa fa-check"></i><b>6.3.4</b> Activity 2 – Try it out questions</a></li>
<li class="chapter" data-level="6.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#grading-criteria-4"><i class="fa fa-check"></i><b>6.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="6.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#footnotes-5"><i class="fa fa-check"></i><b>6.3.6</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><i class="fa fa-check"></i><b>6.4</b> Try it Question 1 - How sensitive is microbial diversity to variables like diet, age and gender</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1a.-plot-alpha-diversity-of-the-full-miso-dataset-by-subject-individuals."><i class="fa fa-check"></i><b>6.4.1</b> Step 1A. Plot alpha diversity of the full MISO dataset by subject (individuals).</a></li>
<li class="chapter" data-level="6.4.2" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1b.-plot-alpha-diversity-of-the-full-miso-dataset-by-timepointgroup."><i class="fa fa-check"></i><b>6.4.2</b> Step 1B. Plot alpha diversity of the full MISO dataset, by timepointgroup.</a></li>
<li class="chapter" data-level="6.4.3" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1c.-plot-alpha-diversity-of-the-full-miso-dataset-by-age."><i class="fa fa-check"></i><b>6.4.3</b> Step 1C. Plot alpha diversity of the full MISO dataset by age.</a></li>
<li class="chapter" data-level="6.4.4" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-1d.-plot-alpha-diversity-of-the-full-miso-dataset-by-gender."><i class="fa fa-check"></i><b>6.4.4</b> Step 1D. Plot alpha diversity of the full MISO dataset by gender.</a></li>
<li class="chapter" data-level="6.4.5" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-2.-plot-alpha-diversity-of-the-full-miso-dataset-grouping-based-on-levels-for-5-different-metabolites-creatinine-pcs-is-hipp-pag."><i class="fa fa-check"></i><b>6.4.5</b> Step 2. Plot alpha diversity of the full MISO dataset, grouping based on levels for 5 different metabolites: Creatinine, PCS, IS, HIPP, PAG.</a></li>
<li class="chapter" data-level="6.4.6" data-path="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html"><a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html#step-3.-select-data-for-poster-and-include-the-code-used-to-generate-the-figure."><i class="fa fa-check"></i><b>6.4.6</b> Step 3. Select data for poster and include the code used to generate the figure.</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><i class="fa fa-check"></i><b>6.5</b> Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals</a>
<ul>
<li class="chapter" data-level="6.5.1" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-1.-make-a-pcoa-plot-ordinating-on-the-entire-miso-dataset-and-coloring-by-individual.-investigate-resulting-plot-shape."><i class="fa fa-check"></i><b>6.5.1</b> Step 1. Make a PcOA plot, ordinating on the entire miso dataset, and coloring by individual. Investigate resulting plot shape.</a></li>
<li class="chapter" data-level="6.5.2" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-2.-using-the-pcoa-plot-from-step-1-above-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-diet-gender-and-age"><i class="fa fa-check"></i><b>6.5.2</b> Step 2. Using the PCoA plot from Step 1 above, look for any correlations between the shape of your PCoA plot and diet, gender and age</a></li>
<li class="chapter" data-level="6.5.3" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-3.-using-the-pcoa-plot-from-step-1-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-the-level-of-metabolites."><i class="fa fa-check"></i><b>6.5.3</b> Step 3. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and the level of metabolites.</a></li>
<li class="chapter" data-level="6.5.4" data-path="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-4.-one-way-of-checking-if-the-metabolite-levels-indeed-correlate-with-your-data-is-to-see-if-subsetting-smaller-chunks-or-specific-chunks-of-the-data-will-still-maintain-the-metabolite-data-relationship-or-break-it."><i class="fa fa-check"></i><b>6.5.4</b> Step 4. One way of checking if the metabolite levels indeed correlate with your data, is to see if subsetting smaller chunks or specific chunks of the data will still maintain the metabolite-data relationship or break it.</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><i class="fa fa-check"></i><b>6.6</b> Try it Question 3 - What microbes (ASVs) differ between males and females, and does age have an impact</a>
<ul>
<li class="chapter" data-level="6.6.1" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3a.-perform-differential-abundance-analysis-between-genders."><i class="fa fa-check"></i><b>6.6.1</b> Step 3A. Perform differential abundance analysis between genders.</a></li>
<li class="chapter" data-level="6.6.2" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3b.-determine-the-impact-of-younger-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>6.6.2</b> Step 3B. Determine the impact of younger age on differential abundance in males and females, by subsetting age &lt; 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
<li class="chapter" data-level="6.6.3" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-3c.-determine-the-impact-of-older-age-on-differential-abundance-in-males-and-females-by-subsetting-age-50-data-from-the-phyloseq-object-and-performing-deseq2-analysis-on-this-subset."><i class="fa fa-check"></i><b>6.6.3</b> Step 3C. Determine the impact of older age on differential abundance in males and females, by subsetting age &gt;= 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.</a></li>
<li class="chapter" data-level="6.6.4" data-path="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html"><a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html#step-4.-select-data-for-poster-and-include-the-code-used-to-generate-the-figure"><i class="fa fa-check"></i><b>6.6.4</b> Step 4. Select data for poster and include the code used to generate the figure</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><i class="fa fa-check"></i><b>6.7</b> Try it Question 4 - Is there diet and age interaction and what microbes (ASVs) correlate with changes in diet-age interaction</a>
<ul>
<li class="chapter" data-level="6.7.1" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html#step-1.-plot-alpha-diversity-based-on-age-for-all-dietary-groups-and-then-for-bd-hd-and-wo."><i class="fa fa-check"></i><b>6.7.1</b> Step 1. Plot alpha diversity based on age for all dietary groups, and then for BD, HD and WO.</a></li>
<li class="chapter" data-level="6.7.2" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html#step-2.-perform-differential-abundance-analysis-comparing-hd-to-bd-for-the-population-in-general-and-then-for-older-individuals-specifically-because-older-individuals-produced-the-biggest-dietage-interaction-based-on-alpha-diversity."><i class="fa fa-check"></i><b>6.7.2</b> Step 2. Perform differential abundance analysis comparing HD to BD for the population in general, and then for older individuals specifically (because older individuals produced the biggest diet–age interaction based on alpha diversity).</a></li>
<li class="chapter" data-level="6.7.3" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html#step-2b.-perform-differential-abundance-analysis-between-hd-and-bd-for-older-individual-50-yo."><i class="fa fa-check"></i><b>6.7.3</b> Step 2B. Perform differential abundance analysis between HD and BD for older individual (&gt; 50 yo.)</a></li>
<li class="chapter" data-level="6.7.4" data-path="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html"><a href="try-it-question-4---is-there-diet-and-age-interaction-and-what-microbes-asvs-correlate-with-changes-in-diet-age-interaction.html#step-3.-select-data-for-poster-and-include-the-code-used-to-generate-the-figure.-1"><i class="fa fa-check"></i><b>6.7.4</b> Step 3. Select data for poster and include the code used to generate the figure.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Course Name</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=10.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/C-MOOR_Template/dasl_thin_main_image.png">
</div>
<div id="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals" class="section level2 hasAnchor" number="6.5">
<h2><span class="header-section-number">6.5</span> Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals<a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Approach: Perform multidimensional scaling (also known as principal component analysis) to establish a relationship between the samples given multivariate data (metadata variables). Using a PcOA plot (via commands ordinate() and plot_ordination() in phyloseq), you will condense the original high-dimensional data into a low-dimensional one by converting data to distance map (matrix) with 2 dimensions, x and y, that best explain variability in your data. From your PCoA plot you will assess the contribution of different study variables to sample diversity and identify variables that help explain sample diversity. In a PcOA plot, samples with similar microbial profiles will be plotted close and may appear as “clusters”.</p>
<div id="step-1.-make-a-pcoa-plot-ordinating-on-the-entire-miso-dataset-and-coloring-by-individual.-investigate-resulting-plot-shape." class="section level3 hasAnchor" number="6.5.1">
<h3><span class="header-section-number">6.5.1</span> Step 1. Make a PcOA plot, ordinating on the entire miso dataset, and coloring by individual. Investigate resulting plot shape.<a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-1.-make-a-pcoa-plot-ordinating-on-the-entire-miso-dataset-and-coloring-by-individual.-investigate-resulting-plot-shape." class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li><p>Ordination is a term used to summarize a multidimensional dataset when projected onto a low-dimensional space (like X &amp; Y axes) and then observing any pattern the data may possess with a visual inspection.</p></li>
<li><p>Subsequent coloring of the pattern with metadata variables can reveal underlying relationships between data and experiment variables.</p></li>
<li><p>color - by subject</p></li>
<li><p>Include a title for your PCoA plot</p></li>
<li><p>Use the following code as a template:</p></li>
</ul>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb28-1" tabindex="-1"></a><span class="co">#Ordinate </span></span>
<span id="cb28-2"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb28-2" tabindex="-1"></a>miso.pcoa <span class="ot">&lt;-</span> <span class="fu">ordinate</span>(miso, <span class="at">method=</span><span class="st">&quot;PCoA&quot;</span>, <span class="at">distance=</span><span class="st">&quot;bray&quot;</span>) </span>
<span id="cb28-3"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb28-3" tabindex="-1"></a></span>
<span id="cb28-4"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb28-4" tabindex="-1"></a><span class="co">#Plot PCoA</span></span>
<span id="cb28-5"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb28-5" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa, </span>
<span id="cb28-6"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb28-6" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb28-7"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb28-7" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">1A-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1A-2. Describe the shape of your PCoA plot. Does the PCoA organize the into groups? Is there any pattern?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1A-3. Do you observe sample clustering by individual? Provide examples and an explanation.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">1A-4. Offer one question you want to ask about this PCoA plot?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="step-2.-using-the-pcoa-plot-from-step-1-above-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-diet-gender-and-age" class="section level3 hasAnchor" number="6.5.2">
<h3><span class="header-section-number">6.5.2</span> Step 2. Using the PCoA plot from Step 1 above, look for any correlations between the shape of your PCoA plot and diet, gender and age<a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-2.-using-the-pcoa-plot-from-step-1-above-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-diet-gender-and-age" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Step 2A. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and diet?</strong></p>
<p>How well is your data explained by the variation in the diet - look for signs of correlation between PCoA shape and variable “timepointgroup” representing BD, HD and WO diets. Note that since we are using the same plot, we do not need to re-ordinate (no ordinate() function).</p>
<ul>
<li>Color - by timepointgroup</li>
<li>Include a title for your PcOA plot</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb29-1"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb29-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa, </span>
<span id="cb29-2"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb29-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb29-3"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb29-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2A-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2A-2 Did coloring the ‘miso’ PCoA plot by timepointgroup explain any variation in your data? E.g. did each of the 3 data groups/clusters correspond to each of the 3 timepointgroups or not? E.g. does the coloring appear randomly distributed or not?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2B. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and gender? </strong></p>
<p>How well is your data explained by the variation in the gender - look for signs of correlation between PcOA shape and variable “gender”.</p>
<ul>
<li>Color - by gender</li>
<li>Include a title for your PcOA plot</li>
<li>Use the following code as a template.</li>
</ul>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb30-1"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb30-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa, </span>
<span id="cb30-2"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb30-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb30-3"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb30-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2B-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2B-2 Did coloring of the ‘miso’ PCoA plot by gender explain any variation in your data? E.g. does the coloring appear randomly distributed or not?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2C. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and age? How well is your data explained by the variation in the age - look for signs of correlation between PcOA shape and variable “age”.</strong></p>
<ul>
<li>Color - age</li>
<li>Include title name for your PcOA plot</li>
<li>Use the following code as a template. To enhance the color range of the age variable, additionally included below (in blue) is the command to specify the color gradient with yellow–to-blue gradient.</li>
<li>Try running the code with and without the scale_colour_gradient() command to appreciate the difference.</li>
</ul>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb31-1"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb31-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa, </span>
<span id="cb31-2"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb31-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb31-3"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb31-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb31-4"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb31-4" tabindex="-1"></a><span class="fu">scale_colour_gradient</span>(<span class="at">low =</span> <span class="st">&quot;red&quot;</span>, <span class="at">high =</span> <span class="st">&quot;green&quot;</span>) </span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2C-1. Paste your plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2C-2 Did coloring of the ‘miso’ PCoA plot by age explain any variation in your data? E.g. Does the coloring appear randomly distributed or not? Is there any hint of a correlation between age and groups of data on the left or the right?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="step-3.-using-the-pcoa-plot-from-step-1-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-the-level-of-metabolites." class="section level3 hasAnchor" number="6.5.3">
<h3><span class="header-section-number">6.5.3</span> Step 3. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and the level of metabolites.<a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-3.-using-the-pcoa-plot-from-step-1-look-for-any-correlations-between-the-shape-of-your-pcoa-plot-and-the-level-of-metabolites." class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>How well is your data correlated with the variation in the 5 metabolites - Creatinine, PCS, IS, HIPP, or PAG? Test each metabolite by coloring each metabolite at a time, then, choose your favorite metabolite (e.g. one with most difference) and show the plot below.</p>
<ul>
<li>Color - metabolite one at a time: Creatinine, PCS, IS, HIPP, PAG</li>
<li>Include a title for your PcOA plot</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb32-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(miso, miso.pcoa fill <span class="cf">in</span> the blank metabolite name, </span>
<span id="cb32-2"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb32-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>,  </span>
<span id="cb32-3"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb32-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb32-4"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb32-4" tabindex="-1"></a><span class="fu">scale_colour_gradient</span>(<span class="at">low =</span> <span class="st">&quot;yellow&quot;</span>, <span class="at">high =</span> <span class="st">&quot;blue&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">3A-1. Paste your metabolite plot(s) below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3A-2. Which metabolite did you choose to show and why? Any difference between microbiomes for individuals with low versus high metabolite levels? Any association between metabolite level and microbiome diversity?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="step-4.-one-way-of-checking-if-the-metabolite-levels-indeed-correlate-with-your-data-is-to-see-if-subsetting-smaller-chunks-or-specific-chunks-of-the-data-will-still-maintain-the-metabolite-data-relationship-or-break-it." class="section level3 hasAnchor" number="6.5.4">
<h3><span class="header-section-number">6.5.4</span> Step 4. One way of checking if the metabolite levels indeed correlate with your data, is to see if subsetting smaller chunks or specific chunks of the data will still maintain the metabolite-data relationship or break it.<a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#step-4.-one-way-of-checking-if-the-metabolite-levels-indeed-correlate-with-your-data-is-to-see-if-subsetting-smaller-chunks-or-specific-chunks-of-the-data-will-still-maintain-the-metabolite-data-relationship-or-break-it." class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Using subset() command, subset out e.g. HD diet specifically, and then BD diet. Then, generate new PcOA plots and see if the correlation with your metabolite of interest still holds.</p>
<p><strong>Step 4A. Subset “HD”, ordinate on “HD”, and make a new PCoA plot.</strong></p>
<ul>
<li>Subset data - by the timepointgroup “HD”</li>
<li>Ordinate - misoHD</li>
<li>Color - by your selected metabolite from part 3</li>
<li>Include a title for your PcOA plot</li>
<li>Use the following code as a template:</li>
</ul>
<ol style="list-style-type: lower-alpha">
<li>Subset only “HD” timepoint from your miso data, creating a new phyloseq object called “misoHD”</li>
</ol>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb33-1" tabindex="-1"></a>misoHD <span class="ot">=</span> <span class="fu">subset_samples</span>(miso, timepointgroup <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>) </span></code></pre></div>
<ol start="2" style="list-style-type: lower-alpha">
<li>Ordinate using only “HD” subset, creating a new ordination matrix called “pcoa.misoHD”</li>
</ol>
<div class="sourceCode" id="cb34"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb34-1"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb34-1" tabindex="-1"></a>pcoa.misoHD <span class="ot">&lt;-</span> <span class="fu">ordinate</span>(misoHD, <span class="at">method=</span><span class="st">&quot;PCoA&quot;</span>, <span class="at">distance=</span><span class="st">&quot;bray&quot;</span>) </span></code></pre></div>
<ol start="3" style="list-style-type: lower-alpha">
<li>Make a PCoA plot using your “HD” subset.</li>
</ol>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb35-1" tabindex="-1"></a><span class="fu">plot_ordination</span>(misoHD, pcoa.misoHD, </span>
<span id="cb35-2"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb35-2" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>, </span>
<span id="cb35-3"><a href="try-it-question-2---do-diet-age-gender-and-levels-of-metabolites-correlate-with-microbe-variation-between-individuals.html#cb35-3" tabindex="-1"></a>        <span class="at">title=</span><span class="st">&quot;fill in the blank&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">4A-1. Paste your plot below for HD subset:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">4A-2 Did subsetting by the timpointgroup “HD” maintain your data’s relationship to your chosen metabolite? Did the shape of the plot change or stay the same with “HD” subsetting? Can you conclude if your metabolite of choice correlates with HD diet?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 4B. Repeat the analysis on only the BD timepoint group samples. Subset “BD”, ordinate on “BD”, and make a new PCoA plot.</strong></p>
<ul>
<li>Subset data - by the timepointgroup “BD”</li>
<li>Ordinate - misoBD</li>
<li>Color - by your selected metabolite from part 3</li>
<li>Include a title for your PcOA plot</li>
<li>Use the code above as a template.</li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">4A-1. Paste your plot below for BD subset:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Did subsetting “BD” maintain your data’s relationship to your chosen metabolitelevels? Did the shape of the plot change or stay the same with “BD” subsetting? Explain your results. Can you conclude if your metabolite of choice correlates with sample groups when only BD timepointgroup data is plotted?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 5A. Select data for poster and include the code used to generate the figure.</strong></p>
<ul>
<li>Prepare your final figure and associated code for the poster.</li>
<li>Figure suggestion: A 2-part figure with Figure A showing PCoA plot based on subject and Figure B, showing PCoA plot with your favorite metabolite (Creatinine, PCS, IS, HIPP, or PAG).</li>
<li>Ensure you have a figure legend that explains your final figure.</li>
<li>Ensure you submit the code you used to generate your final figure.</li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">5A-1. Paste final poster figure and code below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
</div>
<hr>
<center> 
  <div class="footer">
      All materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="try-it-question-1---how-sensitive-is-microbial-diversity-to-variables-like-diet-age-and-gender.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="try-it-question-3---what-microbes-asvs-differ-between-males-and-females-and-does-age-have-an-impact.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
