

<!-- Set up code of OTTR Book-->

```{r, include = FALSE}
ottrpal::set_knitr_image_path()
```

# Analyzing 16S rRNA Data with phyloseq

## Analyzing 16S rRNA Data with phyloseq

In this section we will do more advanced analyses in phyloseq. These include:

1. Profiling biodiveristy of sample through alpha diversity
1. Plotting samples in a multidimensional plot which will allow us to easily see similarities between samples
1. Performing a differential abundance analysis to compare the abundance of microbes between two conditions
1. Relating all the above to different metadata variables in our datasets such as subject, age, and gender

These analyses are slightly more complex to run and interpret, but students who have completed the [chapter on scientific literature](https://sayumiyork.github.io/miniCURE-16S_Test/scientific-literature.html) will have previous exposure to these figures.

## Lecture - Analyzing 16S rRNA Data with phyloseq

*Estimated time: *

```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Image test"}

ottrpal::include_slide("https://docs.google.com/presentation/d/1QZbSBPOGkBeizh1L45C6EaktOxl6pqrTDGNIncHQ5lY/edit#slide=id.g35f391192_00")

```

[Lecture](https://docs.google.com/presentation/d/1QZbSBPOGkBeizh1L45C6EaktOxl6pqrTDGNIncHQ5lY/edit?usp=sharing)


## Activity - Exploring 16S rRNA Data with phyloseq

### Purpose

To analyze bacterial diversity based on 16S rRNA gene sequencing using data from the “Impact of a 7-day homogeneous diet on interpersonal variation in human gut microbiomes and metabolomes” by Guthrie et al., 2023. This study is also referred to as the MISO study for “Microbiome Individuality and Stability Over Time” because the study aims to understand variation (or stability) of microbiomes across individuals.

This activity aims to analyze metagenomic diversity using the following R packages:

- **phyloseq**: A popular package for taxonomy profiling
- **DESeq2**: A package originally designed for differential expression which we will use for differential abundance
- **ggplot2**: A tidyverse package that produces high-quality figures

### Learning objectives

1. Use phyloseq to cluster samples based on the similarity of their microbial compositions using multidimensional scaling methods (NMDS or PCoA)
1. Use phyloseq to profile alpha diversity (Shannon and Simpson’s indices)
1. Use DESeq2 to perform differential abundance analysis


### Introduction

16S data can be manipulated and visualized in a variety of ways. In Google Sheets, we explored the data manually to gain an understanding at the level of 1-10 ASVs. Through phyloseq we collapsed ASVs with the same taxonomic annotation to survey microbial diversity and perform the same exploration across the entire dataset. In this activity, we will use the functions of phyloseq and DESeq2 that are more advanced. These analyses would be extremely difficult and tedious to do manually and will allow us deeper insights into our dataset. We will cluster samples based on similarity of ASV counts, survey alpha diversity, and search for differentially abundant ASVs between different groups.


### Activity 1 – Analyze 16S rRNA Data with phyloseq tutorial

*Estimated time: 35 min*

**Activity 1. Explore a phyloseq object through the “Analyze 16S rRNA Data with phyloseq” tutorial on SciServer.**

1. Access the C-MOOR Tutorials

- If you are using SciServer, log into SciServer, click on compute and open your “C-MOOR LearnR” container. Visit SciServer Guides and FAQs if you need to jog your memory on how to do this.

- If you are using AnVIL, log into AnVIL, navigate to your class Workspace, start up an RStudio Cloud Environment, and open RStudio. Visit the AnVIL Guides and FAQs if you need to jog your memory on how to do this. This module can be found in the “2-analyze-ps” folder of the “16s” curriculum folder.

- If you are using an alternative setup, follow the instructions provided by your instructor.

2. Start the “Analyze 16S rRNA Data with phyloseq” tutorial. Visit SciServer Guides and FAQs. If you need assistance accessing the tutorial.
3. To move through the activities click “Continue” at the bottom of the screen. When you are done with a topic, click “Next Topic” to move on.
4. This tutorial has small boxes in which you can enter and run short lines of code to analyze the data.
5. As you work through the tutorial, take snapshots of your work and paste your answers in the grey boxes below:

|**1-1. Alpha diversity section: Take a snapshot and paste the code for an alpha diversity plot (Simpson) from the quiz question: What male subject has the data point for the LOWEST alpha diversity? HINT: Use subset_samples() to subset males, and specify individuals (subject) on x-axis of the alpha diversity plot**|
|:-|
| <br> |

|**1-2. PCoA section: Take a snapshot and paste the code for a PCoA plot from the quiz question: In a PCoA with only data from ASVs with the class Bacteroidia, what is the percent of variance in the dataset explained by principal coordinate 1? HINT: You will need to change the code subsetting phylum and Firmicutes**|
|:-|
| <br> |

|**1-3. Differential abundance: Take a snapshot and paste the code for the differential abundance plot for the quiz question: Which of the following Phylum have ASVs that are differentially abundant between the subject S02 and subject S03?**|
|:-|
| <br> |

### Activity 2 – Try it out questions

*Estimated time: 60 min*

With your group, perform some exploratory data analysis selecting from one of the four questions below or coming up with your own question.  Each question has its own section with code templates for you to use below. Select a question here, then go to the page specific to your question.

#### Question 1. How sensitive is microbial diversity to variables like diet, age and gender?

Alpha diversity is a measure that estimates how the distribution of microbes changes due to a variable (or metadata category). Alpha diversity measures changes in the richness (the number of different organisms or ASVs) and evenness (how evenly are these organisms distributed in terms of their abundance). Using the “MISO” study dataset we will use Simpson (or specifically, Gini-Simpson) alpha diversity to evaluate changes in microbial diversity in individuals due to different metadata variables.  

Approach: Plot Simpson alpha diversity using plot_richness() command in phyloseq and assess the impact of different study variables on changes in microbial diversity. Identify variables that impact alpha diversity. Visible shifts in alpha diversity  suggest a shift in microbial diversity, and a higher alpha diversity value indicates an increase in alpha diversity (either richness or abundance).

1. Evaluate the impact of diet on alpha diversity by plotting  ASVs based on “diet” variable.
1. Evaluate the impact of individuality on alpha diversity by plotting ASVs based on “subject” variable.
1. Evaluate the impact of gender on alpha diversity by plotting ASVs based on “age” variable.
1. Evaluate the impact of gender on alpha diversity by plotting ASVs based on “gender” variable.
1. Evaluate the impact of gender on alpha diversity by plotting ASVs based on the 5 different levels of metabolites in the study (Creatinine, PCS, IS, HIPP, PAG).

#### Question 2. Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals?  

A PCoA plot is a principal coordinate analysis used to represent similarity between samples (sample microbiomes in our case). Using the “MISO” study dataset, we will use the PCoA plot  to summarize individuals based on ASVs and plot the resulting relationships between individuals. Based on how well color-coding the different variables matches the sample distribution on the PcOA plot, we will aim to help explain potential sources of sample similarity.

Approach: Perform multidimensional scaling (also known as principal component analysis) to establish a relationship between the samples given multivariate data (metadata variables). Using a PcOA plot (via commands ordinate() and plot_ordination() in phyloseq), you will condense the original high-dimensional data into a low-dimensional one by converting data to distance map (matrix) with 2 dimensions, x and y, that best explain variability in your data. From your PCoA plot you will assess the contribution of different study variables to sample diversity and identify variables that correlate with sample diversity. In a PcOA plot, samples with similar microbial profiles will be plotted close and may appear as groups.

1. Make a PCoA plot, ordinate on the entire dataset (all ASVs) and color by individual. Investigate the shape of the resulting PCoA plot. 
1. Correlate PCoA plot shape with metadata variables by coloring the PCoA plot with different variables including diet, subject, age and gender. Do any of the variables correlate with the shape of PCoA plot and data groupings?
1. Correlate PCoA plot shape with the levels of metabolites in the study (Creatinine, PCS, IS, HIPP, PAG). From the color-coding pattern, identify 1. which variables help potentially explain the data groups formed in the PCoA plot.
Subset “HD”, ordinate on “HD”, and make a new PCoA plot.

#### Question 3. What microbes (ASVs) differ between males and females, and does age have an impact?

Approach: Perform DESeq2 differential abundance analysis between females and males and determine how many differentially abundant microbes are there between the sexes. Then examine if age has a further impact on the differential abundance of the microbes between the sexes. 

1. Perform DESeq2 analysis between females and males and identify differences. 
1. Test if younger age contributes to differential microbe abundance between females and males by subsetting younger (< 50 years old) individuals.
1. Test if older  age contributes to differential microbe abundance between females and males by subsetting older (> 50 years old) individuals.

#### Question 4. Is there diet and age interaction and what microbes (ASVs) correlate with changes in diet-age interaction?

Variables and conditions can interact with each other in complex ways. Here we will try to tease apart the relationship between the gut microbiome, diet, and age. What does an interaction look like? In this example, we’re looking at whether or not a change in diet affects people of different ages in different ways. We might imagine for example that a young person’s gut is more resilient to change than an older person’s or vice versa. Changes at one age group may be different in the other. 

Approach: Use alpha diversity measure and DESeq2 tools to answer this question. Using alpha diversity, determine if there is an interaction between diet and age. Then use DESeq2 to see if any ASVs are associated with changes in die-age interaction. Using Simpson alpha diversity measure (or specifically, Gini-Simpson) evaluate how microbial diversity changes with age and diet in general, or age and BD, HD and WO diet specifically. Then, use DESeq2 to evaluate if younger or older age changes ASVs associated with diet.

1. Plot alpha diversity based age for the population in general, and then for individuals subsetted for BD, HD and WO diets. Look for shifts in alpha diversity with change in diet.
1. Perform DESeq2 analysis based on the diet for the population in general, establishing baseline differential abundance between dietary groups HD and BD.
1. Perform DESeq2 analysis based on the diet for the younger (<= 50 yo) and older (>=50 yo) individuals, evaluating age-associated changes in differential abundance between dietary groups HD and BD.



### Grading criteria

- Download the assignment to your local computer as a .docx, complete it, and upload the assignment to your LMS (Blackboard, Canvas, Google Classroom).

### Footnotes

#### Resources

- [Google Doc](https://docs.google.com/document/d/1OWgUwaT2MlSd-qq7wSlB3BnM-cq_GTxEyy38HV5ajoU/edit?usp=sharing)
- [16S rRNA R Quick Reference](https://docs.google.com/document/d/1pYTq_U-e3Fath5yB6N9b5ic-cMynUtclkEgdojkAXwE/edit?usp=sharing)

#### Contributions and affiliations

- Valeriya Gaysinskaya, Johns Hopkins University
- Gauri Paul, Clovis Community College
- Frederick Tan, Johns Hopkins University
- Sayumi York, Notre Dame of Maryland University


## Try it Question 1 - How sensitive is microbial diversity to variables like diet, age and gender

Alpha diversity is a measure that estimates how the distribution of microbes changes due to a variable (or metadata category). Alpha diversity measures changes in the richness (the number of different organisms or ASVs)  and evenness (how evenly are these organisms distributed in terms of their abundance). Using the “MISO” study dataset we will use Simpson (or specifically, Gini-Simpson) alpha diversity to evaluate changes in microbial diversity in individuals due to different metadata variables.  

Approach: Plot Simpson alpha diversity using plot_richness() command in phyloseq and assess the impact of different study variables on changes in microbial diversity. Identify the variables that impact alpha diversity. Visible shifts in alpha diversity measure suggests a shift in microbial diversity, and higher alpha diversity value indicates an increase in alpha diversity.

**Note, when plotting this data you may get a warning below. That is ok!**

```{r, eval=FALSE}
Warning: The data you have provided does not have any singletons. This is highly suspicious. Results of richness estimates (for example) are probably unreliable, or wrong, if you have already trimmed low-abundance taxa from the data.We recommended that you find the un-trimmed data and retry.
```

- Refer to the “alpha diversity” section of the  “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_richness() function.

### Step  1A. Plot alpha diversity of the full MISO dataset by subject (individuals).

- subject is on the x-axis
- Color is by subject
- Use the following code as a template:

```{r, eval=FALSE}
plot_richness(miso_counts, x="fill in the blank", 
              color="subject", 
              measures= "Simpson")
```

|**1A-1. Paste your plot below:**|
|:--|
| <br> |

|**1A-2. Examine the alpha diversity plot you made. Are the points for an individual highly clustered or far apart?**|
|:--|
| <br> |

|**1A-3. Is the alpha diversity metric for one individual similar to another’s?**|
|:--|
| <br> |

|**1A-4. When the points for an individual are highly clustered, this means their gut microbiome has the same level of biodiversity over the length of the experiment (although the exact community of microbes may not be the same). Conversely, when the points are farther apart, their gut microbiome’s biodiversity has changed through the experiment. What does it mean to see a varied pattern of clustering across individuals in the study?**|
|:--|
| <br> |

|**1A-5. Do you think a healthy microbiome has a high or low alpha diversity? Why or why not?**|
|:--|
| <br> |


### Step  1B. Plot alpha diversity of the full MISO dataset by diet

- diet is on the x-axis
- Color is by subject
- Use the following code as a template:


```{r, eval=FALSE}
plot_richness(miso_counts, x="fill in the blank", 
              color="fill in the blank", 
              measures= "Simpson")
```

|**1B-1. Paste your plot below:**|
|:--|
| <br> |

|**1B-2. Based on the plot above, does diet change the overall alpha diversity metric of the gut microbiome (ex. Are the points higher or lower overall in one diet versus the others?)**|
|:--|
| <br> |

|**1B-3. Based on the plot above does diet change how varied the alpha diversity is between individuals? (ex. Are the points more spread out in one diet versus the others?)**|
|:--|
| <br> |

|**1B-4. Do you think diet has an impact on gut microbial alpha diversity? **|
|:--|
| <br> |

### Step  1C. Plot alpha diversity of the full MISO dataset by age.

- age is on the x-axis
- Color is by subject
- Use the following code as a template:

```{r, eval=FALSE}
plot_richness(miso_counts, x="fill in the blank", 
              color="fill in the blank", 
              measures= "Simpson")
```

|**1C-1. Paste your plot below:**|
|:--|
| <br> |

|**1C-2. Do younger or older individuals have a higher alpha diversity of the gut microbiome overall (ex. Which group has an overall higher alpha diversity measure)?**|
|:--|
| <br> |

|**1C-3. Do younger or older individuals have a more consistent alpha diversity of the gut microbiome (ex. Are points clustered more closely in younger or older subjects)?**|
|:--|
| <br> |

|**1C-4. What do you think the impact of age on the alpha diversity of the gut microbiome is?**|
|:--|
| <br> |

### Step  1D. Plot alpha diversity of the full MISO dataset by gender.

- gender is on the x-axis
- Color is by subject
- Use the following code as a template:

```{r, eval=FALSE}
plot_richness(miso_counts, x="fill in the blank", 
              color="fill in the blank", 
              measures= "Simpson")
```


|**1D-1. Paste your plot below:**|
|:--|
| <br> |

|**1D-2. Do males or females have a higher alpha diversity of the gut microbiome overall (ex. Which group has an overall higher alpha diversity measure)?**|
|:--|
| <br> |

|**1D-3. Do males or females have a more consistent alpha diversity of the gut microbiome (ex. Are points clustered more closely in males or females)?**|
|:--|
| <br> |

|**1D-4. What do you think the impact of gender on the alpha diversity of the gut microbiome is?**|
|:--|
| <br> |


### Step  2. Plot alpha diversity of the full MISO dataset, grouping based on levels for 5 different metabolites: Creatinine, PCS, IS, HIPP, PAG.

Plot alpha diversity for the 5 metabolites independently then choose your favorite metabolite (e.g. one with most difference) and show plot below. 

- metabolite on the x-axis  (one at a time: Creatinine, PCS, IS, HIPP, PAG)
- Color is by subject
- Use the following code as a template:

```{r, eval=FALSE}
plot_richness(miso_counts, x="fill in the blank metabolite name", 
              color="fill in the blank", 
              measures= "Simpson")
```

|**2A-1. Paste your metabolite plot(s) below:**|
|:--|
| <br> |

|**2A-2. Which metabolite did you choose to show and why? **|
|:--|
| <br> |

|**2A-3. Are there any trends in alpha diversity as this metabolite increases in level?**|
|:--|
| <br> |


### Footnotes

#### Resources

- [Google Doc](https://docs.google.com/document/d/1OWgUwaT2MlSd-qq7wSlB3BnM-cq_GTxEyy38HV5ajoU/edit?usp=sharing)
- [16S rRNA R Quick Reference](https://docs.google.com/document/d/1pYTq_U-e3Fath5yB6N9b5ic-cMynUtclkEgdojkAXwE/edit?usp=sharing)

#### Contributions and affiliations

- Valeriya Gaysinskaya, Johns Hopkins University
- Gauri Paul, Clovis Community College
- Frederick Tan, Johns Hopkins University
- Sayumi York, Notre Dame of Maryland University

## Try it Question 2 - Do diet, age, gender and levels of metabolites correlate with microbe variation between individuals 

A PCoA plot is a principal coordinate analysis used to represent similarity between samples (sample microbiomes in our case). Using the “MISO” study dataset, we will use the PCoA plot  to summarize individuals based on ASVs and plot the resulting relationships between individuals. Based on how well color-coding the different variables matches the sample distribution on the PcOA plot, we will aim to help explain potential sources of sample similarity.

Approach: Perform multidimensional scaling (also known as principal component analysis) to establish a relationship between the samples given multivariate data (metadata variables). Using a PcOA plot (via commands ordinate() and plot_ordination() in phyloseq), you will condense the original high-dimensional data into a low-dimensional one by converting data to distance map (matrix) with 2 dimensions, x and y, that best explain variability in your data. From your PCoA plot you will assess the contribution of different study variables to sample diversity and identify variables that help explain sample diversity. In a PcOA plot, samples with similar microbial profiles will be plotted close and may appear as “clusters”.

### Step  1. Make a PcOA plot, ordinating on the entire miso dataset, and coloring by individual. Investigate resulting plot shape.

- Ordination is a term used to summarize a multidimensional dataset when projected onto a low-dimensional space (like X & Y axes) and then observing any pattern the data may possess with a visual inspection. 
- Subsequent coloring of the pattern with metadata variables can reveal underlying relationships between data and experiment variables. 

- color - by subject
- Include a title for your PCoA plot
- Use the following code as a template:

```{r, eval=FALSE}
#Ordinate 
miso.pcoa <- ordinate(miso, method="PCoA", distance="bray") 

#Plot PCoA
plot_ordination(miso, miso.pcoa, 
        color = "fill in the blank",  
        title="fill in the blank")
```


|**1A-1. Paste your plot below:**|
|:--|
| <br> |

|**1A-2. Does the PCoA organize samples into groups?  Is there any pattern?**|
|:--|
| <br> |

|**1A-3. Do you observe any sample clustering by individual? This is represented by points of the same color clustering together.  If so, give an example of an individual who shows this pattern.**|
|:--|
| <br> |

|**1A-4. Offer one question you want to ask about this PCoA plot?**|
|:--|
| <br> |

### Step  2. Using the PCoA plot from Step 1 above, look for any correlations between the shape of your PCoA plot and diet, gender and age

**Step 2A. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and diet?** 

How well is your data explained by the variation in the diet - look for signs of correlation between PCoA shape and variable “diet” representing BD, HD and WO diets. Note that since we are using the same plot, we do not need to re-ordinate (no ordinate() function).

- Color - by diet
- Include a title for your PcOA plot
- Use the following code as a template:


```{r, eval=FALSE}
#Ordinate 
miso.pcoa <- ordinate(miso, method="PCoA", distance="bray") 

plot_ordination(miso, miso.pcoa, 
        color = "fill in the blank",  
        title="fill in the blank")
```

|**2A-1. Paste your plot below:**|
|:--|
| <br> |

|**2A-2 Do samples of the same diet treatment (color) cluster together?**|
|:--|
| <br> |

|**2A-3 Based on what you wrote above, are samples from the same diet treatment similar to each other?**|
|:--|
| <br> |


**Step  2B. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and gender? ** 

How well is your data explained by the variation in the gender - look for signs of correlation between PcOA shape and variable “gender”. 

- Color - by gender
- Include a title for your PcOA plot
- Use the following code as a template. 


```{r, eval=FALSE}
#Ordinate 
miso.pcoa <- ordinate(miso, method="PCoA", distance="bray") 

plot_ordination(miso, miso.pcoa, 
        color = "fill in the blank",  
        title="fill in the blank")
```

|**2B-1. Paste your plot below:**|
|:--|
| <br> |

|**2B-2 Are samples from the same gender similar to each other?**|
|:--|
| <br> |

**Step  2C. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and age? How well is your data explained by the variation in the age - look for signs of correlation between PcOA shape and variable “age”. ** 

- Color - age
- Include title name for your PcOA plot
- Use the following code as a template. To enhance the color range of the age variable, additionally included below (in blue) is the command to specify the color gradient with yellow–to-blue gradient. 
- Try running the code with and without the scale_colour_gradient() command to appreciate the difference.

```{r, eval=FALSE}

#Ordinate 
miso.pcoa <- ordinate(miso, method="PCoA", distance="bray") 

plot_ordination(miso, miso.pcoa, 
        color = "fill in the blank",  
        title="fill in the blank") + 
scale_colour_gradient(low = "blue", high = "yellow") 

```

|**2C-1. Paste your plot below:**|
|:--|
| <br> |

|**2C-2 Are samples of a similar age similar to each other?**|
|:--|
| <br> |
<br>

### Step 3. Using the PCoA plot from Step 1, look for any correlations between the shape of your PCoA plot and the level of metabolites. 

How well is your data correlated with the variation in the 5 metabolites: Creatinine, PCS, IS, HIPP, or PAG? Test each metabolite by coloring each metabolite at a time, then, choose your favorite metabolite (e.g. one with most difference) and show the plot below. 

- Color - metabolite one at a time: Creatinine, PCS, IS, HIPP, PAG
- Include a title for your PcOA plot
- Use the following code as a template:

```{r, eval=FALSE}

#Ordinate 
miso.pcoa <- ordinate(miso, method="PCoA", distance="bray") 

plot_ordination(miso, miso.pcoa, 
        color = "fill in the blank",  
        title="fill in the blank") + 
scale_colour_gradient(low = "blue", high = "yellow") 

```

|**3A-1. Paste your metabolite plot(s) below:**|
|:--|
| <br> |
<br>

|**3A-2. Which metabolite did you choose to show and why?**|
|:--|
| <br> |
<br>

|**3A-3. Are samples of the same metabolite level similar to each other?**|
|:--|
| <br> |
<br>


### Step 4. One way of checking if the metabolite levels indeed correlate with your data, is to see if subsetting smaller chunks or specific chunks of the data will still maintain the metabolite-data relationship or break it. 

Using the subset() command, subset out e.g. HD diet specifically, and then BD diet. Then, generate new PCoA plots and see if the correlation with your metabolite of interest still holds. 


**Step 4A. Subset “HD”, ordinate on “HD”, and make a new PCoA plot.**

- Subset data - by the diet “HD”
- Ordinate  - misoHD
- Color - by your selected metabolite from part 3
- Include a title for your PcOA plot
- Use the following code as a template:

```{r, eval=FALSE}
#Subset only “HD” timepoint from your miso data, creating a new phyloseq object called “misoHD”
misoHD = subset_samples(miso, diet == "fill in the blank") 

#Ordinate using only “HD” subset, creating a new ordination matrix called “pcoa.misoHD”
pcoa.misoHD <- ordinate(misoHD, method="PCoA", distance="bray") 

#Make a PCoA plot using your  “HD” subset.
plot_ordination(misoHD, pcoa.misoHD, 
        color = "fill in the blank", 
        title="fill in the blank")
```

|**4A-1. Paste your plot below for HD subset:**|
|:--|
| <br> |

|**4A-2 Are samples of the same metabolite level similar to each other considering only samples from the HD diet?**|
|:--|
| <br> |

|**4A-3 Has the relationship between the metabolite and samples changed within the HD diet compared to the entire dataset? (ex. Have any new patterns emerged or disappeared?)**|
|:--|
| <br> |


**Step 4B. Repeat the analysis on only the BD diet samples. Subset “BD”, ordinate on “BD”, and make a new PCoA plot.**

- Subset data - by the diet “BD”
- Ordinate  - misoBD
- Color - by your selected metabolite from part 3
- Include a title for your PcOA plot
- Use the code below as a template.

```{r, eval=FALSE}
misoBD = subset_samples(miso, diet == "fill in the blank")

pcoa.misoBD <- ordinate(misoBD, method="PCoA", distance="bray") 

plot_ordination(misoBD, pcoa.misoBD, 
        color = "fill in the blank", 
        title="fill in the blank")
```

|**4B-1. Paste your plot below for BD subset:**|
|:--|
| <br> |

|**4B-2 Are samples of the same metabolite level similar to each other considering only samples from the BD diet?**|
|:--|
| <br> |

|**4B-3 Has the relationship between the metabolite and samples changed within the BD diet compared to the entire dataset? (ex. Have any new patterns emerged or disappeared?)**|
|:--|
| <br> |


### Footnotes

#### Resources

- [Google Doc](https://docs.google.com/document/d/1OWgUwaT2MlSd-qq7wSlB3BnM-cq_GTxEyy38HV5ajoU/edit?usp=sharing)
- [16S rRNA R Quick Reference](https://docs.google.com/document/d/1pYTq_U-e3Fath5yB6N9b5ic-cMynUtclkEgdojkAXwE/edit?usp=sharing)

#### Contributions and affiliations

- Valeriya Gaysinskaya, Johns Hopkins University
- Gauri Paul, Clovis Community College
- Frederick Tan, Johns Hopkins University
- Sayumi York, Notre Dame of Maryland University

## Try it Question 3 - What microbes (ASVs) differ between males and females, and does age have an impact

Approach: Perform DESeq analysis between females and males and determine how many differentially abundant microbes are there between the sexes. Then examine if age has a further impact on the abundance of the microbes between the sexes. 

### Step 3A. Perform differential abundance analysis between genders.

- Design -  is based on gender (no double quotes)
- Groups to compare -   females, F  and males, M (baseline) 
- Plot Phylum on the X-axis and color by Class

- Make sure your graph has a title that includes the comparison you are making (ex. WO vs HD)
- Refer to the “Differential abundance” section of the “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_ordination() function.
- Use the code below as a template:

```{r, eval=FALSE}

# STEP 2: Select the groups to compare, where the latter group is your baseline
my_comparison <-c("gender", "fill in the blank", "fill in the blank (baseline)")

# STEP 3: Run the differential abundance analysis 
Significant_DEseq2_ASVs<-Differential_Abundance(miso_counts, DESeq2, my_comparison, 0.05)

# STEP 4: Retrieve the list of ASVs with a significant difference in abundance between the chosen groups
Significant_DEseq2_ASVs

# STEP 5: Plot the results with your chosen x axis and legend
ggplot(significant_ASVs, aes(x = fill in the blank, y=log2FoldChange, color= fill in the blank)) + geom_point(size=4, position="jitter") +
  theme(axis.text.x = element_text(angle = -90, hjust = 0, vjust=0.5))+
  ggtitle("fill in the blank")
```

|**3A-1. Insert the resulting plot below:**|
|:--|
| <br> |

|**3A-2. How many differentially abundant ASVs were identified? This is the number of rows in the result table.**|
|:--|
| <br> |

|**3A-3. Give an example of a differentially abundant ASVs that was significantly higher in females than males. Include ASV ID and taxa. HINT: If male was your baseline, then an example of an ASV that is higher in females will have a positive log2FC, and vice versa if female was your baseline group. Use the arrows in the header row of the table to scroll through the taxonomy of an ASV.**|
|:--|
| <br> |

|**3A-4. Look up this microbe. What function does it serve in the human gut, and how might its abundance be connected to gender?**|
|:--|
| <br> |

|**3A-5. Give an example of a differentially abundant ASVs that was significantly higher in males than females. Include ASV ID and taxa. HINT: If male was your baseline, then an example of an ASV that is higher in males will have a negative log2FC, and vice versa if female was your baseline group. Use the arrows in the header row of the table to scroll through the taxonomy of an ASV.**|
|:--|
| <br> |

|**3A-6. Look up this microbe. What function does it serve in the human gut, and how might its abundance be connected to gender?**|
|:--|
| <br> |


### Step 3B. Determine the impact of younger age on differential abundance in males and females, by subsetting age < 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.

- Subset phyloseq object to only include younger individuals (age < 50)
- For DESeq2 analysis, keep the design the same as in part A - based on gender
- Groups to compare:  females (F) and males (M, baseline)
- Plot Phylum on the X-axis and color by Class
- Use template code below:


```{r, eval=FALSE}

# STEP 1: subset by age < 50
ageA = subset_samples(miso_counts, age <50) 

# STEP 2: Convert the phyloseq object to a DESeq2 object and specify experimental design
DESeq2 <- phyloseq_to_deseq2(ageA, design = ~ fill in the blank)

# STEP 3: Select the groups to compare, where log2FoldChange reported will correspond to  y/x.
my_comparison <-c("gender", "fill in the blank", "fill in the blank (baseline)")

# STEP 4: Run the differential abundance analysis 
Significant_DEseq2_ASVs<-Differential_Abundance(miso_counts, DESeq2, my_comparison, 0.05)

# STEP 5: Retrieve the list of ASVs with a significant difference in abundance between the chosen groups
Significant_DEseq2_ASVs

# STEP 6: Plot the results with your chosen x axis and legend
ggplot(Significant_DEseq2_ASVs, aes(x = fill in the blank, y=log2FoldChange, color= fill in the blank)) + geom_point(size=4, position = "jitter") +
  theme(axis.text.x = element_text(angle = -90, hjust = 0, vjust=0.5))+
  ggtitle("fill in the blank")
```

|**3B-1. Insert the resulting plot below:**|
|:--|
| <br> |

|**3B-2. How many differentially abundant ASVs were identified?**|
|:--|
| <br> |

|**3B-3. Give an example of a differentially abundant ASVs that was significantly higher in younger females than younger males. Include ASV ID and taxa. HINT: If male was your baseline, then an example of an ASV that is higher in females will have a positive log2FC,  and vice versa if female was your baseline group. Use the arrows in the header row of the table to scroll through the taxonomy of an ASV.**|
|:--|
| <br> |

|**3B-4. Give an example of a differentially abundant ASVs that was significantly higher in younger males than younger females. Include ASV ID and taxa. HINT: If male was your baseline, then an example of an ASV that is higher in males will have a negative log2FC,  and vice versa if female was your baseline group. Use the arrows in the header row of the table to scroll through the taxonomy of an ASV.**|
|:--|
| <br> |

|**3B-5. Compared to the overall dataset, do younger individuals have more or less differentially abundant microbes between genders?**|
|:--|
| <br> |


### Step 3C. Determine the impact of older age on differential abundance in males and females, by subsetting age >= 50 data from the phyloseq object, and performing DESeq2 analysis on this subset.

- You can use template code below:

```{r, eval=FALSE}
# STEP 1: subset by age >=  50
ageB = subset_samples(miso_counts, age >= 50) 

# STEP 2: Convert the phyloseq object to a DESeq2 object and specify experimental design
DESeq2 <- phyloseq_to_deseq2(ageB, design = ~ fill in the blank)

# STEP 3: Select the groups to compare, where log2FoldChange reported will correspond to  y/x.
my_comparison <-c("gender", "fill in the blank", "fill in the blank")

# STEP 4: Run the differential abundance analysis 
Significant_DEseq2_ASVs<-Differential_Abundance(miso_counts, DESeq2, my_comparison, 0.05)

# STEP 5: Retrieve the list of ASVs with a significant difference in abundance between the chosen groups
Significant_DEseq2_ASVs

# STEP 6: Plot the results with your chosen x axis and legend
ggplot(Significant_DEseq2_ASVs, aes(x = fill in the blank, y=log2FoldChange, color= fill in the blank)) + geom_point(size=4, position = "jitter") +
  theme(axis.text.x = element_text(angle = -90, hjust = 0, vjust=0.5))+
  ggtitle("fill in the blank")
```

|**3C-1. Insert the resulting plot below:**|
|:--|
| <br> |

|**3C-2. How many differentially abundant ASVs were identified? This is the number of rows in the result table.**|
|:--|
| <br> |

|**3C-3. Compared to the overall dataset, do older individuals have more or less differentially abundant microbes between genders?**|
|:--|
| <br> |

|**3C-4. Compare all the plots you have made. Give a specific example of one difference you see.**|
|:--|
| <br> |



### Footnotes

#### Resources

- [Google Doc](https://docs.google.com/document/d/1OWgUwaT2MlSd-qq7wSlB3BnM-cq_GTxEyy38HV5ajoU/edit?usp=sharing)
- [16S rRNA R Quick Reference](https://docs.google.com/document/d/1pYTq_U-e3Fath5yB6N9b5ic-cMynUtclkEgdojkAXwE/edit?usp=sharing)

#### Contributions and affiliations

- Valeriya Gaysinskaya, Johns Hopkins University
- Gauri Paul, Clovis Community College
- Frederick Tan, Johns Hopkins University
- Sayumi York, Notre Dame of Maryland University

## Try it Question 4 -  Is there an interaction between diet and age and the microbiome

Variables and conditions can interact with each other in complex ways. Here we will try to tease apart the relationship between the gut microbiome, diet, and age. What does an interaction look like? In this example, we’re looking at whether or not a change in diet affects people of different ages in different ways. We might imagine for example that a young person’s gut is more resilient to change than an older person’s or vice versa. Changes at one age group may be different in the other. 

Approach: Use alpha diversity measure and DESeq2 tools to answer this question. Using alpha diversity, determine if there is an interaction between diet and age. Then use DESeq2 to see if any ASVs are associated with changes in die-age interaction. Using Simpson alpha diversity measure (or specifically, Gini-Simpson) evaluate how microbial diversity changes with age and diet in general, or age and BD, HD and WO diet specifically. Remember, shifts in alpha diversity measure suggest a shift in microbial diversity, with high alpha diversity suggesting high microbial diversity. Then, use DESeq2 to evaluate if ageing changes ASVs associated with diet.

Note, when plotting this data you may get a warning below. That is ok!

```{r, eval=FALSE}
Warning: The data you have provided does not have any singletons. This is highly suspicious. Results of richness estimates (for example) are probably unreliable, or wrong, if you have already trimmed low-abundance taxa from the data.We recommended that you find the un-trimmed data and retry.
```

- Refer to the “alpha diversity” section of the  “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_richness() function.

### Step  1. Plot alpha diversity based on age for all dietary groups, and then for BD, HD and WO.

**Step  1A. Plot alpha diversity based on age for all dietary groups, and then for BD, HD and WO.**

Plot alpha diversity based on age for all dietary groups.**

- Subset data - None
- age is on the x-axis
- Color is by subject
- Use the following code as a template:

```{r, eval=FALSE}
plot_richness(miso_counts, x="fill in the blank",
          	color="fill in the blank",
          	title = "miso",
          	measures= c("Simpson"))
```

|**1A-1. Paste your plot below:**|
|:--|
| <br> |


|**1A-2. Does alpha diversity change in value with increasing age (overall alpha diversity measure) ? **|
|:--|
| <br> |


|**1A-3. Does alpha diversity grow more varied between individuals with increasing age (are points less clustered with increasing age) ?**|
|:--|
| <br> |

**Step 1B. Plot alpha diversity based on age for BD diet only.**

Plot alpha diversity based on age for BD diet only.

- Subset data - diet BD
- age is on the x-axis
- Color is by subject
- Use the following code as a template:


```{r, eval=FALSE}
# Subset only BD samples
miso_counts_BD = subset_samples(miso_counts, diet == "BD")

plot_richness(miso_counts_BD, x="fill in the blank",
          	color="fill in the blank",
          	title = "fill in the blank",
          	measures= c("Simpson"))
```

|**1B-2.Paste your plot below:**|
|:--|
| <br> |

|**1B-2. Did the relationship between age and alpha diversity change when only looking at the BD diet data points? Explain your answer by describing the trend or lack thereof in the graph. **|
|:--|
| <br> |


**Step 1C. Plot alpha diversity based on age for HD diet only.**

- Subset data - diet HD
- age is on the x-axis
- Color is by subject
- Use the following code as a template:

```{r, eval=FALSE}
#Step1: Subset data for HD only
miso_counts_HD = subset_samples(miso_counts, diet == "fill in the blank")

plot_richness(miso_counts_HD, x="fill in the blank",
          	color="fill in the blank",
          	title = "fill in the blank",
          	measures= c("Simpson"))
```

|**1C-1. Paste your plot below:**|
|:--|
| <br> |

|**1C-2. Did the relationship between age and alpha diversity change when only looking at the HD diet data points? Explain your answer by describing the trend or lack thereof in the graph.**|
|:--|
| <br> |

**1D-1. Plot alpha diversity based on age for WO diet only.**

- Subset data - diet WO
- age is on the x-axis
- Color is by subject
- Use the following code as a template:


```{r, eval=FALSE}
#Step1: Subset data for WO only
miso_counts_WO = subset_samples(miso_counts, diet == "fill in the blank")

plot_richness(fill in the blank, x="fill in the blank",
          	color="fill in the blank",
          	title = "fill in the blank",
          	measures= c("Simpson"))
```

|**1D-1. Paste your plot below:**|
|:--|
| <br> |

|**1D-2. Did the relationship between age and alpha diversity change when only looking at the WO diet data points? Explain your answer by describing the trend or lack thereof in the graph.  Remember, the WO only has half as many datapoints compared to the BD and HD since there is only 1 WO timepoint (versus 2 for BD and HD).**|
|:--|
| <br> |

### Step  2. Perform differential abundance analysis comparing HD to BD for the population in general, and then for older individuals specifically.

**Step 2A. Perform differential abundance analysis between HD and BD.** 

- Design -  is based on diet (no double quotes)
- Groups to compare -   HD and BD (baseline) 
- Plot Phylum on the X-axis and color by Class

- Make sure your graph has a title that includes the comparison you are making (ex. WO vs HD)
- Refer to the “Differential abundance” section of the “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_ordination() function.
- Use the code below as a template:

```{r, eval=FALSE}

# STEP 1: Convert the phyloseq object to a DESeq2 object and specify experimental design
DESeq2 <- phyloseq_to_deseq2(miso_counts, design = ~ diet)

# STEP 2: Select the groups to compare, where the latter group is your baseline
my_comparison <-c("diet", "HD", "fill in the blank (baseline)")

# STEP 3: Run the differential abundance analysis 
Significant_DEseq2_ASVs<-Differential_Abundance(miso_counts, DESeq2, my_comparison, 0.05)

# STEP 4: Retrieve the list of ASVs with a significant difference in abundance between the chosen groups
Significant_DEseq2_ASVs

# STEP 5: Plot the results with your chosen x axis and legend
ggplot(Significant_DEseq2_ASVs, aes(x = Phylum, y=log2FoldChange, color= fill in the blank)) + geom_point(size=4) +
  theme(axis.text.x = element_text(angle = -90, hjust = 0, vjust=0.5))+
  ggtitle("fill in the blank")
```

|**2A-1. Insert the resulting plot below:**|
|:--|
| <br> |

|**2A-2. How many differentially abundant ASVs were identified? This is the number of rows in the result table.**|
|:--|
| <br> |

|**2A-3. Did a change in diet from BD to HD cause there to be a lot of changes in microbe abundance?**|
|:--|
| <br> |

### Step 2B. Perform differential abundance analysis between HD and BD for older individual (> 50 yo.) 

- Subset data - older age (> 50 yo)
- Design -  is based on diet (no double quotes)
- Groups to compare -   HD and BD (baseline) 
- Plot Phylum on the X-axis and color by Class

- Make sure your graph has a title that includes the comparison you are making (ex. WO vs HD)
- Refer to the “Differential abundance” section of the “Analyze 16S rRNA Data with phyloseq” tutorial for help using the plot_ordination() function.
- Use the code below as a template:

```{r, eval=FALSE}

#First, subset miso_counts object to only include older individuals over age 50
subset = subset_samples(miso_counts, age >50)

# STEP 1: Convert the phyloseq object to a DESeq2 object and specify experimental design
DESeq2_subset <- phyloseq_to_deseq2(subset, design = ~ diet)

# STEP 2: Select the groups to compare, where the latter group is your baseline
comparison_subset <-c("diet", "HD", "fill in the blank (baseline)")

# STEP 3: Run the differential abundance analysis 
Significant_ASVs_subset <- Differential_Abundance(miso_counts, DESeq2_subset, comparison_subset, 0.05)

# STEP 4: Retrieve the list of ASVs with a significant difference in abundance between the chosen groups
Significant_ASVs_subset

# STEP 5: Plot the results with your chosen x axis and legend
ggplot(Significant_ASVs_subset, aes(x = Phylum, y=log2FoldChange, color= fill in the blank)) + geom_point(size=4) +
  theme(axis.text.x = element_text(angle = -90, hjust = 0, vjust=0.5))+
  ggtitle("fill in the blank")
```

|**2B-1. Insert the resulting plot below:**|
|:--|
| <br> |

|**2B-2. How many differentially abundant ASVs were identified? This is the number of rows in the result table.**|
|:--|
| <br> |

|**2B-3. Did a change in diet from BD to HD cause there to be a lot of changes in microbe abundance in older individuals?**|
|:--|
| <br> |

|**2B-4. How do these results compare to the results when we looked at the entire dataset (2A-1)?**|
|:--|
| <br> |

|**2B-5. When this analysis is repeated for a younger population (<=50), 3 differentially abundant ASVs are found. Do you think there is a diet-age interaction? A reminder that an interaction would cause different age groups to react to the change in diet differently.**|
|:--|
| <br> |



### Footnotes

#### Resources

- [Google Doc](https://docs.google.com/document/d/1OWgUwaT2MlSd-qq7wSlB3BnM-cq_GTxEyy38HV5ajoU/edit?usp=sharing)
- [16S rRNA R Quick Reference](https://docs.google.com/document/d/1pYTq_U-e3Fath5yB6N9b5ic-cMynUtclkEgdojkAXwE/edit?usp=sharing)

#### Contributions and affiliations

- Valeriya Gaysinskaya, Johns Hopkins University
- Gauri Paul, Clovis Community College
- Frederick Tan, Johns Hopkins University
- Sayumi York, Notre Dame of Maryland University
