<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5.3 Activity - Exploring 16S rRNA Data with phyloseq | Course Name</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="5.3 Activity - Exploring 16S rRNA Data with phyloseq | Course Name" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5.3 Activity - Exploring 16S rRNA Data with phyloseq | Course Name" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  



<meta name="date" content="2025-07-04" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/dasl_favicon.ico" type="image/x-icon" />
<link rel="prev" href="lecture---exploring-16s-rrna-data-with-phyloseq.html"/>
<link rel="next" href="analyzing-16s-rrna-data-with-phyloseq.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="assets/style.css" type="text/css" />
<link rel="stylesheet" href="assets/C-MOOR_Template/c-moor-ottr.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.c-moor.org/" target="_blank"><img src="assets/C-MOOR_Template/c-moor-logo-vertical.png" style="width: 70%; padding-left: 10px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#learning-goals"><i class="fa fa-check"></i>Learning Goals</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="motivation.html"><a href="motivation.html"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="target-audience.html"><a href="target-audience.html"><i class="fa fa-check"></i><b>1.2</b> Target Audience</a></li>
<li class="chapter" data-level="1.3" data-path="curriculum.html"><a href="curriculum.html"><i class="fa fa-check"></i><b>1.3</b> Curriculum</a></li>
</ul></li>
<li class="part"><span><b>Scientific Literature</b></span></li>
<li class="chapter" data-level="2" data-path="scientific-literature.html"><a href="scientific-literature.html"><i class="fa fa-check"></i><b>2</b> Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.1" data-path="pre-lab.html"><a href="pre-lab.html"><i class="fa fa-check"></i><b>2.1</b> Pre-lab</a></li>
<li class="chapter" data-level="2.2" data-path="lecture---reading-a-paper.html"><a href="lecture---reading-a-paper.html"><i class="fa fa-check"></i><b>2.2</b> Lecture - Reading a paper</a></li>
<li class="chapter" data-level="2.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html"><i class="fa fa-check"></i><b>2.3</b> Activity - Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#purpose"><i class="fa fa-check"></i><b>2.3.1</b> Purpose</a></li>
<li class="chapter" data-level="2.3.2" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#introduction-1"><i class="fa fa-check"></i><b>2.3.2</b> Introduction</a></li>
<li class="chapter" data-level="2.3.3" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#instructions"><i class="fa fa-check"></i><b>2.3.3</b> Instructions</a></li>
<li class="chapter" data-level="2.3.4" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-1---overview-of-the-paper"><i class="fa fa-check"></i><b>2.3.4</b> Activity 1 - Overview of the paper</a></li>
<li class="chapter" data-level="2.3.5" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#activity-2---methods-and-figure-analysis-in-class-and-homework"><i class="fa fa-check"></i><b>2.3.5</b> Activity 2 - Methods and Figure Analysis (in class and homework)</a></li>
<li class="chapter" data-level="2.3.6" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#grading-criteria"><i class="fa fa-check"></i><b>2.3.6</b> Grading criteria</a></li>
<li class="chapter" data-level="2.3.7" data-path="activity---scientific-literature.html"><a href="activity---scientific-literature.html#footnotes"><i class="fa fa-check"></i><b>2.3.7</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Project: 16S rRNA Research</b></span></li>
<li class="chapter" data-level="3" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html"><i class="fa fa-check"></i><b>3</b> Meet the MISO study</a>
<ul>
<li class="chapter" data-level="3.0.1" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#miso-study-diet-and-sampling-schedule"><i class="fa fa-check"></i><b>3.0.1</b> MISO study diet and sampling schedule</a></li>
<li class="chapter" data-level="3.0.2" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#miso-study-variables-and-factors"><i class="fa fa-check"></i><b>3.0.2</b> MISO study variables and factors</a></li>
<li class="chapter" data-level="3.0.3" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#amplicon-sequence-variants-asvs-data"><i class="fa fa-check"></i><b>3.0.3</b> Amplicon sequence variants (ASVs) data</a></li>
<li class="chapter" data-level="3.0.4" data-path="meet-the-miso-study.html"><a href="meet-the-miso-study.html#footnotes-1"><i class="fa fa-check"></i><b>3.0.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="s-rrna-data-in-spreadsheets.html"><a href="s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>4</b> 16S rRNA Data in spreadsheets</a>
<ul>
<li class="chapter" data-level="4.1" data-path="exploring-16s-rrna-data-in-spreadsheets.html"><a href="exploring-16s-rrna-data-in-spreadsheets.html"><i class="fa fa-check"></i><b>4.1</b> Exploring 16S rRNA data in spreadsheets</a></li>
<li class="chapter" data-level="4.2" data-path="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><a href="lecture---introducing-16s-rrna-data-using-spreadsheets.html"><i class="fa fa-check"></i><b>4.2</b> Lecture - Introducing 16S rRNA Data Using Spreadsheets</a></li>
<li class="chapter" data-level="4.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html"><i class="fa fa-check"></i><b>4.3</b> Activity - Introducing 16S rRNA Data Using Spreadsheet</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#learning-objectives"><i class="fa fa-check"></i><b>4.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="4.3.2" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#introduction-2"><i class="fa fa-check"></i><b>4.3.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3.3" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-1-quick-data-overview-based-on-1-sample"><i class="fa fa-check"></i><b>4.3.3</b> Activity 1 – Quick data overview based on 1 sample</a></li>
<li class="chapter" data-level="4.3.4" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-2-explore-variation-between-individuals"><i class="fa fa-check"></i><b>4.3.4</b> Activity 2 – Explore variation between individuals</a></li>
<li class="chapter" data-level="4.3.5" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-3-explore-variation-within-a-single-individual-associated-with-diet"><i class="fa fa-check"></i><b>4.3.5</b> Activity 3 – Explore variation within a single individual associated with diet</a></li>
<li class="chapter" data-level="4.3.6" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#activity-4-try-it-out"><i class="fa fa-check"></i><b>4.3.6</b> Activity 4 – Try it Out!</a></li>
<li class="chapter" data-level="4.3.7" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#grading-criteria-1"><i class="fa fa-check"></i><b>4.3.7</b> Grading criteria</a></li>
<li class="chapter" data-level="4.3.8" data-path="activity---introducing-16s-rrna-data-using-spreadsheet.html"><a href="activity---introducing-16s-rrna-data-using-spreadsheet.html#footnotes-2"><i class="fa fa-check"></i><b>4.3.8</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="exploring-16s-rrna-data-with-phyloseq.html"><a href="exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5</b> Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="5.1" data-path="exploring-16s-rrna-data-with-phyloseq-1.html"><a href="exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>5.1</b> Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="5.2" data-path="lecture---exploring-16s-rrna-data-with-phyloseq.html"><a href="lecture---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5.2</b> Lecture - Exploring 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="5.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>5.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#learning-objectives-1"><i class="fa fa-check"></i><b>5.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="5.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#introduction-3"><i class="fa fa-check"></i><b>5.3.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-1-explore-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>5.3.3</b> Activity 1 – Explore 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="5.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-2-try-it-out"><i class="fa fa-check"></i><b>5.3.4</b> Activity 2 – Try it out!</a></li>
<li class="chapter" data-level="5.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#grading-criteria-2"><i class="fa fa-check"></i><b>5.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="5.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#footnotes-3"><i class="fa fa-check"></i><b>5.3.6</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="analyzing-16s-rrna-data-with-phyloseq.html"><a href="analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>6</b> Analyzing 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="6.1" data-path="analyzing-16s-rrna-data-with-phyloseq-1.html"><a href="analyzing-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>6.1</b> Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="6.2" data-path="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><a href="lecture---analyzing-16s-rrna-data-with-phyloseq.html"><i class="fa fa-check"></i><b>6.2</b> Lecture - Analyzing 16S rRNA Data with phyloseq</a></li>
<li class="chapter" data-level="6.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><i class="fa fa-check"></i><b>6.3</b> Activity - Exploring 16S rRNA Data with phyloseq</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#learning-objectives-2"><i class="fa fa-check"></i><b>6.3.1</b> Learning objectives</a></li>
<li class="chapter" data-level="6.3.2" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#introduction-4"><i class="fa fa-check"></i><b>6.3.2</b> Introduction</a></li>
<li class="chapter" data-level="6.3.3" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-1-analyze-16s-rrna-data-with-phyloseq-tutorial"><i class="fa fa-check"></i><b>6.3.3</b> Activity 1 – Analyze 16S rRNA Data with phyloseq tutorial</a></li>
<li class="chapter" data-level="6.3.4" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#activity-2-try-it-out-1"><i class="fa fa-check"></i><b>6.3.4</b> Activity 2 – Try it out!</a></li>
<li class="chapter" data-level="6.3.5" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#grading-criteria-3"><i class="fa fa-check"></i><b>6.3.5</b> Grading criteria</a></li>
<li class="chapter" data-level="6.3.6" data-path="activity---exploring-16s-rrna-data-with-phyloseq-1.html"><a href="activity---exploring-16s-rrna-data-with-phyloseq-1.html#footnotes-4"><i class="fa fa-check"></i><b>6.3.6</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Course Name</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=10.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/C-MOOR_Template/dasl_thin_main_image.png">
</div>
<div id="activity---exploring-16s-rrna-data-with-phyloseq" class="section level2 hasAnchor" number="5.3">
<h2><span class="header-section-number">5.3</span> Activity - Exploring 16S rRNA Data with phyloseq<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity---exploring-16s-rrna-data-with-phyloseq" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>To explore bacterial diversity based on 16S rRNA gene sequencing using data from the “Impact of a 7-day homogeneous diet on interpersonal variation in human gut microbiomes and metabolomes” by Guthrie et al., 2023. This study is also referred to as the MISO study for “Microbiome Individuality and Stability Over Time” because the study aims to understand variation (or stability) of microbiomes across individuals.</p>
<p>This activity aims to explore and profile the metagenomic diversity using the R/Bioconductor package phyloseq which leverages various R-based tools to produce publication-quality taxonomy profiling and analysis graphics.</p>
<div id="learning-objectives-1" class="section level3 hasAnchor" number="5.3.1">
<h3><span class="header-section-number">5.3.1</span> Learning objectives<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#learning-objectives-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li>Use phyloseq to explore data associated with Amplicon Sequence Variants (ASVs).</li>
<li>In phyloseq, subset data based on metadata and taxonomy.</li>
<li>In phyloseq, profile and plot taxonomy.</li>
</ol>
</div>
<div id="introduction-3" class="section level3 hasAnchor" number="5.3.2">
<h3><span class="header-section-number">5.3.2</span> Introduction<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#introduction-3" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The most popular sequencing technique for the analysis of bacterial diversity is targeted sequencing, or sequencing of a specific gene (or region of a gene, e.g. a hypervariable region of the bacterial 16S ribosomal rRNA gene) using polymerase chain reaction (PCR) to create sequences called amplicons. Sequence variation in the resulting amplicons creates amplicon sequence variants (ASVs). ASVs varying from as little as one single nucleotide can be defined as separate ASVs, which can be further clustered into OTUs (Operational Taxonomic Units) based on sequence similarity; e.g. as ASVs within 1% sequence difference can clustered into the same species/OTU.</p>
</div>
<div id="activity-1-explore-16s-rrna-data-with-phyloseq-tutorial" class="section level3 hasAnchor" number="5.3.3">
<h3><span class="header-section-number">5.3.3</span> Activity 1 – Explore 16S rRNA Data with phyloseq tutorial<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-1-explore-16s-rrna-data-with-phyloseq-tutorial" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 25 min</em></p>
<p>Explore a phyloseq object through the “Explore 16S rRNA Data with phyloseq” tutorial on SciServer.
Log into SciServer, click on compute, and create a new C-MOOR LearnR container. When creating a container, remember to:
1. Use the “C-MOOR LearnR (Bioconductor 3.17)” image (not the Bioconductor 3.16)
2. Check the box next to Data volume “C-MOOR Data”
3. Start the “Explore 16S rRNA Data with phyloseq” tutorial. Visit SciServer Guides and FAQs. If you need assistance accessing the tutorial.
4. To move through the activities click “Continue” at the bottom of the screen. When you are done with a topic, click “Next Topic” to move on.
5. This tutorial has small boxes in which you can enter and run short lines of code to analyze the data.
As you work through the tutorial, take snapshots of your work and paste your answers in the grey boxes below:</p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1-1. OTU_table Exercise – Take a snapshot and paste your code and the output for the following question in the tutorial: “What is the normalized count for ASV115 in sample 10?”</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1-2. Visualizing Taxonomy Exercise – Take a snapshot and paste your code and the output for the following question in the tutorial: “What timepoint has the lowest proportion of phylum Verrucomicrobia?”</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1-3. Using subset_taxa() Exercise – Take a snapshot and paste your code and the output for the following question in the tutorial: “What are the 2 most abundant orders in the class Gammaproteobacteria?”</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="activity-2-try-it-out" class="section level3 hasAnchor" number="5.3.4">
<h3><span class="header-section-number">5.3.4</span> Activity 2 – Try it out!<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#activity-2-try-it-out" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 60 min</em></p>
<p>With your group, perform some exploratory data analysis selecting from one of the four questions below or coming up with your own question. When you are done, copy and paste your results into the class poster.<br />
<a href="https://docs.google.com/presentation/d/1eE5AltbkW6km0r3uQO8KKnUVRLCLx70SDNuBHp6aASU/edit?slide=id.g343cb6dd56c_2_0#slide=id.g343cb6dd56c_2_0">Class Poster</a></p>
<div id="question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality" class="section level4 hasAnchor" number="5.3.4.1">
<h4><span class="header-section-number">5.3.4.1</span> Question 1 - What has more impact on human microbial variation, diet or individuality?<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#question-1---what-has-more-impact-on-human-microbial-variation-diet-or-individuality" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Approach: Plot high level taxa (Phyla) for all timepoints and subjects and compare the plots to determine which factor has a stronger effect on microbiome composition.</p>
<p><strong>Step 1A: Plot high-level taxonomy (Phylum level) for each timepoint using the plot_bar() function.</strong> Specifically, your plot should show Phyla diversity across all 5 timepoints.
- Refer to the “Explore 16S rRNA Data with phyloseq” tutorial for help using the plot_bar() function.
- Ensure your plot has a title
- Remember, the BD diet includes timepoints 1 and 2, the HD diet includes timepoints 3 and 4, and the WO diet includes timepoint 5.
- Use the following code as a template:</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb3-1" tabindex="-1"></a><span class="fu">plot_bar</span>(miso, <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;choose a name for your graph&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb3-2" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">1A-1. Paste figure 1A below.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>1A-2. State your conclusion for Figure 1A below.</strong>
- Include a list of the identified Phyla and their names
- Include if you think there is a difference between conditions and your reasoning why or why not
||
|:–|
| <br> |
<br></p>
<p>Step 1B: Plot high-level taxonomy (Phylum level) for each individual, using only normalized homogenized diet “HD” timepoints to minimize contribution of diet to the analysis using the plot_bar() function. Specifically, your plot should show Phyla diversity across all 21 individuals at HD timepoints.</p>
<ul>
<li>Use the command below to create a phyloseq object with only “HD” timepoints</li>
</ul>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb4-1" tabindex="-1"></a>misoHD <span class="ot">=</span> <span class="fu">subset_samples</span>(miso, timepointgroup <span class="sc">==</span> <span class="st">&quot;HD&quot;</span>)</span></code></pre></div>
<ul>
<li>Refer to the “Explore 16S rRNA Data with phyloseq” tutorial for help using the plot_bar() function.</li>
<li>Ensure your plot has a title</li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">1B-1. Paste figure 1B below.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>1B-2. State your conclusion for Figure 1B below.</strong>
- Include a list of the identified Phyla and their names
- Include if you think there is a difference between conditions and your reasoning why or why not
||
|:–|
| <br> |
<br></p>
<p>A suggestion for your poster:
1. Combine plot 1A-1 and 1B-1 into a single figure. Include their titles and short figure description (figure legend).
1. Paste the code you used to generate your final figure(s) below:
||
|:–|
| <br> |
<br></p>
</div>
<div id="question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions" class="section level4 hasAnchor" number="5.3.4.2">
<h4><span class="header-section-number">5.3.4.2</span> Question 2 - What are some of the most abundant microbes when viewed at different taxonomic resolutions?<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#question-2---what-are-some-of-the-most-abundant-microbes-when-viewed-at-different-taxonomic-resolutions" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Approach: Plot progressively lower level taxa for all individuals, subsetting most abundant from each rank.</p>
<p><strong>Step 2A. Plot Phyla across all subjects and identify the most abundant Phyla.</strong> Specifically, your plot should show Phyla diversity across all 21 individuals.
- Refer to the “Explore 16S rRNA Data with phyloseq” tutorial for help using the plot_bar() function.
- Use the following code as a template:</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb5-1" tabindex="-1"></a><span class="fu">plot_bar</span>(miso, <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;choose a name for your graph&quot;</span>) <span class="sc">+</span> <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2A-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">2A-2. Enter the name of the most abundant Phyla below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2B. Subset the most abundant Phyla, and plot Order (Skip Class!)</strong>
- Refer to the “Subset taxonomy” section of the “Explore 16S rRNA Data with phyloseq” tutorial for help using the subset_taxa and plot_bar() functions.
- Use the following code as a template:</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb6-1" tabindex="-1"></a>subset <span class="ot">=</span> <span class="fu">subset_taxa</span>(miso, Phylum <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>)</span>
<span id="cb6-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb6-2" tabindex="-1"></a><span class="fu">plot_bar</span>(subset, <span class="st">&quot;subject&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;Order&quot;</span>, <span class="at">title =</span> <span class="st">&quot;choose a name for your graph&quot;</span>) <span class="sc">+</span> </span>
<span id="cb6-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb6-3" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2B-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">2A-2. Enter the name of the most abundant Order below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2C. Subset the most abundant Order, and plot associated Family</strong>
- Refer to the “Subset taxonomy” section of the “Explore 16S rRNA Data with phyloseq” tutorial for help using the subset_taxa and plot_bar() functions.
- Use the following code as a template:</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb7-1" tabindex="-1"></a>subset <span class="ot">=</span> <span class="fu">subset_taxa</span>(miso, Order <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>)</span>
<span id="cb7-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb7-2" tabindex="-1"></a><span class="fu">plot_bar</span>(subset, <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;Family&quot;</span>, <span class="at">title =</span> <span class="st">&quot;choose a name for your graph&quot;</span>) <span class="sc">+</span> </span>
<span id="cb7-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb7-3" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2C-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">2C-2. Enter the name of the most abundant Family below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2D. Subset most abundant Family, and plot associated Genus</strong>
- Refer to the “Subset taxonomy” section of the “Explore 16S rRNA Data with phyloseq” tutorial for help using the subset_taxa and plot_bar() functions.
- Use the following code as a template:</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb8-1" tabindex="-1"></a>subset <span class="ot">=</span> <span class="fu">subset_taxa</span>(miso, Family <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>)</span>
<span id="cb8-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb8-2" tabindex="-1"></a><span class="fu">plot_bar</span>(subset, <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;Genus&quot;</span>, <span class="at">title =</span> <span class="st">&quot;choose a name for your graph&quot;</span>) <span class="sc">+</span> </span>
<span id="cb8-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb8-3" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2D-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2D-2. Enter the name of the most abundant Genus of your chosen Family below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 2E. Subset the most abundant Genus, and plot associated Species</strong>
- Refer to the “Subset taxonomy” section of the “Explore 16S rRNA Data with phyloseq” tutorial for help using the subset_taxa and plot_bar() functions.
- Use the following code as a template:</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb9-1" tabindex="-1"></a>subset <span class="ot">=</span> <span class="fu">subset_taxa</span>(miso, Genus <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>)</span>
<span id="cb9-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb9-2" tabindex="-1"></a><span class="fu">plot_bar</span>(subset, <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;choose a name for your graph&quot;</span>) <span class="sc">+</span> </span>
<span id="cb9-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb9-3" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<ul>
<li>Note, given the many different Species associated with an abundant Genus, we may need to reduce the legend text and key size in order to visualize the plot better.</li>
<li>You can do so by adding the following lines of code:</li>
</ul>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb10-1" tabindex="-1"></a><span class="fu">theme</span>(<span class="at">legend.text=</span><span class="fu">element_text</span>(<span class="at">size=</span><span class="dv">6</span>)) <span class="sc">+</span> </span>
<span id="cb10-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb10-2" tabindex="-1"></a><span class="fu">theme</span>(<span class="at">legend.key.size =</span> <span class="fu">unit</span>(<span class="dv">6</span>, <span class="st">&quot;pt&quot;</span>))</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">2E-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">2E-2. Enter the name of the some of the most abundant species below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2E-3. After looking through all your plots, summarize your conclusion regarding individual microbiome composition diversity. Explain what reasoning led you to your conclusion.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>A suggestion for your poster:
- Combine 4 plots into a single Figure showing your subsetting of the most abundant Order, Family, Genus, and Species. Skip showing Phyla because the group working on Activity 1 will have Phyla data and show a plot for that. Include title and short figure description (figure legend).
- Paste the code you used to generate your final figure(s) below:
| |
|:–|
| <br> |
<br></p>
</div>
<div id="question-3---does-gender-contribute-to-individual-microbiome-variation" class="section level4 hasAnchor" number="5.3.4.3">
<h4><span class="header-section-number">5.3.4.3</span> Question 3 - Does gender contribute to individual microbiome variation?<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#question-3---does-gender-contribute-to-individual-microbiome-variation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Approach: First survey the differences in microbiome across gender at the level of phyla and species, and identify potential candidates that may differ between males and females. Then, plot your candidates of interest and additional candidate microbes previously associated with gender-based variation.</p>
<p><strong>Step 3A. Perform an initial survey to see if there are any differences in phyla and species composition between genders.</strong> Specifically, your plot should show phyla diversity within male and female groups.
- Refer to the “Explore 16S rRNA Data with phyloseq” tutorial for help using the plot_bar() function.
- Use the following code as a template:</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb11-1" tabindex="-1"></a><span class="fu">plot_bar</span>(miso, <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb11-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb11-2" tabindex="-1"></a><span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">3A-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p>Consider any initial differences you see, then choose one high-abundant and one low-abundant candidate Phylum that shows hints of being differentially abundant between genders.</p>
<table>
<thead>
<tr class="header">
<th align="left">3A-2. Record the phylum you chose:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 3B. Test gender-based difference in Species for the high abundant phylum you selected.</strong> Specifically, your plot should show species diversity within male and female groups for only your phylum of choice.</p>
<ul>
<li>Refer to the “Subset taxonomy” section of the “Explore 16S rRNA Data with phyloseq” tutorial for help using the subset_taxa and plot_bar() functions.</li>
<li>Use the following code as a starter:</li>
</ul>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb12-1" tabindex="-1"></a>subset <span class="ot">=</span> <span class="fu">subset_taxa</span>(miso, Phylum <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>)</span>
<span id="cb12-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb12-2" tabindex="-1"></a></span>
<span id="cb12-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb12-3" tabindex="-1"></a><span class="fu">plot_bar</span>(subset, <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;Species&quot;</span>, <span class="at">title =</span> <span class="st">&quot;choose a name for your graph&quot;</span>) <span class="sc">+</span> </span>
<span id="cb12-4"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb12-4" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<ul>
<li>Note, given the many different Species associated with an abundant Genus, we may need to reduce the legend text and key size in order to visualize the plot better.</li>
<li>You can do so by adding the following lines of code:</li>
</ul>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb13-1" tabindex="-1"></a><span class="fu">theme</span>(<span class="at">legend.text=</span><span class="fu">element_text</span>(<span class="at">size=</span><span class="dv">6</span>)) <span class="sc">+</span> </span>
<span id="cb13-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb13-2" tabindex="-1"></a><span class="fu">theme</span>(<span class="at">legend.key.size =</span> <span class="fu">unit</span>(<span class="dv">6</span>, <span class="st">&quot;pt&quot;</span>))</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">3B-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">3B-2. Do you observe any differences in species based on gender?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 3C. Test gender-based difference in Species for the low abundant phylum you selected.</strong> Specifically, your plot should show species diversity within male and female groups for only your phylum of choice.
- Refer to the “Subset taxonomy” section of the “Explore 16S rRNA Data with phyloseq” tutorial for help using the subset_taxa and plot_bar() functions.
- Use the code you used in Step 3B as a template for your code</p>
<table>
<thead>
<tr class="header">
<th align="left">3C-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 3D: Plot the individual abundance of 2 candidate microbes for gender differences based on your analysis above, and/or the curated menu of microbes below (previously associated with gender variation), with color denoting gender.</strong> Specifically, your plot should show the chosen microbial group across 21 subjects with the bars colored by gender.</p>
<ul>
<li>Refer to the “Subset taxonomy” section of the “Explore 16S rRNA Data with phyloseq” tutorial for help using the subset_taxa and plot_bar() functions.</li>
<li>Use the following code as a template:</li>
</ul>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb14-1" tabindex="-1"></a>subset <span class="ot">=</span> <span class="fu">subset_taxa</span>(miso,  Phylum <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>)</span>
<span id="cb14-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb14-2" tabindex="-1"></a><span class="fu">plot_bar</span>(subset, <span class="st">&quot;subject&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;gender&quot;</span>, <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb14-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb14-3" tabindex="-1"></a> <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> gender, <span class="at">fill =</span> gender), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<p>Here is the candidate list of different microbes previously implicated in gender-based variation.
- Phylum: Proteobacteria
- Order: Clostridiales
- Family: Ruminococcaceae
- Genus: Prevotella
- Genus: Fusobacterium
- Species: muciniphila</p>
<table>
<thead>
<tr class="header">
<th align="left">3D-1. Insert the plot from your first chosen microbial group below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">3D-2. Insert the plot from your second chosen microbial group below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3D-3. Based on the all graphs you made, did you identify any candidate Phylum or Species that may differ between genders?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3D-4. Based on the graphs you made, what conclusion can you make about microbiome differences between genders?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3D-5. Which seems to have more effect on microbial compositions? Gender or individual subject?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>A suggestion for your poster:</strong>
- Combine the 2 plots that cover the entire dataset (3A-1 and 3B-1) and 2 plots from exploring microbial groups (3D) to make one figure summarizing your findings for gender based differences in microbial composition.
- Include titles and a short figure description (figure legend).
- Paste the code you used to generate your final figure(s) below:
| |
|:–|
| <br> |
<br></p>
</div>
<div id="question-4---does-age-contribute-to-individual-microbiome-variation" class="section level4 hasAnchor" number="5.3.4.4">
<h4><span class="header-section-number">5.3.4.4</span> Question 4 - Does age contribute to individual microbiome variation?<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#question-4---does-age-contribute-to-individual-microbiome-variation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Approach: Survey the differences in microbiome across age, refining the plot to accommodate a continuous variable (age) and unequal sample sizes (some individuals in the study are the same age). Then, test if there is a shift in microbiome abundance of phyla Firmicutes and Bacteroidetes, two phyla suggested previously to change with age.</p>
<p><strong>Step 4A. Perform an initial survey to see if there are any differences in phyla related to age.</strong> Specifically, your plot should show the relationship phyla diversity and age of an individual.
- Refer to the “Explore 16S rRNA Data with phyloseq” tutorial for help using the plot_bar() function.
- Use the following code as a template
plot_bar(miso, “fill in the blank”, fill = “fill in the blank”, title = “fill in the blank”) +
geom_bar(aes(color = fill in the blank, fill = fill in the blank), stat = “identity”, position = “stack”)</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb15-1" tabindex="-1"></a><span class="fu">plot_bar</span>(miso, <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb15-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb15-2" tabindex="-1"></a><span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">4A-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<ul>
<li>You probably found it difficult to discern any differences.</li>
<li>Abundance is additive, so having more samples of the same age will result in adding the abundances together, so if there are say 2 samples with age 27 (as is the case), the abundance of the bar will be much higher than other ages that have only one individual.</li>
</ul>
<p><strong>Step 4B. Merge samples by age, then re-normalize and finally re-plot Phyla by age.</strong></p>
<ol style="list-style-type: decimal">
<li>Merge samples of the same age, saving the resulting phyloseq object as ‘merge’. You will probably get a warning message letting you know “NAs were introduced by coercion.” That is OK.</li>
</ol>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb16-1" tabindex="-1"></a>merge <span class="ot">&lt;-</span> <span class="fu">merge_samples</span>(miso, <span class="st">&quot;age&quot;</span>)</span></code></pre></div>
<ol start="2" style="list-style-type: decimal">
<li>Check your merged phyloseq object. Your new ‘merge’ object should have 16 samples corresponding to 16 unique ages in the MISO dataset.</li>
</ol>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb17-1" tabindex="-1"></a>merge</span>
<span id="cb17-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb17-2" tabindex="-1"></a></span>
<span id="cb17-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb17-3" tabindex="-1"></a>phyloseq<span class="sc">-</span>class experiment<span class="sc">-</span>level object</span>
<span id="cb17-4"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb17-4" tabindex="-1"></a><span class="fu">otu_table</span>()   OTU Table<span class="sc">:</span>         [ <span class="dv">1702</span> taxa and <span class="dv">16</span> samples ]</span>
<span id="cb17-5"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb17-5" tabindex="-1"></a><span class="fu">sample_data</span>() Sample Data<span class="sc">:</span>       [ <span class="dv">16</span> samples by <span class="dv">32</span> sample variables ]</span>
<span id="cb17-6"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb17-6" tabindex="-1"></a><span class="fu">tax_table</span>()   Taxonomy Table<span class="sc">:</span>    [ <span class="dv">1702</span> taxa by <span class="dv">9</span> taxonomic ranks ]</span></code></pre></div>
<ol start="3" style="list-style-type: decimal">
<li>Normalize the sample counts using the function transform_sample_counts. Our function, (100* x/sum(x)) will transform all the samples into proportions.</li>
</ol>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb18-1" tabindex="-1"></a>merge <span class="ot">&lt;-</span> <span class="fu">transform_sample_counts</span>(merge, <span class="cf">function</span>(x) <span class="dv">100</span> <span class="sc">*</span> x<span class="sc">/</span><span class="fu">sum</span>(x))</span></code></pre></div>
<ol start="4" style="list-style-type: decimal">
<li>Re-plot Phyla by age.</li>
</ol>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb19-1" tabindex="-1"></a><span class="fu">plot_bar</span>(merge, <span class="at">x=</span> <span class="st">&quot;age&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span></span>
<span id="cb19-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb19-2" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<ol start="5" style="list-style-type: decimal">
<li>If you need to decrease the size of your legend key and text to fit, add code below to your plot after adding a + to the end of the geom_bar line:</li>
</ol>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb20-1" tabindex="-1"></a><span class="fu">theme</span>(<span class="at">legend.text=</span><span class="fu">element_text</span>(<span class="at">size=</span><span class="dv">6</span>)) <span class="sc">+</span> </span>
<span id="cb20-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb20-2" tabindex="-1"></a><span class="fu">theme</span>(<span class="at">legend.key.size =</span> <span class="fu">unit</span>(<span class="dv">6</span>, <span class="st">&quot;pt&quot;</span>))</span></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">4B-1. Insert the resulting plot below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">4B-2. Do you observe any candidate Phyla that changes with age?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Step 4C. Candidate Phyla implicated in age variation include Firmicutes and Bacteroidetes. Plot these specific phyla on their own, separately, to observe any shift that occurs with age.</strong> Use your new merged and normalized ‘merge’ phyloseq object as input.
- Refer to the “Subset taxonomy” section of the “Explore 16S rRNA Data with phyloseq” tutorial for help using the subset_taxa and plot_bar() functions.
- Use the following code as a template:</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb21-1" tabindex="-1"></a>subset <span class="ot">=</span> <span class="fu">subset_taxa</span>(merge,  Phylum <span class="sc">==</span> <span class="st">&quot;fill in the blank&quot;</span>)</span>
<span id="cb21-2"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb21-2" tabindex="-1"></a><span class="fu">plot_bar</span>(subset, <span class="st">&quot;age&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;fill in the blank&quot;</span>, <span class="at">title =</span> <span class="st">&quot;fill in the blank&quot;</span>) <span class="sc">+</span> </span>
<span id="cb21-3"><a href="activity---exploring-16s-rrna-data-with-phyloseq.html#cb21-3" tabindex="-1"></a> <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">color =</span> fill <span class="cf">in</span> the blank, <span class="at">fill =</span> fill <span class="cf">in</span> the blank), <span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">position =</span> <span class="st">&quot;stack&quot;</span>)</span></code></pre></div>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">4C-1. Insert the plot that graphs the relationship of age to the Phylum Firmicutes below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">4C-2. Do you observe any changes in Phyla based on age?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">4C-3. Insert the plot that graphs the relationship of age to the Phylum Bacteroidetes below:</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">4C-4 Do you observe any changes in Phyla based on age?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Step 4C-5: Based on the graphs you made, did you identify any candidate Phylum that may differ between genders?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><br></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>A suggestion for your poster:</strong>
- Combine 2-4 of your plots to make one figure summarizing Phylum and Species age-based differences you observed in the MISO study. Include titles and short figure description (figure legend).
- Paste the code you used to generate your final figure(s) below:
| |
|:–|
| <br> |
<br></p>
</div>
</div>
<div id="grading-criteria-2" class="section level3 hasAnchor" number="5.3.5">
<h3><span class="header-section-number">5.3.5</span> Grading criteria<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#grading-criteria-2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Download this assignment as Microsoft Word (.docx) and upload on Canvas</li>
</ul>
</div>
<div id="footnotes-3" class="section level3 hasAnchor" number="5.3.6">
<h3><span class="header-section-number">5.3.6</span> Footnotes<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#footnotes-3" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="resources-3" class="section level4 hasAnchor" number="5.3.6.1">
<h4><span class="header-section-number">5.3.6.1</span> Resources<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#resources-3" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><a href="https://docs.google.com/document/d/1dTRjAHHXeuzXqBOl-h80MRvNxxpMKfT-F_YLIKaxzmo/edit?usp=sharing">Google Doc</a></p>
</div>
<div id="contributions-and-affiliations-3" class="section level4 hasAnchor" number="5.3.6.2">
<h4><span class="header-section-number">5.3.6.2</span> Contributions and affiliations<a href="activity---exploring-16s-rrna-data-with-phyloseq.html#contributions-and-affiliations-3" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>Valeriya Gaysinskaya, Johns Hopkins University</li>
<li>Gauri Paul, Clovis Community College</li>
<li>Frederick Tan, Johns Hopkins University</li>
<li>Sayumi York, Notre Dame of Maryland University</li>
</ul>

<!-- Set up code of OTTR Book-->
</div>
</div>
</div>
<!-- </div> -->
<hr>
<center> 
  <div class="footer">
      All materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="lecture---exploring-16s-rrna-data-with-phyloseq.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="analyzing-16s-rrna-data-with-phyloseq.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
